--- a/Source/WebCore/rendering/RenderBox.h
+++ b/Source/WebCore/rendering/RenderBox.h
@@ -34,5 +34,7 @@
 #include <WebCore/ShapeOutsideInfo.h>
 #include <WebCore/StyleUnevaluatedCalculation.h>
 #include <wtf/TypeCasts.h>
+
+#include <cmath>
 
 namespace WebCore {
@@ -695,6 +697,6 @@
 
     void computePreferredLogicalWidths(const Style::MinimumSize& minLogicalWidth, const Style::MaximumSize& maxLogicalWidth, LayoutUnit borderAndPaddingLogicalWidth);
 
-    bool isAspectRatioDegenerate(double aspectRatio) const { return !aspectRatio || isnan(aspectRatio); }
+    bool isAspectRatioDegenerate(double aspectRatio) const { return !aspectRatio || std::isnan(aspectRatio); }
 
     bool overflowChangesMayAffectLayout() const final;
