--- a/Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackAndGenerateCode.cpp	2026-05-20 15:14:23.568856789 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackAndGenerateCode.cpp	2026-05-20 15:17:50.706337780 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "AirAllocateRegistersAndStackAndGenerateCode.h"
 
--- a/Source/JavaScriptCore/b3/air/AirAllocateRegistersByGraphColoring.cpp	2026-05-20 15:14:23.569856799 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateRegistersByGraphColoring.cpp	2026-05-20 15:17:50.728337934 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "AirAllocateRegistersByGraphColoring.h"
 
--- a/Source/JavaScriptCore/b3/air/AirAllocateRegistersByGreedy.cpp	2026-05-20 15:14:23.570856808 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateRegistersByGreedy.cpp	2026-05-20 15:17:50.751338096 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "AirAllocateRegistersByGreedy.h"
 
--- a/Source/JavaScriptCore/b3/air/AirAllocateStackByGraphColoring.cpp	2026-05-20 15:14:23.570856808 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateStackByGraphColoring.cpp	2026-05-20 15:17:50.774338258 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "AirAllocateStackByGraphColoring.h"
 
--- a/Source/JavaScriptCore/b3/air/AirArg.h	2026-05-20 15:14:23.546856573 +0000
+++ b/Source/JavaScriptCore/b3/air/AirArg.h	2026-05-20 15:17:50.795338406 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/air/AirCCallingConvention.cpp	2026-05-20 15:14:23.571856818 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCCallingConvention.cpp	2026-05-20 15:17:50.817338561 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "AirCCallingConvention.h"
 
--- a/Source/JavaScriptCore/b3/air/AirCode.cpp	2026-05-20 15:14:23.571856818 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCode.cpp	2026-05-20 15:17:50.840338723 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "AirCode.h"
 
--- a/Source/JavaScriptCore/b3/air/AirCode.h	2026-05-20 15:14:23.547856582 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCode.h	2026-05-20 15:17:50.862338877 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/air/AirDisassembler.cpp	2026-05-20 15:14:23.572856828 +0000
+++ b/Source/JavaScriptCore/b3/air/AirDisassembler.cpp	2026-05-20 15:17:50.882339018 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "AirDisassembler.h"
 
--- a/Source/JavaScriptCore/b3/air/AirFixObviousSpills.cpp	2026-05-20 15:14:23.572856828 +0000
+++ b/Source/JavaScriptCore/b3/air/AirFixObviousSpills.cpp	2026-05-20 15:17:50.903339166 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "AirFixObviousSpills.h"
 
--- a/Source/JavaScriptCore/b3/air/AirFormTable.h	2026-05-20 15:14:23.549856602 +0000
+++ b/Source/JavaScriptCore/b3/air/AirFormTable.h	2026-05-20 15:17:50.925339321 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/air/AirLowerStackArgs.cpp	2026-05-20 15:14:23.574856848 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLowerStackArgs.cpp	2026-05-20 15:17:50.945339462 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "AirLowerStackArgs.h"
 
--- a/Source/JavaScriptCore/b3/air/AirOptimizePairedLoadStore.cpp	2026-05-20 15:14:23.575856858 +0000
+++ b/Source/JavaScriptCore/b3/air/AirOptimizePairedLoadStore.cpp	2026-05-20 15:17:50.965339602 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "AirOptimizePairedLoadStore.h"
 
--- a/Source/JavaScriptCore/b3/air/AirStackSlot.cpp	2026-05-20 15:14:23.576856867 +0000
+++ b/Source/JavaScriptCore/b3/air/AirStackSlot.cpp	2026-05-20 15:17:50.988339764 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "AirStackSlot.h"
 
--- a/Source/JavaScriptCore/b3/air/AirStackSlot.h	2026-05-20 15:14:23.553856641 +0000
+++ b/Source/JavaScriptCore/b3/air/AirStackSlot.h	2026-05-20 15:17:51.011339926 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirStackSlotKind.h	2026-05-20 15:14:23.553856641 +0000
+++ b/Source/JavaScriptCore/b3/air/AirStackSlotKind.h	2026-05-20 15:17:51.032340074 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirTmpWidth.cpp	2026-05-20 15:14:23.576856867 +0000
+++ b/Source/JavaScriptCore/b3/air/AirTmpWidth.cpp	2026-05-20 15:17:51.053340222 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "AirTmpWidth.h"
 
--- a/Source/JavaScriptCore/b3/air/AirUseCounts.h	2026-05-20 15:14:23.554856651 +0000
+++ b/Source/JavaScriptCore/b3/air/AirUseCounts.h	2026-05-20 15:17:51.072340356 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/testair.cpp	2026-05-20 15:14:23.577856877 +0000
+++ b/Source/JavaScriptCore/b3/air/testair.cpp	2026-05-20 15:17:51.091340490 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 
 #include "AirCode.h"
--- a/Source/JavaScriptCore/b3/B3AbstractHeapRepository.cpp	2026-05-20 15:14:23.578856887 +0000
+++ b/Source/JavaScriptCore/b3/B3AbstractHeapRepository.cpp	2026-05-20 15:17:51.111340630 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3AbstractHeapRepository.h"
 
--- a/Source/JavaScriptCore/b3/B3AbstractHeapRepository.h	2026-05-20 15:02:39.837907791 +0000
+++ b/Source/JavaScriptCore/b3/B3AbstractHeapRepository.h	2026-05-20 15:17:51.132340778 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Bank.h	2026-01-20 14:08:31.621707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Bank.h	2026-05-20 15:17:51.155340940 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3BasicBlock.cpp	2026-05-20 15:14:23.578856887 +0000
+++ b/Source/JavaScriptCore/b3/B3BasicBlock.cpp	2026-05-20 15:17:51.175341081 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3BasicBlock.h"
 
--- a/Source/JavaScriptCore/b3/B3BasicBlock.h	2026-05-20 15:05:39.103686956 +0000
+++ b/Source/JavaScriptCore/b3/B3BasicBlock.h	2026-05-20 15:17:51.197341236 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #include <wtf/Platform.h>
--- a/Source/JavaScriptCore/b3/B3Common.h	2026-02-23 14:40:52.230318000 +0000
+++ b/Source/JavaScriptCore/b3/B3Common.h	2026-05-20 15:17:51.219341391 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ComputeDivisionMagic.h	2026-05-20 15:14:23.556856671 +0000
+++ b/Source/JavaScriptCore/b3/B3ComputeDivisionMagic.h	2026-05-20 15:17:51.239341532 +0000
@@ -73,6 +73,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Const32Value.cpp	2026-05-20 15:14:23.581856917 +0000
+++ b/Source/JavaScriptCore/b3/B3Const32Value.cpp	2026-05-20 15:17:51.259341673 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3Const32Value.h"
 
--- a/Source/JavaScriptCore/b3/B3Const32Value.h	2026-05-20 15:14:23.557856681 +0000
+++ b/Source/JavaScriptCore/b3/B3Const32Value.h	2026-05-20 15:17:51.280341821 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3Const64Value.cpp	2026-05-20 15:14:23.581856917 +0000
+++ b/Source/JavaScriptCore/b3/B3Const64Value.cpp	2026-05-20 15:17:51.299341954 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3Const64Value.h"
 
--- a/Source/JavaScriptCore/b3/B3Const64Value.h	2026-05-20 15:14:23.557856681 +0000
+++ b/Source/JavaScriptCore/b3/B3Const64Value.h	2026-05-20 15:17:51.319342095 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3ConstDoubleValue.cpp	2026-05-20 15:14:23.581856917 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstDoubleValue.cpp	2026-05-20 15:17:51.340342243 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3ConstDoubleValue.h"
 
--- a/Source/JavaScriptCore/b3/B3ConstDoubleValue.h	2026-05-20 15:14:23.557856681 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstDoubleValue.h	2026-05-20 15:17:51.362342398 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3ConstFloatValue.cpp	2026-05-20 15:14:23.582856926 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstFloatValue.cpp	2026-05-20 15:17:51.383342546 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3ConstFloatValue.h"
 
--- a/Source/JavaScriptCore/b3/B3ConstFloatValue.h	2026-05-20 15:14:23.557856681 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstFloatValue.h	2026-05-20 15:17:51.402342680 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3Effects.h	2026-05-20 15:14:23.558856691 +0000
+++ b/Source/JavaScriptCore/b3/B3Effects.h	2026-05-20 15:17:51.423342828 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3ExtractValue.h	2026-05-20 15:14:23.559856700 +0000
+++ b/Source/JavaScriptCore/b3/B3ExtractValue.h	2026-05-20 15:17:51.443342968 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3FoldPathConstants.cpp	2026-05-20 15:14:23.583856936 +0000
+++ b/Source/JavaScriptCore/b3/B3FoldPathConstants.cpp	2026-05-20 15:17:51.465343123 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3FoldPathConstants.h"
 
--- a/Source/JavaScriptCore/b3/B3FrequencyClass.h	2026-05-20 15:14:23.559856700 +0000
+++ b/Source/JavaScriptCore/b3/B3FrequencyClass.h	2026-05-20 15:17:51.485343264 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3HeapRange.h	2026-05-20 15:14:23.560856710 +0000
+++ b/Source/JavaScriptCore/b3/B3HeapRange.h	2026-05-20 15:17:51.505343405 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3InferSwitches.cpp	2026-05-20 15:14:23.584856946 +0000
+++ b/Source/JavaScriptCore/b3/B3InferSwitches.cpp	2026-05-20 15:17:51.523343532 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3InferSwitches.h"
 
--- a/Source/JavaScriptCore/b3/B3InsertionSet.cpp	2026-05-20 15:14:23.584856946 +0000
+++ b/Source/JavaScriptCore/b3/B3InsertionSet.cpp	2026-05-20 15:17:51.545343687 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3InsertionSet.h"
 
--- a/Source/JavaScriptCore/b3/B3InsertionSet.h	2026-05-20 15:14:23.560856710 +0000
+++ b/Source/JavaScriptCore/b3/B3InsertionSet.h	2026-05-20 15:17:51.565343828 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3LowerInt64.cpp	2026-02-23 14:40:52.232318200 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerInt64.cpp	2026-05-20 15:17:51.586343976 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include "B3LowerInt64.h"
 
 #if USE(JSVALUE32_64) && ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3LowerMacros.cpp	2026-05-20 15:14:23.585856956 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerMacros.cpp	2026-05-20 15:17:51.607344123 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3LowerMacros.h"
 
--- a/Source/JavaScriptCore/b3/B3LowerToAir32_64.cpp	2026-05-20 15:14:23.586856966 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerToAir32_64.cpp	2026-05-20 15:17:51.629344278 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3LowerToAir.h"
 
--- a/Source/JavaScriptCore/b3/B3LowerToAir.cpp	2026-05-20 15:14:23.587856975 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerToAir.cpp	2026-05-20 15:17:51.653344447 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3LowerToAir.h"
 
--- a/Source/JavaScriptCore/b3/B3MemoryValue.cpp	2026-05-20 15:14:23.587856975 +0000
+++ b/Source/JavaScriptCore/b3/B3MemoryValue.cpp	2026-05-20 15:17:51.678344623 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3MemoryValue.h"
 
--- a/Source/JavaScriptCore/b3/B3MemoryValue.h	2026-05-20 15:14:23.561856720 +0000
+++ b/Source/JavaScriptCore/b3/B3MemoryValue.h	2026-05-20 15:17:51.699344771 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3MemoryValueInlines.h	2026-05-20 15:14:23.562856730 +0000
+++ b/Source/JavaScriptCore/b3/B3MemoryValueInlines.h	2026-05-20 15:17:51.722344933 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3MoveConstants.cpp	2026-05-20 15:14:23.588856985 +0000
+++ b/Source/JavaScriptCore/b3/B3MoveConstants.cpp	2026-05-20 15:17:51.744345088 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3MoveConstants.h"
 
--- a/Source/JavaScriptCore/b3/B3NativeTraits.h	2026-05-20 15:14:23.562856730 +0000
+++ b/Source/JavaScriptCore/b3/B3NativeTraits.h	2026-05-20 15:17:51.763345222 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Opcode.h	2026-05-20 15:14:23.562856730 +0000
+++ b/Source/JavaScriptCore/b3/B3Opcode.h	2026-05-20 15:17:51.783345363 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3Operations.cpp	2026-05-20 15:14:23.588856985 +0000
+++ b/Source/JavaScriptCore/b3/B3Operations.cpp	2026-05-20 15:17:51.803345504 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3Operations.h"
 
--- a/Source/JavaScriptCore/b3/B3Operations.h	2026-02-23 14:40:52.236318000 +0000
+++ b/Source/JavaScriptCore/b3/B3Operations.h	2026-05-20 15:17:51.824345652 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #if ENABLE(B3_JIT)
 
 #include <JavaScriptCore/JITOperationValidation.h>
--- a/Source/JavaScriptCore/b3/B3OptimizeAssociativeExpressionTrees.cpp	2026-05-20 15:14:23.588856985 +0000
+++ b/Source/JavaScriptCore/b3/B3OptimizeAssociativeExpressionTrees.cpp	2026-05-20 15:17:51.846345807 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3OptimizeAssociativeExpressionTrees.h"
 
--- a/Source/JavaScriptCore/b3/B3Origin.h	2026-05-20 15:14:23.563856740 +0000
+++ b/Source/JavaScriptCore/b3/B3Origin.h	2026-05-20 15:17:51.868345962 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3PatchpointValue.h	2026-05-20 15:14:23.563856740 +0000
+++ b/Source/JavaScriptCore/b3/B3PatchpointValue.h	2026-05-20 15:17:51.889346109 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3Procedure.cpp	2026-05-20 15:14:23.589856995 +0000
+++ b/Source/JavaScriptCore/b3/B3Procedure.cpp	2026-05-20 15:17:51.911346264 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3Procedure.h"
 
--- a/Source/JavaScriptCore/b3/B3Procedure.h	2026-05-20 15:14:23.564856749 +0000
+++ b/Source/JavaScriptCore/b3/B3Procedure.h	2026-05-20 15:17:51.933346419 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3ReduceStrength.cpp	2026-05-20 15:14:23.590857005 +0000
+++ b/Source/JavaScriptCore/b3/B3ReduceStrength.cpp	2026-05-20 15:17:51.957346588 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3ReduceStrength.h"
 
--- a/Source/JavaScriptCore/b3/B3SIMDValue.h	2026-05-20 15:14:23.564856749 +0000
+++ b/Source/JavaScriptCore/b3/B3SIMDValue.h	2026-05-20 15:17:51.978346736 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3StackmapSpecial.cpp	2026-05-20 15:14:23.591857015 +0000
+++ b/Source/JavaScriptCore/b3/B3StackmapSpecial.cpp	2026-05-20 15:17:51.998346877 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3StackmapSpecial.h"
 
--- a/Source/JavaScriptCore/b3/B3StackmapSpecial.h	2026-05-20 15:14:23.565856759 +0000
+++ b/Source/JavaScriptCore/b3/B3StackmapSpecial.h	2026-05-20 15:17:52.018347018 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3SwitchCase.h	2026-05-20 15:14:23.565856759 +0000
+++ b/Source/JavaScriptCore/b3/B3SwitchCase.h	2026-05-20 15:17:52.038347159 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3SwitchValue.h	2026-05-20 15:14:23.566856769 +0000
+++ b/Source/JavaScriptCore/b3/B3SwitchValue.h	2026-05-20 15:17:52.058347300 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3Type.h	2026-02-23 14:40:52.237318000 +0000
+++ b/Source/JavaScriptCore/b3/B3Type.h	2026-05-20 15:17:52.079347448 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT) || ENABLE(WEBASSEMBLY_BBQJIT)
--- a/Source/JavaScriptCore/b3/B3Validate.cpp	2026-05-20 15:14:23.592857025 +0000
+++ b/Source/JavaScriptCore/b3/B3Validate.cpp	2026-05-20 15:17:52.101347603 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3Validate.h"
 #include "B3HeapRange.h"
--- a/Source/JavaScriptCore/b3/B3Value.cpp	2026-05-20 15:14:23.593857034 +0000
+++ b/Source/JavaScriptCore/b3/B3Value.cpp	2026-05-20 15:17:52.120347737 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3Value.h"
 
--- a/Source/JavaScriptCore/b3/B3Value.h	2026-05-20 15:14:23.567856779 +0000
+++ b/Source/JavaScriptCore/b3/B3Value.h	2026-05-20 15:17:52.141347885 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/B3ValueInlines.h	2026-05-20 15:14:23.567856779 +0000
+++ b/Source/JavaScriptCore/b3/B3ValueInlines.h	2026-05-20 15:17:52.161348026 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ValueKey.cpp	2026-05-20 15:14:23.593857034 +0000
+++ b/Source/JavaScriptCore/b3/B3ValueKey.cpp	2026-05-20 15:17:52.182348174 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3ValueKey.h"
 
--- a/Source/JavaScriptCore/b3/B3ValueKey.h	2026-05-20 15:14:23.567856779 +0000
+++ b/Source/JavaScriptCore/b3/B3ValueKey.h	2026-05-20 15:17:52.202348315 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ValueKeyInlines.h	2026-05-20 15:14:23.567856779 +0000
+++ b/Source/JavaScriptCore/b3/B3ValueKeyInlines.h	2026-05-20 15:17:52.222348456 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ValueRep.h	2026-05-20 15:05:39.112687045 +0000
+++ b/Source/JavaScriptCore/b3/B3ValueRep.h	2026-05-20 15:17:52.244348611 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #include <wtf/Platform.h>
--- a/Source/JavaScriptCore/b3/B3WasmBoundsCheckValue.cpp	2026-05-20 15:14:23.594857044 +0000
+++ b/Source/JavaScriptCore/b3/B3WasmBoundsCheckValue.cpp	2026-05-20 15:17:52.266348766 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "B3WasmBoundsCheckValue.h"
 #include "WasmMemory.h"
--- a/Source/JavaScriptCore/b3/B3WasmBoundsCheckValue.h	2026-05-20 15:14:23.568856789 +0000
+++ b/Source/JavaScriptCore/b3/B3WasmBoundsCheckValue.h	2026-05-20 15:17:52.286348907 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include <JavaScriptCore/JSExportMacros.h>
--- a/Source/JavaScriptCore/b3/testb3_1.cpp	2026-05-20 15:14:23.594857044 +0000
+++ b/Source/JavaScriptCore/b3/testb3_1.cpp	2026-05-20 15:17:52.309349069 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "testb3.h"
 
--- a/Source/JavaScriptCore/b3/testb3_2.cpp	2026-05-20 15:14:23.595857054 +0000
+++ b/Source/JavaScriptCore/b3/testb3_2.cpp	2026-05-20 15:17:52.332349231 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "testb3.h"
 
--- a/Source/JavaScriptCore/b3/testb3_3.cpp	2026-05-20 15:14:23.596857064 +0000
+++ b/Source/JavaScriptCore/b3/testb3_3.cpp	2026-05-20 15:17:52.356349400 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "testb3.h"
 
--- a/Source/JavaScriptCore/b3/testb3_4.cpp	2026-05-20 15:14:23.597857074 +0000
+++ b/Source/JavaScriptCore/b3/testb3_4.cpp	2026-05-20 15:17:52.380349569 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "testb3.h"
 
--- a/Source/JavaScriptCore/b3/testb3_5.cpp	2026-05-20 15:14:23.597857074 +0000
+++ b/Source/JavaScriptCore/b3/testb3_5.cpp	2026-05-20 15:17:52.404349738 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "testb3.h"
 
--- a/Source/JavaScriptCore/b3/testb3_6.cpp	2026-05-20 15:14:23.598857084 +0000
+++ b/Source/JavaScriptCore/b3/testb3_6.cpp	2026-05-20 15:17:52.424349879 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "testb3.h"
 
--- a/Source/JavaScriptCore/b3/testb3_7.cpp	2026-05-20 15:14:23.598857084 +0000
+++ b/Source/JavaScriptCore/b3/testb3_7.cpp	2026-05-20 15:17:52.446350034 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "testb3.h"
 #include <wtf/WasmSIMD128.h>
--- a/Source/JavaScriptCore/b3/testb3_8.cpp	2026-05-20 15:14:23.599857093 +0000
+++ b/Source/JavaScriptCore/b3/testb3_8.cpp	2026-05-20 15:17:52.466350175 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstdint>
 #include <wtf/Platform.h>
 #include "testb3.h"
 
--- a/Source/JavaScriptCore/b3/testb3.h	2026-05-20 15:14:23.568856789 +0000
+++ b/Source/JavaScriptCore/b3/testb3.h	2026-05-20 15:17:52.489350337 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include <wtf/Platform.h>
 
 #include "AirCCallingConvention.h"
