--- a/Source/JavaScriptCore/shell/CMakeLists.txt
+++ b/Source/JavaScriptCore/shell/CMakeLists.txt
@@ -25,6 +25,16 @@
 
 WEBKIT_EXECUTABLE_DECLARE(jsc)
 
+if (${JavaScriptCore_LIBRARY_TYPE} STREQUAL "OBJECT")
+    add_library(JavaScriptCoreShellArchive STATIC
+        $<TARGET_OBJECTS:JavaScriptCore>
+        $<TARGET_OBJECTS:LowLevelInterpreterLib>
+    )
+    list(REMOVE_ITEM jsc_FRAMEWORKS JavaScriptCore)
+    list(APPEND jsc_DEPENDENCIES JavaScriptCoreShellArchive)
+    list(PREPEND jsc_LIBRARIES "-Wl,--whole-archive" JavaScriptCoreShellArchive "-Wl,--no-whole-archive")
+endif ()
+
 if (DEVELOPER_MODE)
     set(testapi_SOURCES
         ../API/tests/CompareAndSwapTest.cpp
