--- a/Source/JavaScriptCore/wasm/debugger/tests/BinaryTests.cpp	2026-02-23 14:40:52.522316700 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/BinaryTests.cpp	2026-05-20 16:34:29.135850870 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "TestUtilities.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/tests/ControlFlowTests.cpp	2026-02-23 14:40:52.523414900 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/ControlFlowTests.cpp	2026-05-20 16:34:29.156851079 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "TestUtilities.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/tests/ExecutionHandlerTest.cpp	2026-02-23 14:40:52.523414900 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/ExecutionHandlerTest.cpp	2026-05-20 16:34:29.177851287 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "ExecutionHandlerTest.h"
 
 #include <wtf/DataLog.h>
--- a/Source/JavaScriptCore/wasm/debugger/tests/ExecutionHandlerTest.h	2026-02-23 14:40:52.523414900 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/ExecutionHandlerTest.h	2026-05-20 16:34:29.200851515 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 namespace ExecutionHandlerTest {
--- a/Source/JavaScriptCore/wasm/debugger/tests/ExecutionHandlerTestSupport.cpp	2026-02-23 14:40:52.523414900 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/ExecutionHandlerTestSupport.cpp	2026-05-20 16:34:29.220851714 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "ExecutionHandlerTestSupport.h"
 
 #if ENABLE(WEBASSEMBLY) && ENABLE(REMOTE_INSPECTOR)
--- a/Source/JavaScriptCore/wasm/debugger/tests/ExecutionHandlerTestSupport.h	2026-02-23 14:40:52.523414900 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/ExecutionHandlerTestSupport.h	2026-05-20 16:34:29.241851922 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY) && ENABLE(REMOTE_INSPECTOR)
 
 #include <atomic>
--- a/Source/JavaScriptCore/wasm/debugger/tests/ExtGCTests.cpp	2026-02-23 14:40:52.523414900 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/ExtGCTests.cpp	2026-05-20 16:34:29.261852121 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "TestUtilities.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/tests/GDBPacketParserTest.cpp	2026-02-23 14:40:52.523414900 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/GDBPacketParserTest.cpp	2026-05-20 16:34:29.281852319 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "GDBPacketParserTest.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/tests/MemoryTests.cpp	2026-02-23 14:40:52.523414900 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/MemoryTests.cpp	2026-05-20 16:34:29.303852538 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "TestUtilities.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/tests/SpecialTests.cpp	2026-02-23 14:40:52.523414900 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/SpecialTests.cpp	2026-05-20 16:34:29.324852746 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "TestUtilities.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/tests/TestScripts.cpp	2026-02-23 14:40:52.523414900 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/TestScripts.cpp	2026-05-20 16:34:29.347852974 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "TestScripts.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/tests/TestScripts.h	2026-02-23 14:40:52.523414900 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/TestScripts.h	2026-05-20 16:34:29.368853183 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include <span>
--- a/Source/JavaScriptCore/wasm/debugger/tests/TestUtilities.cpp	2026-02-23 14:40:52.523414900 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/TestUtilities.cpp	2026-05-20 16:34:29.389853391 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "TestUtilities.h"
 
 #include <wtf/DataLog.h>
--- a/Source/JavaScriptCore/wasm/debugger/tests/TestUtilities.h	2026-02-23 14:40:52.524316500 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/TestUtilities.h	2026-05-20 16:34:29.411853609 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "InitializeThreading.h"
--- a/Source/JavaScriptCore/wasm/debugger/tests/UnaryTests.cpp	2026-02-23 14:40:52.524316500 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/tests/UnaryTests.cpp	2026-05-20 16:34:29.431853808 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "TestUtilities.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/testwasmdebugger.cpp	2026-02-23 14:40:52.524316500 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/testwasmdebugger.cpp	2026-05-20 16:34:29.451854006 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #include <wtf/DataLog.h>
 
--- a/Source/JavaScriptCore/wasm/debugger/WasmBreakpointManager.cpp	2026-02-23 14:40:52.520387000 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmBreakpointManager.cpp	2026-05-20 16:34:29.470854195 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmBreakpointManager.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/WasmBreakpointManager.h	2026-02-23 14:40:52.520387000 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmBreakpointManager.h	2026-05-20 16:34:29.489854384 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmDebugServerUtilities.h"
--- a/Source/JavaScriptCore/wasm/debugger/WasmDebugServer.cpp	2026-02-23 14:40:52.520387000 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmDebugServer.cpp	2026-05-20 16:34:29.509854582 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmDebugServer.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/WasmDebugServerUtilities.cpp	2026-02-23 14:40:52.520387000 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmDebugServerUtilities.cpp	2026-05-20 16:34:29.531854800 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmDebugServerUtilities.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/WasmExecutionHandler.cpp	2026-02-23 14:40:52.520387000 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmExecutionHandler.cpp	2026-05-20 16:34:29.554855029 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmExecutionHandler.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/WasmExecutionHandler.h	2026-02-23 14:40:52.520387000 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmExecutionHandler.h	2026-05-20 16:34:29.573855217 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "StopTheWorldCallback.h"
--- a/Source/JavaScriptCore/wasm/debugger/WasmGDBPacketParser.cpp	2026-02-23 14:40:52.521316500 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmGDBPacketParser.cpp	2026-05-20 16:34:29.590855386 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmGDBPacketParser.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/WasmMemoryHandler.cpp	2026-02-23 14:40:52.521316500 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmMemoryHandler.cpp	2026-05-20 16:34:29.611855594 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmMemoryHandler.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/WasmMemoryHandler.h	2026-02-23 14:40:52.521316500 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmMemoryHandler.h	2026-05-20 16:34:29.632855803 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmDebugServerUtilities.h"
--- a/Source/JavaScriptCore/wasm/debugger/WasmModuleDebugInfo.cpp	2026-02-23 14:40:52.521316500 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmModuleDebugInfo.cpp	2026-05-20 16:34:29.651855991 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmModuleDebugInfo.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/WasmModuleManager.cpp	2026-02-23 14:40:52.521316500 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmModuleManager.cpp	2026-05-20 16:34:29.668856160 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmModuleManager.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/WasmModuleManager.h	2026-02-23 14:40:52.521316500 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmModuleManager.h	2026-05-20 16:34:29.688856359 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmInstanceAnchor.h"
--- a/Source/JavaScriptCore/wasm/debugger/WasmQueryHandler.cpp	2026-02-23 14:40:52.522316700 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmQueryHandler.cpp	2026-05-20 16:34:29.709856567 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmQueryHandler.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/debugger/WasmQueryHandler.h	2026-02-23 14:40:52.522316700 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmQueryHandler.h	2026-05-20 16:34:29.728856756 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 #include "WasmVirtualAddress.h"
 
--- a/Source/JavaScriptCore/wasm/debugger/WasmVirtualAddress.cpp	2026-02-23 14:40:52.522316700 +0000
+++ b/Source/JavaScriptCore/wasm/debugger/WasmVirtualAddress.cpp	2026-05-20 16:34:29.745856924 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmVirtualAddress.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/JSToWasm.cpp	2026-02-23 14:40:52.524316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSToWasm.cpp	2026-05-20 16:34:29.763857103 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSToWasm.h"
 #include "WasmCallee.h"
 #include <wtf/Gigacage.h>
--- a/Source/JavaScriptCore/wasm/js/JSToWasm.h	2026-02-23 14:40:52.524316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSToWasm.h	2026-05-20 16:34:29.782857292 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY) && ENABLE(JIT)
 
 #include "InternalFunction.h"
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyArray.cpp	2026-02-23 14:40:52.525316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyArray.cpp	2026-05-20 16:34:29.799857460 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssemblyArray.h"
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyArray.h	2026-02-23 14:40:52.525316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyArray.h	2026-05-20 16:34:29.819857659 +0000
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmFormat.h"
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyArrayInlines.h	2026-02-23 14:40:52.525316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyArrayInlines.h	2026-05-20 16:34:29.841857877 +0000
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "HeapCellInlines.h"
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyCompileError.cpp	2026-01-20 14:08:32.665705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyCompileError.cpp	2026-05-20 16:34:29.861858076 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssemblyCompileError.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyCompileError.h	2026-01-20 14:08:32.665705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyCompileError.h	2026-05-20 16:34:29.883858294 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "ErrorInstance.h"
--- a/Source/JavaScriptCore/wasm/js/JSWebAssembly.cpp	2026-02-23 14:40:52.524316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssembly.cpp	2026-05-20 16:34:29.903858492 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssembly.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyException.cpp	2026-02-23 14:40:52.525316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyException.cpp	2026-05-20 16:34:29.925858711 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssemblyException.h"
 #include "WasmExceptionType.h"
 #include "WasmOps.h"
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyException.h	2026-02-23 14:40:52.525316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyException.h	2026-05-20 16:34:29.946858919 +0000
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSObject.h"
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyGlobal.cpp	2026-02-23 14:40:52.525316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyGlobal.cpp	2026-05-20 16:34:29.965859108 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssemblyGlobal.h"
 #include "ObjectConstructor.h"
 
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyHelpers.h	2026-02-23 14:40:52.526316600 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyHelpers.h	2026-05-20 16:34:29.986859316 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "Error.h"
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyInstance.cpp	2026-02-23 14:40:52.526316600 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyInstance.cpp	2026-05-20 16:34:30.007859525 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssemblyInstance.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyLinkError.cpp	2026-01-20 14:08:32.667705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyLinkError.cpp	2026-05-20 16:34:30.028859733 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssemblyLinkError.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyMemory.cpp	2026-02-23 14:40:52.526316600 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyMemory.cpp	2026-05-20 16:34:30.047859922 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssemblyMemory.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyModule.cpp	2026-02-23 14:40:52.526316600 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyModule.cpp	2026-05-20 16:34:30.067860120 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssemblyModule.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyRuntimeError.cpp	2026-01-20 14:08:32.668705200 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyRuntimeError.cpp	2026-05-20 16:34:30.086860309 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssemblyRuntimeError.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyRuntimeError.h	2026-01-20 14:08:32.668705200 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyRuntimeError.h	2026-05-20 16:34:30.107860517 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "ErrorInstance.h"
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyStruct.cpp	2026-02-23 14:40:52.527316600 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyStruct.cpp	2026-05-20 16:34:30.128860725 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssemblyStruct.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyStruct.h	2026-02-23 14:40:52.527316600 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyStruct.h	2026-05-20 16:34:30.150860944 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSObject.h"
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyTable.cpp	2026-02-23 14:40:52.527316600 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyTable.cpp	2026-05-20 16:34:30.170861142 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssemblyTable.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyTag.cpp	2026-01-20 14:08:32.669705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyTag.cpp	2026-05-20 16:34:30.193861370 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JSWebAssemblyTag.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/JSWebAssemblyTag.h	2026-01-20 14:08:32.670705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/JSWebAssemblyTag.h	2026-05-20 16:34:30.215861589 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSObject.h"
--- a/Source/JavaScriptCore/wasm/js/WasmToJS.cpp	2026-03-03 21:30:17.031498700 +0000
+++ b/Source/JavaScriptCore/wasm/js/WasmToJS.cpp	2026-05-20 16:34:30.236861797 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmToJS.h"
 
 #if ENABLE(WEBASSEMBLY) && ENABLE(JIT)
--- a/Source/JavaScriptCore/wasm/js/WasmToJS.h	2026-01-20 14:08:32.671705200 +0000
+++ b/Source/JavaScriptCore/wasm/js/WasmToJS.h	2026-05-20 16:34:30.254861976 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY) && ENABLE(JIT)
 
 #include "JITCompilation.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyArrayConstructor.cpp	2026-01-20 14:08:32.671705200 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyArrayConstructor.cpp	2026-05-20 16:34:30.273862164 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyArrayConstructor.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyArrayConstructor.h	2026-01-20 14:08:32.671705200 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyArrayConstructor.h	2026-05-20 16:34:30.294862373 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "InternalFunction.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyArrayPrototype.cpp	2026-01-20 14:08:32.671705200 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyArrayPrototype.cpp	2026-05-20 16:34:30.316862591 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyArrayPrototype.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyArrayPrototype.h	2026-01-20 14:08:32.671705200 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyArrayPrototype.h	2026-05-20 16:34:30.335862780 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSObject.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyBuiltin.cpp	2026-02-23 14:40:52.527316600 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyBuiltin.cpp	2026-05-20 16:34:30.356862988 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyBuiltin.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyBuiltinTrampoline.cpp	2026-02-23 14:40:52.527316600 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyBuiltinTrampoline.cpp	2026-05-20 16:34:30.377863197 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyBuiltinTrampoline.h"
 
 #if ENABLE(WEBASSEMBLY) && ENABLE(JIT)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyBuiltinTrampoline.h	2026-02-23 14:40:52.528316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyBuiltinTrampoline.h	2026-05-20 16:34:30.398863405 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY) && ENABLE(JIT)
 
 #include "JIT.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyCompileErrorConstructor.cpp	2026-01-20 14:08:32.673705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyCompileErrorConstructor.cpp	2026-05-20 16:34:30.415863574 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyCompileErrorConstructor.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyCompileErrorConstructor.h	2026-01-20 14:08:32.673705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyCompileErrorConstructor.h	2026-05-20 16:34:30.436863782 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "InternalFunction.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyCompileErrorPrototype.cpp	2026-01-20 14:08:32.673705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyCompileErrorPrototype.cpp	2026-05-20 16:34:30.455863971 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyCompileErrorPrototype.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyCompileErrorPrototype.h	2026-01-20 14:08:32.673705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyCompileErrorPrototype.h	2026-05-20 16:34:30.477864189 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSDestructibleObject.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyCompileOptions.cpp	2026-02-23 14:40:52.528316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyCompileOptions.cpp	2026-05-20 16:34:30.497864387 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyCompileOptions.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyCompileOptions.h	2026-02-23 14:40:52.528316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyCompileOptions.h	2026-05-20 16:34:30.516864576 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "IteratorOperations.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyExceptionConstructor.cpp	2026-02-23 14:40:52.528316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyExceptionConstructor.cpp	2026-05-20 16:34:30.537864784 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyExceptionConstructor.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyExceptionConstructor.h	2026-01-20 14:08:32.673705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyExceptionConstructor.h	2026-05-20 16:34:30.558864993 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "InternalFunction.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyExceptionPrototype.cpp	2026-01-20 14:08:32.674705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyExceptionPrototype.cpp	2026-05-20 16:34:30.578865191 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyExceptionPrototype.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyExceptionPrototype.h	2026-01-20 14:08:32.674705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyExceptionPrototype.h	2026-05-20 16:34:30.602865429 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSDestructibleObject.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyFunctionBase.cpp	2026-02-23 14:40:52.528316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyFunctionBase.cpp	2026-05-20 16:34:30.624865648 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyFunctionBase.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyFunction.cpp	2026-02-23 14:40:52.528316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyFunction.cpp	2026-05-20 16:34:30.645865856 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyFunction.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyGCObjectBase.cpp	2026-02-23 14:40:52.528316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyGCObjectBase.cpp	2026-05-20 16:34:30.666866065 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyGCObjectBase.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyGCObjectBase.h	2026-02-23 14:40:52.528316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyGCObjectBase.h	2026-05-20 16:34:30.687866273 +0000
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSObject.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyGCStructure.cpp	2026-03-25 16:38:32.901259700 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyGCStructure.cpp	2026-05-20 16:34:30.706866462 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyGCStructure.h"
 
 #include "JSCInlines.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyGlobalConstructor.cpp	2026-02-23 14:40:52.529316700 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyGlobalConstructor.cpp	2026-05-20 16:34:30.726866660 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyGlobalConstructor.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyGlobalPrototype.cpp	2026-01-20 14:08:32.676705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyGlobalPrototype.cpp	2026-05-20 16:34:30.749866888 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyGlobalPrototype.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyGlobalPrototype.h	2026-01-20 14:08:32.676705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyGlobalPrototype.h	2026-05-20 16:34:30.771867107 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSObject.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyInstanceConstructor.cpp	2026-02-23 14:40:52.529316700 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyInstanceConstructor.cpp	2026-05-20 16:34:30.792867315 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyInstanceConstructor.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyInstanceConstructor.h	2026-01-20 14:08:32.676705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyInstanceConstructor.h	2026-05-20 16:34:30.813867523 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "InternalFunction.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyInstancePrototype.cpp	2026-01-20 14:08:32.676705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyInstancePrototype.cpp	2026-05-20 16:34:30.835867742 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyInstancePrototype.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyInstancePrototype.h	2026-01-20 14:08:32.676705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyInstancePrototype.h	2026-05-20 16:34:30.857867960 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSDestructibleObject.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyLinkErrorConstructor.cpp	2026-01-20 14:08:32.677705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyLinkErrorConstructor.cpp	2026-05-20 16:34:30.878868169 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyLinkErrorConstructor.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyLinkErrorPrototype.cpp	2026-01-20 14:08:32.677705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyLinkErrorPrototype.cpp	2026-05-20 16:34:30.899868377 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyLinkErrorPrototype.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyMemoryConstructor.cpp	2026-01-20 14:08:32.677705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyMemoryConstructor.cpp	2026-05-20 16:34:30.918868565 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyMemoryConstructor.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyMemoryConstructor.h	2026-01-20 14:08:32.677705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyMemoryConstructor.h	2026-05-20 16:34:30.937868754 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "InternalFunction.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyMemoryPrototype.cpp	2026-02-23 14:40:52.529316700 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyMemoryPrototype.cpp	2026-05-20 16:34:30.957868953 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyMemoryPrototype.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyMemoryPrototype.h	2026-02-23 14:40:52.529316700 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyMemoryPrototype.h	2026-05-20 16:34:30.975869131 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSDestructibleObject.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyModuleConstructor.cpp	2026-02-23 14:40:52.529316700 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyModuleConstructor.cpp	2026-05-20 16:34:30.993869310 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyModuleConstructor.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyModuleConstructor.h	2026-02-23 14:40:52.538316500 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyModuleConstructor.h	2026-05-20 16:34:31.011869488 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "InternalFunction.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyModulePrototype.cpp	2026-01-20 14:08:32.678705200 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyModulePrototype.cpp	2026-05-20 16:34:31.031869687 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyModulePrototype.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyModulePrototype.h	2026-01-20 14:08:32.678705200 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyModulePrototype.h	2026-05-20 16:34:31.050869875 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSDestructibleObject.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyModuleRecord.cpp	2026-03-25 16:38:32.901259700 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyModuleRecord.cpp	2026-05-20 16:34:31.071870084 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyModuleRecord.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyRuntimeErrorConstructor.cpp	2026-01-20 14:08:32.679705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyRuntimeErrorConstructor.cpp	2026-05-20 16:34:31.092870292 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyRuntimeErrorConstructor.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyRuntimeErrorConstructor.h	2026-01-20 14:08:32.679705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyRuntimeErrorConstructor.h	2026-05-20 16:34:31.112870491 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "InternalFunction.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyRuntimeErrorPrototype.cpp	2026-01-20 14:08:32.679705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyRuntimeErrorPrototype.cpp	2026-05-20 16:34:31.135870719 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyRuntimeErrorPrototype.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyRuntimeErrorPrototype.h	2026-01-20 14:08:32.679705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyRuntimeErrorPrototype.h	2026-05-20 16:34:31.157870937 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSDestructibleObject.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyStructConstructor.cpp	2026-01-20 14:08:32.679705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyStructConstructor.cpp	2026-05-20 16:34:31.177871136 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyStructConstructor.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyStructConstructor.h	2026-01-20 14:08:32.679705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyStructConstructor.h	2026-05-20 16:34:31.197871334 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "InternalFunction.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyStructPrototype.cpp	2026-01-20 14:08:32.679705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyStructPrototype.cpp	2026-05-20 16:34:31.218871543 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyStructPrototype.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyStructPrototype.h	2026-01-20 14:08:32.679705400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyStructPrototype.h	2026-05-20 16:34:31.239871751 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSObject.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyTableConstructor.cpp	2026-02-23 14:40:52.539316400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyTableConstructor.cpp	2026-05-20 16:34:31.264871999 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyTableConstructor.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyTableConstructor.h	2026-01-20 14:08:32.680705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyTableConstructor.h	2026-05-20 16:34:31.287872227 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "InternalFunction.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyTablePrototype.cpp	2026-01-20 14:08:32.680705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyTablePrototype.cpp	2026-05-20 16:34:31.309872446 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyTablePrototype.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyTablePrototype.h	2026-01-20 14:08:32.680705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyTablePrototype.h	2026-05-20 16:34:31.328872634 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSDestructibleObject.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyTagConstructor.cpp	2026-01-20 14:08:32.680705300 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyTagConstructor.cpp	2026-05-20 16:34:31.349872843 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyTagConstructor.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyTagConstructor.h	2026-01-20 14:08:32.681705200 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyTagConstructor.h	2026-05-20 16:34:31.370873051 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "InternalFunction.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyTagPrototype.cpp	2026-01-20 14:08:32.681705200 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyTagPrototype.cpp	2026-05-20 16:34:31.390873249 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyTagPrototype.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyTagPrototype.h	2026-01-20 14:08:32.681705200 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyTagPrototype.h	2026-05-20 16:34:31.410873448 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "JSObject.h"
--- a/Source/JavaScriptCore/wasm/js/WebAssemblyWrapperFunction.cpp	2026-02-23 14:40:52.539316400 +0000
+++ b/Source/JavaScriptCore/wasm/js/WebAssemblyWrapperFunction.cpp	2026-05-20 16:34:31.430873646 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WebAssemblyWrapperFunction.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmBBQDisassembler.cpp	2026-02-23 14:40:52.500316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBBQDisassembler.cpp	2026-05-20 16:34:31.451873855 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmBBQDisassembler.h"
 
 #if ENABLE(WEBASSEMBLY_BBQJIT)
--- a/Source/JavaScriptCore/wasm/WasmBBQDisassembler.h	2026-02-23 14:40:52.501316800 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBBQDisassembler.h	2026-05-20 16:34:31.473874073 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY_BBQJIT)
 
 #include "BytecodeIndex.h"
--- a/Source/JavaScriptCore/wasm/WasmBBQJIT32_64.cpp	2026-03-03 21:30:17.020498500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBBQJIT32_64.cpp	2026-05-20 16:34:31.497874311 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmBBQJIT32_64.h"
 
 #include "WasmBBQJIT.h"
--- a/Source/JavaScriptCore/wasm/WasmBBQJIT32_64.h	2026-02-23 14:40:52.502316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBBQJIT32_64.h	2026-05-20 16:34:31.518874520 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY_BBQJIT)
 #if USE(JSVALUE32_64)
 
--- a/Source/JavaScriptCore/wasm/WasmBBQJIT64.cpp	2026-03-03 21:30:17.021498400 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBBQJIT64.cpp	2026-05-20 16:34:31.540874738 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmBBQJIT64.h"
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/wasm/WasmBBQJIT64.h	2026-02-23 14:40:52.502316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBBQJIT64.h	2026-05-20 16:34:31.562874956 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY_BBQJIT)
 #if USE(JSVALUE64)
 
--- a/Source/JavaScriptCore/wasm/WasmBBQJIT.cpp	2026-03-03 21:30:17.018498700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBBQJIT.cpp	2026-05-20 16:34:31.585875185 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmBBQJIT.h"
 #include "WasmBBQJIT32_64.h"
 #include "WasmBBQJIT64.h"
--- a/Source/JavaScriptCore/wasm/WasmBBQJIT.h	2026-03-03 21:30:17.019498600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBBQJIT.h	2026-05-20 16:34:31.609875423 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY_BBQJIT)
 
 #include "PCToCodeOriginMap.h"
--- a/Source/JavaScriptCore/wasm/WasmBBQPlan.cpp	2026-02-23 14:40:52.503316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBBQPlan.cpp	2026-05-20 16:34:31.633875661 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmBBQPlan.h"
 
 #if ENABLE(WEBASSEMBLY_BBQJIT)
--- a/Source/JavaScriptCore/wasm/WasmBBQPlan.h	2026-02-23 14:40:52.503316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBBQPlan.h	2026-05-20 16:34:31.658875909 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY_BBQJIT)
 
 #include "CompilationResult.h"
--- a/Source/JavaScriptCore/wasm/WasmBinding.cpp	2026-02-23 14:40:52.503316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBinding.cpp	2026-05-20 16:34:31.679876117 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmBinding.h"
 
 #include "CallFrame.h"
--- a/Source/JavaScriptCore/wasm/WasmBinding.h	2026-01-20 14:08:32.613705400 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBinding.h	2026-05-20 16:34:31.701876336 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY) && ENABLE(JIT)
 
 #include "JITCompilation.h"
--- a/Source/JavaScriptCore/wasm/WasmBranchHintsSectionParser.cpp	2026-02-23 14:40:52.503316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBranchHintsSectionParser.cpp	2026-05-20 16:34:31.721876534 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(WEBASSEMBLY)
 
--- a/Source/JavaScriptCore/wasm/WasmBranchHintsSectionParser.h	2026-01-20 14:08:32.615705500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmBranchHintsSectionParser.h	2026-05-20 16:34:31.741876733 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmParser.h"
--- a/Source/JavaScriptCore/wasm/WasmCallee.cpp	2026-02-23 14:40:52.504316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmCallee.cpp	2026-05-20 16:34:31.762876941 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmCallee.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmCalleeGroup.cpp	2026-02-23 14:40:52.504316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmCalleeGroup.cpp	2026-05-20 16:34:31.781877130 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmCalleeGroup.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmCallingConvention.cpp	2026-02-23 14:40:52.504316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmCallingConvention.cpp	2026-05-20 16:34:31.801877328 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmCallingConvention.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmCallProfile.cpp	2026-02-23 14:40:52.503316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmCallProfile.cpp	2026-05-20 16:34:31.821877526 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmCallProfile.h"
 
 #include "WasmMergedProfile.h"
--- a/Source/JavaScriptCore/wasm/WasmConstExprGenerator.cpp	2026-03-25 16:38:32.896259500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmConstExprGenerator.cpp	2026-05-20 16:34:31.840877715 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmConstExprGenerator.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmConstExprGenerator.h	2026-03-25 16:38:32.897259700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmConstExprGenerator.h	2026-05-20 16:34:31.861877923 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmModuleInformation.h"
--- a/Source/JavaScriptCore/wasm/WasmContext.cpp	2026-02-23 14:40:52.505316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmContext.cpp	2026-05-20 16:34:31.881878122 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmContext.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmEntryPlan.cpp	2026-02-23 14:40:52.505316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmEntryPlan.cpp	2026-05-20 16:34:31.905878360 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmEntryPlan.h"
 
 #include "LLIntData.h"
--- a/Source/JavaScriptCore/wasm/WasmEntryPlan.h	2026-02-23 14:40:52.505316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmEntryPlan.h	2026-05-20 16:34:31.927878578 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmModuleDebugInfo.h"
--- a/Source/JavaScriptCore/wasm/WasmFaultSignalHandler.cpp	2026-02-23 14:40:52.506316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmFaultSignalHandler.cpp	2026-05-20 16:34:31.946878767 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmFaultSignalHandler.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmFormat.cpp	2026-02-23 14:40:52.506316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmFormat.cpp	2026-05-20 16:34:31.965878955 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #include "WasmFormat.h"
 
--- a/Source/JavaScriptCore/wasm/WasmFunctionIPIntMetadataGenerator.cpp	2026-02-23 14:40:52.507316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmFunctionIPIntMetadataGenerator.cpp	2026-05-20 16:34:31.985879154 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmFunctionIPIntMetadataGenerator.h"
 
 #include <numeric>
--- a/Source/JavaScriptCore/wasm/WasmGlobal.cpp	2026-02-23 14:40:52.508316800 +0000
+++ b/Source/JavaScriptCore/wasm/WasmGlobal.cpp	2026-05-20 16:34:32.003879333 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmGlobal.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmHandlerInfo.cpp	2026-01-20 14:08:32.626705400 +0000
+++ b/Source/JavaScriptCore/wasm/WasmHandlerInfo.cpp	2026-05-20 16:34:32.025879551 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmHandlerInfo.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmIndexOrName.cpp	2026-02-23 14:40:52.509316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmIndexOrName.cpp	2026-05-20 16:34:32.044879739 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmIndexOrName.h"
 
 #include <wtf/PrintStream.h>
--- a/Source/JavaScriptCore/wasm/WasmIndexOrName.h	2026-02-23 14:40:52.509316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmIndexOrName.h	2026-05-20 16:34:32.066879958 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/WasmName.h>
 #include <JavaScriptCore/WasmNameSection.h>
 #include <wtf/RefPtr.h>
--- a/Source/JavaScriptCore/wasm/WasmInliningDecision.cpp	2026-02-23 14:40:52.509316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmInliningDecision.cpp	2026-05-20 16:34:32.086880156 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmInliningDecision.h"
 
 #include "WasmMergedProfile.h"
--- a/Source/JavaScriptCore/wasm/WasmInliningDecision.h	2026-02-23 14:40:52.510316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmInliningDecision.h	2026-05-20 16:34:32.107880365 +0000
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include <JavaScriptCore/WasmBaselineData.h>
--- a/Source/JavaScriptCore/wasm/WasmInstanceAnchor.cpp	2026-02-23 14:40:52.510316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmInstanceAnchor.cpp	2026-05-20 16:34:32.127880563 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmInstanceAnchor.h"
 
 #include <wtf/NeverDestroyed.h>
--- a/Source/JavaScriptCore/wasm/WasmIPIntGenerator.cpp	2026-03-03 21:30:17.025498600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmIPIntGenerator.cpp	2026-05-20 16:34:32.148880771 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmIPIntGenerator.h"
 #include <cstdint>
 
--- a/Source/JavaScriptCore/wasm/WasmIPIntPlan.cpp	2026-02-23 14:40:52.508316800 +0000
+++ b/Source/JavaScriptCore/wasm/WasmIPIntPlan.cpp	2026-05-20 16:34:32.170880990 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmIPIntPlan.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmIPIntPlan.h	2026-02-23 14:40:52.508316800 +0000
+++ b/Source/JavaScriptCore/wasm/WasmIPIntPlan.h	2026-05-20 16:34:32.190881188 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmCallee.h"
--- a/Source/JavaScriptCore/wasm/WasmIPIntSlowPaths.cpp	2026-02-23 14:40:52.509316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmIPIntSlowPaths.cpp	2026-05-20 16:34:32.208881367 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmIPIntSlowPaths.h"
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/wasm/WasmIPIntSlowPaths.h	2026-02-23 14:40:52.509316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmIPIntSlowPaths.h	2026-05-20 16:34:32.231881595 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "CommonSlowPaths.h"
--- a/Source/JavaScriptCore/wasm/WasmIRGeneratorHelpers.h	2026-02-23 14:40:52.509316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmIRGeneratorHelpers.h	2026-05-20 16:34:32.252881803 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY_OMGJIT) || ENABLE(WEBASSEMBLY_BBQJIT)
 
 #include "AirCode.h"
--- a/Source/JavaScriptCore/wasm/WasmMachineThreads.cpp	2026-01-20 14:08:32.631705500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmMachineThreads.cpp	2026-05-20 16:34:32.274882022 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmMachineThreads.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmMachineThreads.h	2026-01-20 14:08:32.632705500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmMachineThreads.h	2026-05-20 16:34:32.293882210 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmCalleeGroup.h"
--- a/Source/JavaScriptCore/wasm/WasmMemoryInformation.cpp	2026-02-23 14:40:52.510316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmMemoryInformation.cpp	2026-05-20 16:34:32.313882409 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmMemoryInformation.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmMergedProfile.cpp	2026-02-23 14:40:52.511316500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmMergedProfile.cpp	2026-05-20 16:34:32.333882607 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmMergedProfile.h"
 
 #include "WasmModule.h"
--- a/Source/JavaScriptCore/wasm/WasmMergedProfile.h	2026-02-23 14:40:52.511316500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmMergedProfile.h	2026-05-20 16:34:32.355882825 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include <JavaScriptCore/WasmBaselineData.h>
--- a/Source/JavaScriptCore/wasm/WasmModule.cpp	2026-02-23 14:40:52.511316500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmModule.cpp	2026-05-20 16:34:32.374883014 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmModule.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmModuleInformation.cpp	2026-02-23 14:40:52.511316500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmModuleInformation.cpp	2026-05-20 16:34:32.394883212 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmModuleInformation.h"
 #include "WebAssemblyBuiltin.h"
 #include "WebAssemblyCompileOptions.h"
--- a/Source/JavaScriptCore/wasm/WasmNameSectionParser.cpp	2026-02-23 14:40:52.511316500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmNameSectionParser.cpp	2026-05-20 16:34:32.415883421 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmNameSectionParser.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmNameSectionParser.h	2026-02-23 14:40:52.512316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmNameSectionParser.h	2026-05-20 16:34:32.436883629 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmFormat.h"
--- a/Source/JavaScriptCore/wasm/WasmOMGIRGenerator32_64.cpp	2026-03-03 21:30:17.029498600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOMGIRGenerator32_64.cpp	2026-05-20 16:34:32.461883877 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmOMGIRGenerator.h"
 
 #if ENABLE(WEBASSEMBLY_OMGJIT)
--- a/Source/JavaScriptCore/wasm/WasmOMGIRGenerator.cpp	2026-03-03 21:30:17.027498500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOMGIRGenerator.cpp	2026-05-20 16:34:32.487884135 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmOMGIRGenerator.h"
 
 #if ENABLE(WEBASSEMBLY_OMGJIT)
--- a/Source/JavaScriptCore/wasm/WasmOMGIRGenerator.h	2026-02-23 14:40:52.513316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOMGIRGenerator.h	2026-05-20 16:34:32.512884383 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY_OMGJIT)
 
 #include "B3Common.h"
--- a/Source/JavaScriptCore/wasm/WasmOMGPlan.cpp	2026-02-23 14:40:52.513316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOMGPlan.cpp	2026-05-20 16:34:32.532884582 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmOMGPlan.h"
 #include "JSToWasm.h"
 
--- a/Source/JavaScriptCore/wasm/WasmOMGPlan.h	2026-02-23 14:40:52.514316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOMGPlan.h	2026-05-20 16:34:32.553884790 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY_OMGJIT)
 
 #include "WasmContext.h"
--- a/Source/JavaScriptCore/wasm/WasmOpcodeCounter.cpp	2026-02-23 14:40:52.514316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOpcodeCounter.cpp	2026-05-20 16:34:32.572884979 +0000
@@ -23,6 +23,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmOpcodeCounter.h"
 
 #if ENABLE(WEBASSEMBLY) && ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/wasm/WasmOpcodeCounter.h	2026-01-20 14:08:32.642705400 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOpcodeCounter.h	2026-05-20 16:34:32.592885177 +0000
@@ -24,6 +24,8 @@
  */
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY) && ENABLE(B3_JIT)
 
 #include "WasmTypeDefinition.h"
--- a/Source/JavaScriptCore/wasm/WasmOpcodeOrigin.cpp	2026-02-23 14:40:52.514316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOpcodeOrigin.cpp	2026-05-20 16:34:32.612885376 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmOpcodeOrigin.h"
 
 #include "B3Origin.h"
--- a/Source/JavaScriptCore/wasm/WasmOperations.cpp	2026-03-16 09:30:13.580824000 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOperations.cpp	2026-05-20 16:34:32.632885574 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmOperations.h"
 #include "DeferGC.h"
 #include "ObjectAllocationProfile.h"
--- a/Source/JavaScriptCore/wasm/WasmOperations.h	2026-02-23 14:40:52.515316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOperations.h	2026-05-20 16:34:32.654885792 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "IndexingType.h"
--- a/Source/JavaScriptCore/wasm/WasmOperationsInlines.h	2026-02-23 14:40:52.515316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOperationsInlines.h	2026-05-20 16:34:32.676886011 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "WasmOperations.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmOSREntryPlan.cpp	2026-02-23 14:40:52.514316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOSREntryPlan.cpp	2026-05-20 16:34:32.697886219 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmOSREntryPlan.h"
 
 #if ENABLE(WEBASSEMBLY_OMGJIT)
--- a/Source/JavaScriptCore/wasm/WasmOSREntryPlan.h	2026-02-23 14:40:52.514316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmOSREntryPlan.h	2026-05-20 16:34:32.716886408 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY_OMGJIT)
 
 #include "WasmCallee.h"
--- a/Source/JavaScriptCore/wasm/WasmParser.h	2026-02-23 14:40:52.515316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmParser.h	2026-05-20 16:34:32.736886606 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "B3Procedure.h"
--- a/Source/JavaScriptCore/wasm/WasmPlan.cpp	2026-02-23 14:40:52.515316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmPlan.cpp	2026-05-20 16:34:32.759886834 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmPlan.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmPlan.h	2026-02-23 14:40:52.516316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmPlan.h	2026-05-20 16:34:32.777887013 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "CompilationResult.h"
--- a/Source/JavaScriptCore/wasm/WasmSectionParser.cpp	2026-03-25 16:38:32.899259600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmSectionParser.cpp	2026-05-20 16:34:32.799887231 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmSectionParser.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmSectionParser.h	2026-02-23 14:40:52.516316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmSectionParser.h	2026-05-20 16:34:32.818887420 +0000
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmFormat.h"
--- a/Source/JavaScriptCore/wasm/WasmSourceMappingURLSectionParser.cpp	2026-02-23 14:40:52.516316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmSourceMappingURLSectionParser.cpp	2026-05-20 16:34:32.837887608 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(WEBASSEMBLY)
 
--- a/Source/JavaScriptCore/wasm/WasmSourceMappingURLSectionParser.h	2026-01-20 14:08:32.647705300 +0000
+++ b/Source/JavaScriptCore/wasm/WasmSourceMappingURLSectionParser.h	2026-05-20 16:34:32.858887817 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmParser.h"
--- a/Source/JavaScriptCore/wasm/WasmStreamingCompiler.cpp	2026-02-23 14:40:52.516316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmStreamingCompiler.cpp	2026-05-20 16:34:32.877888005 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmStreamingCompiler.h"
 
 #include "DeferredWorkTimerInlines.h"
--- a/Source/JavaScriptCore/wasm/WasmStreamingParser.cpp	2026-02-23 14:40:52.517316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmStreamingParser.cpp	2026-05-20 16:34:32.897888204 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmStreamingParser.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmStreamingPlan.cpp	2026-02-23 14:40:52.517316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmStreamingPlan.cpp	2026-05-20 16:34:32.918888412 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmStreamingPlan.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmTable.cpp	2026-02-23 14:40:52.517316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmTable.cpp	2026-05-20 16:34:32.937888600 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmTable.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmTag.cpp	2026-02-23 14:40:52.517316600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmTag.cpp	2026-05-20 16:34:32.958888809 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmTag.h"
 
 #include <wtf/NeverDestroyed.h>
--- a/Source/JavaScriptCore/wasm/WasmTag.h	2026-03-25 16:38:32.899259600 +0000
+++ b/Source/JavaScriptCore/wasm/WasmTag.h	2026-05-20 16:34:32.980889027 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(WEBASSEMBLY)
 
 #include "WasmTypeDefinition.h"
--- a/Source/JavaScriptCore/wasm/WasmThunks.cpp	2026-02-23 14:40:52.518316500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmThunks.cpp	2026-05-20 16:34:32.999889216 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmThunks.h"
 
 #if ENABLE(WEBASSEMBLY) && ENABLE(JIT)
--- a/Source/JavaScriptCore/wasm/WasmTierUpCount.cpp	2026-02-23 14:40:52.518316500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmTierUpCount.cpp	2026-05-20 16:34:33.020889424 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmTierUpCount.h"
 
 #if ENABLE(WEBASSEMBLY_OMGJIT) || ENABLE(WEBASSEMBLY_BBQJIT)
--- a/Source/JavaScriptCore/wasm/WasmTypeDefinition.cpp	2026-03-25 16:38:32.900259500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmTypeDefinition.cpp	2026-05-20 16:34:33.043889652 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmTypeDefinition.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmTZoneImpls.cpp	2026-01-20 14:08:32.648705500 +0000
+++ b/Source/JavaScriptCore/wasm/WasmTZoneImpls.cpp	2026-05-20 16:34:33.064889861 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmOSREntryData.h"
 #include <wtf/TZoneMallocInlines.h>
 
--- a/Source/JavaScriptCore/wasm/WasmValueLocation.cpp	2026-01-20 14:08:32.652705400 +0000
+++ b/Source/JavaScriptCore/wasm/WasmValueLocation.cpp	2026-05-20 16:34:33.084890059 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmValueLocation.h"
 
 #if ENABLE(WEBASSEMBLY)
--- a/Source/JavaScriptCore/wasm/WasmWorklist.cpp	2026-02-23 14:40:52.519316700 +0000
+++ b/Source/JavaScriptCore/wasm/WasmWorklist.cpp	2026-05-20 16:34:33.104890258 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "WasmWorklist.h"
 
 #if ENABLE(WEBASSEMBLY)
