--- a/Source/JavaScriptCore/jit/AssemblyHelpers.cpp	2026-02-23 14:40:52.354317400 +0000
+++ b/Source/JavaScriptCore/jit/AssemblyHelpers.cpp	2026-05-20 16:09:04.641959306 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AssemblyHelpers.h"
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/jit/BaselineJITCode.cpp	2026-02-23 14:40:52.355317600 +0000
+++ b/Source/JavaScriptCore/jit/BaselineJITCode.cpp	2026-05-20 16:09:04.663959512 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "BaselineJITCode.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/BaselineJITCode.h	2026-02-23 14:40:52.355317600 +0000
+++ b/Source/JavaScriptCore/jit/BaselineJITCode.h	2026-05-20 16:09:04.683959699 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CallLinkInfo.h"
 #include "JITCode.h"
 #include "JITCodeMap.h"
--- a/Source/JavaScriptCore/jit/BaselineJITPlan.cpp	2026-02-23 14:40:52.355317600 +0000
+++ b/Source/JavaScriptCore/jit/BaselineJITPlan.cpp	2026-05-20 16:09:04.705959905 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "BaselineJITPlan.h"
 #include "LOLJIT.h"
 
--- a/Source/JavaScriptCore/jit/BaselineJITPlan.h	2026-01-20 14:08:32.138706700 +0000
+++ b/Source/JavaScriptCore/jit/BaselineJITPlan.h	2026-05-20 16:09:04.726960102 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "JITPlan.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/BinarySwitch.cpp	2026-01-20 14:08:32.139706600 +0000
+++ b/Source/JavaScriptCore/jit/BinarySwitch.cpp	2026-05-20 16:09:04.747960298 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "BinarySwitch.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/CachedRecovery.cpp	2026-01-20 14:08:32.140706500 +0000
+++ b/Source/JavaScriptCore/jit/CachedRecovery.cpp	2026-05-20 16:09:04.768960495 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "CachedRecovery.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/CachedRecovery.h	2026-01-20 14:08:32.140706500 +0000
+++ b/Source/JavaScriptCore/jit/CachedRecovery.h	2026-05-20 16:09:04.789960691 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "ValueRecovery.h"
--- a/Source/JavaScriptCore/jit/CallFrameShuffleData.cpp	2026-01-20 14:08:32.141706500 +0000
+++ b/Source/JavaScriptCore/jit/CallFrameShuffleData.cpp	2026-05-20 16:09:04.810960888 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "CallFrameShuffleData.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/CallFrameShuffler32_64.cpp	2026-02-23 14:40:52.356317500 +0000
+++ b/Source/JavaScriptCore/jit/CallFrameShuffler32_64.cpp	2026-05-20 16:09:04.840961169 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "CallFrameShuffler.h"
 
 #if ENABLE(JIT) && USE(JSVALUE32_64)
--- a/Source/JavaScriptCore/jit/CallFrameShuffler64.cpp	2026-01-20 14:08:32.142706600 +0000
+++ b/Source/JavaScriptCore/jit/CallFrameShuffler64.cpp	2026-05-20 16:09:04.862961375 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "CallFrameShuffler.h"
 
 #if ENABLE(JIT) && USE(JSVALUE64)
--- a/Source/JavaScriptCore/jit/CallFrameShuffler.cpp	2026-02-23 14:40:52.356317500 +0000
+++ b/Source/JavaScriptCore/jit/CallFrameShuffler.cpp	2026-05-20 16:09:04.881961553 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "CallFrameShuffler.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/CallFrameShuffler.h	2026-01-20 14:08:32.141706500 +0000
+++ b/Source/JavaScriptCore/jit/CallFrameShuffler.h	2026-05-20 16:09:04.900961731 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "CachedRecovery.h"
--- a/Source/JavaScriptCore/jit/CCallHelpers.cpp	2026-01-20 14:08:32.139706600 +0000
+++ b/Source/JavaScriptCore/jit/CCallHelpers.cpp	2026-05-20 16:09:04.920961918 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "CCallHelpers.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/ExecutableAllocator.cpp	2026-02-23 14:40:52.356317500 +0000
+++ b/Source/JavaScriptCore/jit/ExecutableAllocator.cpp	2026-05-20 16:09:04.940962105 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "ExecutableAllocator.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/ExecutableAllocator.h	2026-02-23 14:40:52.357317400 +0000
+++ b/Source/JavaScriptCore/jit/ExecutableAllocator.h	2026-05-20 16:09:04.967962358 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/AssemblerCommon.h>
 #include <JavaScriptCore/ExecutableMemoryHandle.h>
 #include <JavaScriptCore/FastJITPermissions.h>
--- a/Source/JavaScriptCore/jit/ExecutableMemoryHandle.cpp	2026-01-20 14:08:32.145706700 +0000
+++ b/Source/JavaScriptCore/jit/ExecutableMemoryHandle.cpp	2026-05-20 16:09:04.988962555 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "ExecutableMemoryHandle.h"
 
 namespace JSC {
--- a/Source/JavaScriptCore/jit/ExecutableMemoryHandle.h	2026-02-23 14:40:52.357317400 +0000
+++ b/Source/JavaScriptCore/jit/ExecutableMemoryHandle.h	2026-05-20 16:09:05.009962751 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 #include <wtf/CodePtr.h>
 #include <wtf/RawPointer.h>
--- a/Source/JavaScriptCore/jit/GCAwareJITStubRoutine.cpp	2026-02-23 14:40:52.357317400 +0000
+++ b/Source/JavaScriptCore/jit/GCAwareJITStubRoutine.cpp	2026-05-20 16:09:05.029962939 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "GCAwareJITStubRoutine.h"
 
 #include "AccessCase.h"
--- a/Source/JavaScriptCore/jit/GCAwareJITStubRoutine.h	2026-02-23 14:40:52.357317400 +0000
+++ b/Source/JavaScriptCore/jit/GCAwareJITStubRoutine.h	2026-05-20 16:09:05.052963154 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/DFGCodeOriginPool.h>
 #include <JavaScriptCore/JITStubRoutine.h>
 #include <JavaScriptCore/JSObject.h>
--- a/Source/JavaScriptCore/jit/GdbJIT.cpp	2026-02-23 14:40:52.358317400 +0000
+++ b/Source/JavaScriptCore/jit/GdbJIT.cpp	2026-05-20 16:09:05.075963369 +0000
@@ -24,6 +24,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 #include "config.h"
+#include <wtf/Platform.h>
 #include "GdbJIT.h"
 
 #include <wtf/TZoneMallocInlines.h>
--- a/Source/JavaScriptCore/jit/GPRInfo.cpp	2026-01-20 14:08:32.146706600 +0000
+++ b/Source/JavaScriptCore/jit/GPRInfo.cpp	2026-05-20 16:09:05.098963585 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "GPRInfo.h"
 
 #if ENABLE(ASSEMBLER)
--- a/Source/JavaScriptCore/jit/JITAddGenerator.cpp	2026-02-23 14:40:52.359317500 +0000
+++ b/Source/JavaScriptCore/jit/JITAddGenerator.cpp	2026-05-20 16:09:05.119963781 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITAddGenerator.h"
 
 #include "ArithProfile.h"
--- a/Source/JavaScriptCore/jit/JITAddGenerator.h	2026-01-20 14:08:32.150706500 +0000
+++ b/Source/JavaScriptCore/jit/JITAddGenerator.h	2026-05-20 16:09:05.142963997 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/jit/JITArithmetic.cpp	2026-02-23 14:40:52.359317500 +0000
+++ b/Source/JavaScriptCore/jit/JITArithmetic.cpp	2026-05-20 16:09:05.164964203 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(JIT)
 #include "JIT.h"
--- a/Source/JavaScriptCore/jit/JITBitAndGenerator.cpp	2026-01-20 14:08:32.152706600 +0000
+++ b/Source/JavaScriptCore/jit/JITBitAndGenerator.cpp	2026-05-20 16:09:05.183964381 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITBitAndGenerator.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITBitAndGenerator.h	2026-01-20 14:08:32.152706600 +0000
+++ b/Source/JavaScriptCore/jit/JITBitAndGenerator.h	2026-05-20 16:09:05.205964587 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "JITBitBinaryOpGenerator.h"
--- a/Source/JavaScriptCore/jit/JITBitBinaryOpGenerator.h	2026-01-20 14:08:32.152706600 +0000
+++ b/Source/JavaScriptCore/jit/JITBitBinaryOpGenerator.h	2026-05-20 16:09:05.226964783 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/jit/JITBitOrGenerator.cpp	2026-01-20 14:08:32.152706600 +0000
+++ b/Source/JavaScriptCore/jit/JITBitOrGenerator.cpp	2026-05-20 16:09:05.248964989 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITBitOrGenerator.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITBitOrGenerator.h	2026-01-20 14:08:32.153706600 +0000
+++ b/Source/JavaScriptCore/jit/JITBitOrGenerator.h	2026-05-20 16:09:05.267965167 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "JITBitBinaryOpGenerator.h"
--- a/Source/JavaScriptCore/jit/JITBitXorGenerator.cpp	2026-01-20 14:08:32.153706600 +0000
+++ b/Source/JavaScriptCore/jit/JITBitXorGenerator.cpp	2026-05-20 16:09:05.287965355 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITBitXorGenerator.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITBitXorGenerator.h	2026-01-20 14:08:32.153706600 +0000
+++ b/Source/JavaScriptCore/jit/JITBitXorGenerator.h	2026-05-20 16:09:05.308965551 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "JITBitBinaryOpGenerator.h"
--- a/Source/JavaScriptCore/jit/JITCall.cpp	2026-02-23 14:40:52.359317500 +0000
+++ b/Source/JavaScriptCore/jit/JITCall.cpp	2026-05-20 16:09:05.328965738 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(JIT)
 
--- a/Source/JavaScriptCore/jit/JITCode.cpp	2026-01-20 14:08:32.154706500 +0000
+++ b/Source/JavaScriptCore/jit/JITCode.cpp	2026-05-20 16:09:05.349965935 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITCode.h"
 
 #include "DFGJITCode.h"
--- a/Source/JavaScriptCore/jit/JITCode.h	2026-02-23 14:40:52.360317500 +0000
+++ b/Source/JavaScriptCore/jit/JITCode.h	2026-05-20 16:09:05.370966132 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/ArityCheckMode.h>
 #include <JavaScriptCore/CallFrame.h>
 #include <JavaScriptCore/CodeOrigin.h>
--- a/Source/JavaScriptCore/jit/JITCodeMap.cpp	2026-01-20 14:08:32.154706500 +0000
+++ b/Source/JavaScriptCore/jit/JITCodeMap.cpp	2026-05-20 16:09:05.391966328 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(JIT)
 #include "JITCodeMap.h"
--- a/Source/JavaScriptCore/jit/JITCompilation.cpp	2026-02-23 14:40:52.360317500 +0000
+++ b/Source/JavaScriptCore/jit/JITCompilation.cpp	2026-05-20 16:09:05.412966525 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITCompilation.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITCompilationKey.cpp	2026-01-20 14:08:32.156706600 +0000
+++ b/Source/JavaScriptCore/jit/JITCompilationKey.cpp	2026-05-20 16:09:05.434966731 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITCompilationKey.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITCompilationMode.cpp	2026-01-20 14:08:32.158706400 +0000
+++ b/Source/JavaScriptCore/jit/JITCompilationMode.cpp	2026-05-20 16:09:05.457966946 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITCompilationMode.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JIT.cpp	2026-02-23 14:40:52.358317400 +0000
+++ b/Source/JavaScriptCore/jit/JIT.cpp	2026-05-20 16:09:05.477967134 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(JIT)
 
--- a/Source/JavaScriptCore/jit/JITDisassembler.cpp	2026-02-23 14:40:52.360317500 +0000
+++ b/Source/JavaScriptCore/jit/JITDisassembler.cpp	2026-05-20 16:09:05.499967340 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITDisassembler.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITDivGenerator.cpp	2026-02-23 14:40:52.361317400 +0000
+++ b/Source/JavaScriptCore/jit/JITDivGenerator.cpp	2026-05-20 16:09:05.520967536 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITDivGenerator.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITDivGenerator.h	2026-01-20 14:08:32.159706600 +0000
+++ b/Source/JavaScriptCore/jit/JITDivGenerator.h	2026-05-20 16:09:05.540967723 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/jit/JITExceptions.cpp	2026-02-23 14:40:52.361317400 +0000
+++ b/Source/JavaScriptCore/jit/JITExceptions.cpp	2026-05-20 16:09:05.560967911 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITExceptions.h"
 
 #include "CallFrame.h"
--- a/Source/JavaScriptCore/jit/JIT.h	2026-02-23 14:40:52.359317500 +0000
+++ b/Source/JavaScriptCore/jit/JIT.h	2026-05-20 16:09:05.584968135 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #define ASSERT_JIT_OFFSET(actual, expected) ASSERT_WITH_MESSAGE(actual == expected, "JIT Offset \"%s\" should be %d, not %d.\n", #expected, static_cast<int>(expected), static_cast<int>(actual));
--- a/Source/JavaScriptCore/jit/JITInlineCacheGenerator.cpp	2026-01-20 14:08:32.160706500 +0000
+++ b/Source/JavaScriptCore/jit/JITInlineCacheGenerator.cpp	2026-05-20 16:09:05.604968323 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITInlineCacheGenerator.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITInlines.h	2026-02-23 14:40:52.362317600 +0000
+++ b/Source/JavaScriptCore/jit/JITInlines.h	2026-05-20 16:09:05.624968510 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 #include "BytecodeOperandsForCheckpoint.h"
 #include "CommonSlowPathsInlines.h"
--- a/Source/JavaScriptCore/jit/JITLeftShiftGenerator.cpp	2026-01-20 14:08:32.161706400 +0000
+++ b/Source/JavaScriptCore/jit/JITLeftShiftGenerator.cpp	2026-05-20 16:09:05.644968697 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITLeftShiftGenerator.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITLeftShiftGenerator.h	2026-01-20 14:08:32.161706400 +0000
+++ b/Source/JavaScriptCore/jit/JITLeftShiftGenerator.h	2026-05-20 16:09:05.665968894 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "JITBitBinaryOpGenerator.h"
--- a/Source/JavaScriptCore/jit/JITMathIC.h	2026-01-20 14:08:32.162706600 +0000
+++ b/Source/JavaScriptCore/jit/JITMathIC.h	2026-05-20 16:09:05.687969100 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "ArithProfile.h"
--- a/Source/JavaScriptCore/jit/JITMulGenerator.cpp	2026-02-23 14:40:52.362317600 +0000
+++ b/Source/JavaScriptCore/jit/JITMulGenerator.cpp	2026-05-20 16:09:05.707969287 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITMulGenerator.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITMulGenerator.h	2026-01-20 14:08:32.163706500 +0000
+++ b/Source/JavaScriptCore/jit/JITMulGenerator.h	2026-05-20 16:09:05.727969474 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/jit/JITNegGenerator.cpp	2026-01-20 14:08:32.163706500 +0000
+++ b/Source/JavaScriptCore/jit/JITNegGenerator.cpp	2026-05-20 16:09:05.747969662 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITNegGenerator.h"
 
 #include "ArithProfile.h"
--- a/Source/JavaScriptCore/jit/JITNegGenerator.h	2026-01-20 14:08:32.163706500 +0000
+++ b/Source/JavaScriptCore/jit/JITNegGenerator.h	2026-05-20 16:09:05.770969877 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/jit/JITOpaqueByproduct.h	2026-01-20 14:08:32.163706500 +0000
+++ b/Source/JavaScriptCore/jit/JITOpaqueByproduct.h	2026-05-20 16:09:05.789970055 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include <wtf/Noncopyable.h>
--- a/Source/JavaScriptCore/jit/JITOpaqueByproducts.cpp	2026-02-23 14:40:52.362317600 +0000
+++ b/Source/JavaScriptCore/jit/JITOpaqueByproducts.cpp	2026-05-20 16:09:05.809970242 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITOpaqueByproducts.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITOpaqueByproducts.h	2026-01-20 14:08:32.164706500 +0000
+++ b/Source/JavaScriptCore/jit/JITOpaqueByproducts.h	2026-05-20 16:09:05.830970439 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "JITOpaqueByproduct.h"
--- a/Source/JavaScriptCore/jit/JITOpcodes32_64.cpp	2026-01-20 14:08:32.165706400 +0000
+++ b/Source/JavaScriptCore/jit/JITOpcodes32_64.cpp	2026-05-20 16:09:05.850970626 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(JIT)
 #if USE(JSVALUE32_64)
--- a/Source/JavaScriptCore/jit/JITOpcodes.cpp	2026-02-23 14:40:52.362317600 +0000
+++ b/Source/JavaScriptCore/jit/JITOpcodes.cpp	2026-05-20 16:09:05.875970860 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #if ENABLE(JIT)
 #include "JIT.h"
 
--- a/Source/JavaScriptCore/jit/JITOperations.cpp	2026-02-23 14:40:52.363317500 +0000
+++ b/Source/JavaScriptCore/jit/JITOperations.cpp	2026-05-20 16:09:05.901971104 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITOperations.h"
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/jit/JITPlan.cpp	2026-02-23 14:40:52.363317500 +0000
+++ b/Source/JavaScriptCore/jit/JITPlan.cpp	2026-05-20 16:09:05.925971328 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITPlan.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITPlan.h	2026-02-23 14:40:52.364317400 +0000
+++ b/Source/JavaScriptCore/jit/JITPlan.h	2026-05-20 16:09:05.947971534 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "CompilationResult.h"
--- a/Source/JavaScriptCore/jit/JITPropertyAccess.cpp	2026-02-23 14:40:52.364317400 +0000
+++ b/Source/JavaScriptCore/jit/JITPropertyAccess.cpp	2026-05-20 16:09:05.969971740 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(JIT)
 #include "JIT.h"
--- a/Source/JavaScriptCore/jit/JITRightShiftGenerator.cpp	2026-01-20 14:08:32.169706600 +0000
+++ b/Source/JavaScriptCore/jit/JITRightShiftGenerator.cpp	2026-05-20 16:09:05.992971956 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITRightShiftGenerator.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITRightShiftGenerator.h	2026-01-20 14:08:32.169706600 +0000
+++ b/Source/JavaScriptCore/jit/JITRightShiftGenerator.h	2026-05-20 16:09:06.014972162 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "JITBitBinaryOpGenerator.h"
--- a/Source/JavaScriptCore/jit/JITSafepoint.cpp	2026-02-23 14:40:52.364317400 +0000
+++ b/Source/JavaScriptCore/jit/JITSafepoint.cpp	2026-05-20 16:09:06.036972368 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITSafepoint.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITSafepoint.h	2026-01-20 14:08:32.170706500 +0000
+++ b/Source/JavaScriptCore/jit/JITSafepoint.h	2026-05-20 16:09:06.058972574 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include <wtf/Vector.h>
--- a/Source/JavaScriptCore/jit/JITScannable.h	2026-01-20 14:08:32.170706500 +0000
+++ b/Source/JavaScriptCore/jit/JITScannable.h	2026-05-20 16:09:06.079972770 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 namespace JSC {
--- a/Source/JavaScriptCore/jit/JITSizeStatistics.cpp	2026-02-23 14:40:52.364317400 +0000
+++ b/Source/JavaScriptCore/jit/JITSizeStatistics.cpp	2026-05-20 16:09:06.101972976 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITSizeStatistics.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITSizeStatistics.h	2026-01-20 14:08:32.171706400 +0000
+++ b/Source/JavaScriptCore/jit/JITSizeStatistics.h	2026-05-20 16:09:06.121973164 +0000
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/jit/JITStubRoutine.cpp	2026-01-20 14:08:32.171706400 +0000
+++ b/Source/JavaScriptCore/jit/JITStubRoutine.cpp	2026-05-20 16:09:06.141973351 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITStubRoutine.h"
 
 #include "AccessCase.h"
--- a/Source/JavaScriptCore/jit/JITStubRoutine.h	2026-02-23 14:40:52.365317300 +0000
+++ b/Source/JavaScriptCore/jit/JITStubRoutine.h	2026-05-20 16:09:06.162973548 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/ExecutableAllocator.h>
 #include <JavaScriptCore/MacroAssemblerCodeRef.h>
 #include <JavaScriptCore/StructureID.h>
--- a/Source/JavaScriptCore/jit/JITSubGenerator.cpp	2026-02-23 14:40:52.365317300 +0000
+++ b/Source/JavaScriptCore/jit/JITSubGenerator.cpp	2026-05-20 16:09:06.182973735 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITSubGenerator.h"
 
 #include "ArithProfile.h"
--- a/Source/JavaScriptCore/jit/JITSubGenerator.h	2026-01-20 14:08:32.172706600 +0000
+++ b/Source/JavaScriptCore/jit/JITSubGenerator.h	2026-05-20 16:09:06.204973941 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/jit/JITThunks.cpp	2026-02-23 14:40:52.365317300 +0000
+++ b/Source/JavaScriptCore/jit/JITThunks.cpp	2026-05-20 16:09:06.226974147 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITThunks.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITToDFGDeferredCompilationCallback.cpp	2026-01-20 14:08:32.173706500 +0000
+++ b/Source/JavaScriptCore/jit/JITToDFGDeferredCompilationCallback.cpp	2026-05-20 16:09:06.246974334 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITToDFGDeferredCompilationCallback.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/jit/JITTZoneImpls.cpp	2026-01-20 14:08:32.172706600 +0000
+++ b/Source/JavaScriptCore/jit/JITTZoneImpls.cpp	2026-05-20 16:09:06.270974559 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(JIT)
 
--- a/Source/JavaScriptCore/jit/JITWorklist.cpp	2026-02-23 14:40:52.365317300 +0000
+++ b/Source/JavaScriptCore/jit/JITWorklist.cpp	2026-05-20 16:09:06.293974774 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITWorklist.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITWorklist.h	2026-02-23 14:40:52.365317300 +0000
+++ b/Source/JavaScriptCore/jit/JITWorklist.h	2026-05-20 16:09:06.310974933 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "JITPlan.h"
--- a/Source/JavaScriptCore/jit/JITWorklistInlines.h	2026-01-20 14:08:32.174706500 +0000
+++ b/Source/JavaScriptCore/jit/JITWorklistInlines.h	2026-05-20 16:09:06.331975130 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "JITWorklist.h"
 
 namespace JSC {
--- a/Source/JavaScriptCore/jit/JITWorklistThread.cpp	2026-02-23 14:40:52.365317300 +0000
+++ b/Source/JavaScriptCore/jit/JITWorklistThread.cpp	2026-05-20 16:09:06.352975327 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "JITWorklistThread.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/JITWorklistThread.h	2026-02-23 14:40:52.366317500 +0000
+++ b/Source/JavaScriptCore/jit/JITWorklistThread.h	2026-05-20 16:09:06.371975505 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "JITPlan.h"
--- a/Source/JavaScriptCore/jit/JSInterfaceJIT.h	2026-02-23 14:40:52.366317500 +0000
+++ b/Source/JavaScriptCore/jit/JSInterfaceJIT.h	2026-05-20 16:09:06.393975711 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/BytecodeConventions.h>
 #include <JavaScriptCore/CCallHelpers.h>
 #include <JavaScriptCore/FPRInfo.h>
--- a/Source/JavaScriptCore/jit/OperationResult.h	2026-02-23 14:40:52.366317500 +0000
+++ b/Source/JavaScriptCore/jit/OperationResult.h	2026-05-20 16:09:06.414975907 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JITOperationValidation.h>
 #include <JavaScriptCore/ThrowScope.h>
 
--- a/Source/JavaScriptCore/jit/PCToCodeOriginMap.cpp	2026-02-23 14:40:52.366317500 +0000
+++ b/Source/JavaScriptCore/jit/PCToCodeOriginMap.cpp	2026-05-20 16:09:06.436976113 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "PCToCodeOriginMap.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/Reg.cpp	2026-01-20 14:08:32.177706500 +0000
+++ b/Source/JavaScriptCore/jit/Reg.cpp	2026-05-20 16:09:06.456976300 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "Reg.h"
 
 #if ENABLE(ASSEMBLER)
--- a/Source/JavaScriptCore/jit/RegisterAtOffset.cpp	2026-01-20 14:08:32.178706400 +0000
+++ b/Source/JavaScriptCore/jit/RegisterAtOffset.cpp	2026-05-20 16:09:06.476976488 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "RegisterAtOffset.h"
 
 #if ENABLE(ASSEMBLER)
--- a/Source/JavaScriptCore/jit/RegisterAtOffsetList.cpp	2026-02-23 14:40:52.367317400 +0000
+++ b/Source/JavaScriptCore/jit/RegisterAtOffsetList.cpp	2026-05-20 16:09:06.496976675 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "RegisterAtOffsetList.h"
 
 #if ENABLE(ASSEMBLER)
--- a/Source/JavaScriptCore/jit/RegisterSet.cpp	2026-02-23 14:40:52.368317400 +0000
+++ b/Source/JavaScriptCore/jit/RegisterSet.cpp	2026-05-20 16:09:06.516976862 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "RegisterSet.h"
 
 #if ENABLE(ASSEMBLER)
--- a/Source/JavaScriptCore/jit/ScratchRegisterAllocator.cpp	2026-01-20 14:08:32.181706400 +0000
+++ b/Source/JavaScriptCore/jit/ScratchRegisterAllocator.cpp	2026-05-20 16:09:06.537977059 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "ScratchRegisterAllocator.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/SetupVarargsFrame.cpp	2026-01-20 14:08:32.181706400 +0000
+++ b/Source/JavaScriptCore/jit/SetupVarargsFrame.cpp	2026-05-20 16:09:06.556977237 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "SetupVarargsFrame.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/SimpleRegisterAllocator.h	2026-02-23 14:40:52.369317500 +0000
+++ b/Source/JavaScriptCore/jit/SimpleRegisterAllocator.h	2026-05-20 16:09:06.578977443 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/RegisterSet.h>
 #include <array>
 
--- a/Source/JavaScriptCore/jit/SlowPathCall.cpp	2026-01-20 14:08:32.183706500 +0000
+++ b/Source/JavaScriptCore/jit/SlowPathCall.cpp	2026-05-20 16:09:06.596977611 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "SlowPathCall.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/jit/SlowPathCall.h	2026-01-20 14:08:32.184706400 +0000
+++ b/Source/JavaScriptCore/jit/SlowPathCall.h	2026-05-20 16:09:06.616977799 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "JIT.h"
 #include "MacroAssemblerCodeRef.h"
 #include "SlowPathFunction.h"
--- a/Source/JavaScriptCore/jit/SnippetOperand.h	2026-02-23 14:40:52.369317500 +0000
+++ b/Source/JavaScriptCore/jit/SnippetOperand.h	2026-05-20 16:09:06.635977977 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "ResultType.h"
--- a/Source/JavaScriptCore/jit/SnippetReg.h	2026-02-23 14:40:52.369317500 +0000
+++ b/Source/JavaScriptCore/jit/SnippetReg.h	2026-05-20 16:09:06.655978164 +0000
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/FPRInfo.h>
 #include <JavaScriptCore/GPRInfo.h>
 #include <JavaScriptCore/Reg.h>
--- a/Source/JavaScriptCore/jit/SpecializedThunkJIT.h	2026-01-20 14:08:32.185706400 +0000
+++ b/Source/JavaScriptCore/jit/SpecializedThunkJIT.h	2026-05-20 16:09:06.676978360 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "JIT.h"
--- a/Source/JavaScriptCore/jit/ThunkGenerators.cpp	2026-02-23 14:40:52.370317500 +0000
+++ b/Source/JavaScriptCore/jit/ThunkGenerators.cpp	2026-05-20 16:09:06.696978548 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "ThunkGenerators.h"
 
 #include "JITOperations.h"
--- a/Source/JavaScriptCore/jit/Width.h	2026-02-23 14:40:52.370317500 +0000
+++ b/Source/JavaScriptCore/jit/Width.h	2026-05-20 16:09:06.718978754 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/CPU.h>
 
 #include <wtf/PrintStream.h>
