--- a/Source/WTF/wtf/ProcessMemoryStatus.h
+++ b/Source/WTF/wtf/ProcessMemoryStatus.h
@@ -25,6 +25,7 @@
  */
 
 #pragma once
+#include <wtf/Platform.h>
 
 #if OS(LINUX) || OS(HAIKU)
 
--- a/Source/WTF/wtf/linux/CurrentProcessMemoryStatus.h
+++ b/Source/WTF/wtf/linux/CurrentProcessMemoryStatus.h
@@ -25,6 +25,7 @@
  */
 
 #pragma once
+#include <wtf/ExportMacros.h>
 
 #include <wtf/ProcessMemoryStatus.h>
 
--- a/Source/WTF/wtf/linux/ProcessMemoryFootprint.h
+++ b/Source/WTF/wtf/linux/ProcessMemoryFootprint.h
@@ -25,6 +25,7 @@
  */
 
 #pragma once
+#include <wtf/Platform.h>
 
 #if OS(LINUX)
 
--- a/Source/WTF/wtf/linux/RealTimeThreads.h
+++ b/Source/WTF/wtf/linux/RealTimeThreads.h
@@ -17,6 +17,8 @@
  */
 
 #pragma once
+#include <wtf/ExportMacros.h>
+#include <wtf/Platform.h>
 
 #include <wtf/CanMakeWeakPtr.h>
 #include <wtf/FastMalloc.h>
