--- a/Source/JavaScriptCore/bytecode/AccessCase.cpp	2026-02-23 14:40:52.253318000 +0000
+++ b/Source/JavaScriptCore/bytecode/AccessCase.cpp	2026-05-20 15:24:28.347419953 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AccessCase.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/bytecode/AccessCaseSnippetParams.cpp	2026-01-20 14:08:31.734283400 +0000
+++ b/Source/JavaScriptCore/bytecode/AccessCaseSnippetParams.cpp	2026-05-20 15:24:28.368420127 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AccessCaseSnippetParams.h"
 
 #include "InlineCacheCompiler.h"
--- a/Source/JavaScriptCore/bytecode/ArithProfile.cpp	2026-02-23 14:40:52.254318000 +0000
+++ b/Source/JavaScriptCore/bytecode/ArithProfile.cpp	2026-05-20 15:24:28.389420302 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "ArithProfile.h"
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/bytecode/ArithProfile.h	2026-02-23 14:40:52.254318000 +0000
+++ b/Source/JavaScriptCore/bytecode/ArithProfile.h	2026-05-20 15:24:28.410420476 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/GPRInfo.h>
 #include <JavaScriptCore/JSCJSValue.h>
 #include <JavaScriptCore/ResultType.h>
--- a/Source/JavaScriptCore/bytecode/BytecodeRewriter.h	2026-02-23 14:40:52.255318000 +0000
+++ b/Source/JavaScriptCore/bytecode/BytecodeRewriter.h	2026-05-20 15:24:28.431420650 +0000
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "BytecodeGenerator.h"
 #include "BytecodeGraph.h"
 #include "BytecodeStructs.h"
--- a/Source/JavaScriptCore/bytecode/CallLinkInfoBase.cpp	2026-02-23 14:40:52.257318000 +0000
+++ b/Source/JavaScriptCore/bytecode/CallLinkInfoBase.cpp	2026-05-20 15:24:28.450420808 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "CallLinkInfoBase.h"
 
 #include "CachedCall.h"
--- a/Source/JavaScriptCore/bytecode/CallLinkInfoBase.h	2026-02-23 14:40:52.257318000 +0000
+++ b/Source/JavaScriptCore/bytecode/CallLinkInfoBase.h	2026-05-20 15:24:28.470420974 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/ArityCheckMode.h>
 #include <JavaScriptCore/CallMode.h>
 #include <JavaScriptCore/JSCPtrTag.h>
--- a/Source/JavaScriptCore/bytecode/CallLinkInfo.cpp	2026-02-23 14:40:52.256318000 +0000
+++ b/Source/JavaScriptCore/bytecode/CallLinkInfo.cpp	2026-05-20 15:24:28.491421148 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "CallLinkInfo.h"
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/bytecode/CallLinkInfo.h	2026-02-23 14:40:52.257318000 +0000
+++ b/Source/JavaScriptCore/bytecode/CallLinkInfo.h	2026-05-20 15:24:28.513421331 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/BaselineJITRegisters.h>
 #include <JavaScriptCore/CallFrame.h>
 #include <JavaScriptCore/CallFrameShuffleData.h>
--- a/Source/JavaScriptCore/bytecode/CallLinkStatus.cpp	2026-02-23 14:40:52.257318000 +0000
+++ b/Source/JavaScriptCore/bytecode/CallLinkStatus.cpp	2026-05-20 15:24:28.535421514 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "CallLinkStatus.h"
 
 #include "BytecodeStructs.h"
--- a/Source/JavaScriptCore/bytecode/CallLinkStatus.h	2026-01-20 14:08:31.742707500 +0000
+++ b/Source/JavaScriptCore/bytecode/CallLinkStatus.h	2026-05-20 15:24:28.555421680 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CallLinkInfo.h"
 #include "CallVariant.h"
 #include "CodeOrigin.h"
--- a/Source/JavaScriptCore/bytecode/CheckPrivateBrandStatus.cpp	2026-01-20 14:08:31.744707600 +0000
+++ b/Source/JavaScriptCore/bytecode/CheckPrivateBrandStatus.cpp	2026-05-20 15:24:28.581421896 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "CheckPrivateBrandStatus.h"
 
 #include "CacheableIdentifierInlines.h"
--- a/Source/JavaScriptCore/bytecode/CheckPrivateBrandStatus.h	2026-01-20 14:08:31.745707500 +0000
+++ b/Source/JavaScriptCore/bytecode/CheckPrivateBrandStatus.h	2026-05-20 15:24:28.599422045 +0000
@@ -27,6 +27,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CacheableIdentifier.h"
 #include "CheckPrivateBrandVariant.h"
 #include "CodeOrigin.h"
--- a/Source/JavaScriptCore/bytecode/CodeBlock.cpp	2026-02-23 14:40:52.258318000 +0000
+++ b/Source/JavaScriptCore/bytecode/CodeBlock.cpp	2026-05-20 15:24:28.621422228 +0000
@@ -28,6 +28,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "CodeBlock.h"
 
 #include "ArithProfile.h"
--- a/Source/JavaScriptCore/bytecode/CodeBlock.h	2026-02-23 14:40:52.258318000 +0000
+++ b/Source/JavaScriptCore/bytecode/CodeBlock.h	2026-05-20 15:24:28.645422427 +0000
@@ -29,6 +29,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/ArrayProfile.h>
 #include <JavaScriptCore/BytecodeConventions.h>
 #include <JavaScriptCore/CallLinkInfo.h>
--- a/Source/JavaScriptCore/bytecode/CodeBlockInlines.h	2026-01-20 14:08:31.746707400 +0000
+++ b/Source/JavaScriptCore/bytecode/CodeBlockInlines.h	2026-05-20 15:24:28.665422593 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "BaselineJITCode.h"
 #include "BytecodeStructs.h"
 #include "CodeBlock.h"
--- a/Source/JavaScriptCore/bytecode/CodeOrigin.h	2026-02-23 14:40:52.259318000 +0000
+++ b/Source/JavaScriptCore/bytecode/CodeOrigin.h	2026-05-20 15:24:28.686422767 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/BytecodeIndex.h>
 #include <JavaScriptCore/JSExportMacros.h>
 
--- a/Source/JavaScriptCore/bytecode/DeleteByStatus.cpp	2026-01-20 14:08:31.750707400 +0000
+++ b/Source/JavaScriptCore/bytecode/DeleteByStatus.cpp	2026-05-20 15:24:28.708422950 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DeleteByStatus.h"
 
 #include "CacheableIdentifierInlines.h"
--- a/Source/JavaScriptCore/bytecode/DeleteByStatus.h	2026-01-20 14:08:31.750707400 +0000
+++ b/Source/JavaScriptCore/bytecode/DeleteByStatus.h	2026-05-20 15:24:28.728423116 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CacheableIdentifier.h"
 #include "CallLinkStatus.h"
 #include "CodeOrigin.h"
--- a/Source/JavaScriptCore/bytecode/DFGExitProfile.cpp	2026-01-20 14:08:31.748707500 +0000
+++ b/Source/JavaScriptCore/bytecode/DFGExitProfile.cpp	2026-05-20 15:24:28.748423282 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGExitProfile.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/bytecode/ExecutionCounter.cpp	2026-01-20 14:08:31.753707400 +0000
+++ b/Source/JavaScriptCore/bytecode/ExecutionCounter.cpp	2026-05-20 15:24:28.770423465 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "ExecutionCounter.h"
 
 #include "CodeBlock.h"
--- a/Source/JavaScriptCore/bytecode/GetByIdMetadata.h	2026-01-20 14:08:31.758707500 +0000
+++ b/Source/JavaScriptCore/bytecode/GetByIdMetadata.h	2026-05-20 15:24:28.791423639 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 namespace JSC {
 
 enum class GetByIdMode : uint8_t {
--- a/Source/JavaScriptCore/bytecode/GetByStatus.cpp	2026-03-03 21:30:17.008498400 +0000
+++ b/Source/JavaScriptCore/bytecode/GetByStatus.cpp	2026-05-20 15:24:28.812423813 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "GetByStatus.h"
 
 #include "BytecodeStructs.h"
--- a/Source/JavaScriptCore/bytecode/GetByStatus.h	2026-01-20 14:08:31.759707500 +0000
+++ b/Source/JavaScriptCore/bytecode/GetByStatus.h	2026-05-20 15:24:28.832423979 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CacheableIdentifier.h"
 #include "CallLinkStatus.h"
 #include "CodeOrigin.h"
--- a/Source/JavaScriptCore/bytecode/GetterSetterAccessCase.cpp	2026-02-23 14:40:52.262318000 +0000
+++ b/Source/JavaScriptCore/bytecode/GetterSetterAccessCase.cpp	2026-05-20 15:24:28.853424154 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "GetterSetterAccessCase.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/bytecode/GetterSetterAccessCase.h	2026-01-20 14:08:31.759707500 +0000
+++ b/Source/JavaScriptCore/bytecode/GetterSetterAccessCase.h	2026-05-20 15:24:28.875424336 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "MacroAssemblerCodeRef.h"
--- a/Source/JavaScriptCore/bytecode/HandlerInfo.h	2026-02-23 14:40:52.262318000 +0000
+++ b/Source/JavaScriptCore/bytecode/HandlerInfo.h	2026-05-20 15:24:28.896424511 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/CodeLocation.h>
 #include <wtf/Forward.h>
 
--- a/Source/JavaScriptCore/bytecode/ICStatusUtils.cpp	2026-01-20 14:08:31.760707400 +0000
+++ b/Source/JavaScriptCore/bytecode/ICStatusUtils.cpp	2026-05-20 15:24:28.916424677 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "ICStatusUtils.h"
 
 #include "CodeBlock.h"
--- a/Source/JavaScriptCore/bytecode/InByStatus.cpp	2026-02-23 14:40:52.262318000 +0000
+++ b/Source/JavaScriptCore/bytecode/InByStatus.cpp	2026-05-20 15:24:28.937424851 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "InByStatus.h"
 
 #include "CacheableIdentifierInlines.h"
--- a/Source/JavaScriptCore/bytecode/InByStatus.h	2026-01-20 14:08:31.760707400 +0000
+++ b/Source/JavaScriptCore/bytecode/InByStatus.h	2026-05-20 15:24:28.958425025 +0000
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CallLinkStatus.h"
 #include "CodeOrigin.h"
 #include "ConcurrentJSLock.h"
--- a/Source/JavaScriptCore/bytecode/InlineAccess.cpp	2026-01-20 14:08:31.761707500 +0000
+++ b/Source/JavaScriptCore/bytecode/InlineAccess.cpp	2026-05-20 15:24:28.979425200 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "InlineAccess.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/bytecode/InlineCacheCompiler.cpp	2026-04-10 12:18:32.159458000 +0000
+++ b/Source/JavaScriptCore/bytecode/InlineCacheCompiler.cpp	2026-05-20 15:24:29.003425399 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "InlineCacheCompiler.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/bytecode/InlineCacheCompiler.h	2026-04-15 11:49:33.418031000 +0000
+++ b/Source/JavaScriptCore/bytecode/InlineCacheCompiler.h	2026-05-20 15:24:29.026425590 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "AccessCase.h"
--- a/Source/JavaScriptCore/bytecode/InstanceOfAccessCase.cpp	2026-01-20 14:08:31.765707500 +0000
+++ b/Source/JavaScriptCore/bytecode/InstanceOfAccessCase.cpp	2026-05-20 15:24:29.047425764 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "InstanceOfAccessCase.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/bytecode/InstanceOfAccessCase.h	2026-01-20 14:08:31.765707500 +0000
+++ b/Source/JavaScriptCore/bytecode/InstanceOfAccessCase.h	2026-05-20 15:24:29.066425922 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "AccessCase.h"
--- a/Source/JavaScriptCore/bytecode/InstanceOfStatus.cpp	2026-01-20 14:08:31.765707500 +0000
+++ b/Source/JavaScriptCore/bytecode/InstanceOfStatus.cpp	2026-05-20 15:24:29.087426097 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "InstanceOfStatus.h"
 
 #include "ICStatusUtils.h"
--- a/Source/JavaScriptCore/bytecode/InstanceOfStatus.h	2026-01-20 14:08:31.766707400 +0000
+++ b/Source/JavaScriptCore/bytecode/InstanceOfStatus.h	2026-05-20 15:24:29.110426288 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "ConcurrentJSLock.h"
 #include "ICStatusMap.h"
 #include "InstanceOfVariant.h"
--- a/Source/JavaScriptCore/bytecode/IntrinsicGetterAccessCase.cpp	2026-02-23 14:40:52.264318000 +0000
+++ b/Source/JavaScriptCore/bytecode/IntrinsicGetterAccessCase.cpp	2026-05-20 15:24:29.130426454 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "IntrinsicGetterAccessCase.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/bytecode/IntrinsicGetterAccessCase.h	2026-01-20 14:08:31.767707300 +0000
+++ b/Source/JavaScriptCore/bytecode/IntrinsicGetterAccessCase.h	2026-05-20 15:24:29.153426645 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "AccessCase.h"
--- a/Source/JavaScriptCore/bytecode/JumpTable.h	2026-02-23 14:40:52.264318000 +0000
+++ b/Source/JavaScriptCore/bytecode/JumpTable.h	2026-05-20 15:24:29.174426819 +0000
@@ -29,6 +29,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/CodeLocation.h>
 #include <JavaScriptCore/UnlinkedCodeBlock.h>
 #include <wtf/FixedVector.h>
--- a/Source/JavaScriptCore/bytecode/MethodOfGettingAValueProfile.cpp	2026-01-20 14:08:31.770707400 +0000
+++ b/Source/JavaScriptCore/bytecode/MethodOfGettingAValueProfile.cpp	2026-05-20 15:24:29.195426993 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "MethodOfGettingAValueProfile.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/bytecode/ModuleNamespaceAccessCase.cpp	2026-01-20 14:08:31.771707500 +0000
+++ b/Source/JavaScriptCore/bytecode/ModuleNamespaceAccessCase.cpp	2026-05-20 15:24:29.215427160 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "ModuleNamespaceAccessCase.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/bytecode/ModuleNamespaceAccessCase.h	2026-01-20 14:08:31.771707500 +0000
+++ b/Source/JavaScriptCore/bytecode/ModuleNamespaceAccessCase.h	2026-05-20 15:24:29.237427342 +0000
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "AccessCase.h"
--- a/Source/JavaScriptCore/bytecode/Opcode.cpp	2026-02-23 14:40:52.266318000 +0000
+++ b/Source/JavaScriptCore/bytecode/Opcode.cpp	2026-05-20 15:24:29.256427500 +0000
@@ -28,6 +28,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "Opcode.h"
 
 #include "BytecodeStructs.h"
--- a/Source/JavaScriptCore/bytecode/Opcode.h	2026-02-23 14:40:52.266318000 +0000
+++ b/Source/JavaScriptCore/bytecode/Opcode.h	2026-05-20 15:24:29.278427683 +0000
@@ -29,6 +29,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/Bytecodes.h>
 #include <JavaScriptCore/LLIntOpcode.h>
 #include <JavaScriptCore/OpcodeSize.h>
--- a/Source/JavaScriptCore/bytecode/Operands.h	2026-02-23 14:40:52.267318000 +0000
+++ b/Source/JavaScriptCore/bytecode/Operands.h	2026-05-20 15:24:29.298427849 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/CallFrame.h>
 #include <JavaScriptCore/VirtualRegister.h>
 
--- a/Source/JavaScriptCore/bytecode/ProxyableAccessCase.cpp	2026-02-23 14:40:52.268318000 +0000
+++ b/Source/JavaScriptCore/bytecode/ProxyableAccessCase.cpp	2026-05-20 15:24:29.320428031 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "ProxyableAccessCase.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/bytecode/ProxyableAccessCase.h	2026-01-20 14:08:31.777707300 +0000
+++ b/Source/JavaScriptCore/bytecode/ProxyableAccessCase.h	2026-05-20 15:24:29.339428189 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "AccessCase.h"
--- a/Source/JavaScriptCore/bytecode/PutByStatus.cpp	2026-02-23 14:40:52.269317900 +0000
+++ b/Source/JavaScriptCore/bytecode/PutByStatus.cpp	2026-05-20 15:24:29.358428347 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "PutByStatus.h"
 
 #include "BytecodeStructs.h"
--- a/Source/JavaScriptCore/bytecode/PutByStatus.h	2026-01-20 14:08:31.778707500 +0000
+++ b/Source/JavaScriptCore/bytecode/PutByStatus.h	2026-05-20 15:24:29.379428521 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CallLinkStatus.h"
 #include "ExitFlag.h"
 #include "ICStatusMap.h"
--- a/Source/JavaScriptCore/bytecode/Repatch.cpp	2026-02-23 14:40:52.269317900 +0000
+++ b/Source/JavaScriptCore/bytecode/Repatch.cpp	2026-05-20 15:24:29.402428712 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "Repatch.h"
 
 #include "BinarySwitch.h"
--- a/Source/JavaScriptCore/bytecode/SetPrivateBrandStatus.cpp	2026-01-20 14:08:31.782707500 +0000
+++ b/Source/JavaScriptCore/bytecode/SetPrivateBrandStatus.cpp	2026-05-20 15:24:29.422428878 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "SetPrivateBrandStatus.h"
 
 #include "CacheableIdentifierInlines.h"
--- a/Source/JavaScriptCore/bytecode/SetPrivateBrandStatus.h	2026-01-20 14:08:31.782707500 +0000
+++ b/Source/JavaScriptCore/bytecode/SetPrivateBrandStatus.h	2026-05-20 15:24:29.443429053 +0000
@@ -27,6 +27,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CacheableIdentifier.h"
 #include "CodeOrigin.h"
 #include "ConcurrentJSLock.h"
--- a/Source/JavaScriptCore/bytecode/SharedJITStubSet.cpp	2026-02-23 14:40:52.270318000 +0000
+++ b/Source/JavaScriptCore/bytecode/SharedJITStubSet.cpp	2026-05-20 15:24:29.464429227 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "SharedJITStubSet.h"
 
 #include "BaselineJITRegisters.h"
--- a/Source/JavaScriptCore/bytecode/SharedJITStubSet.h	2026-02-23 14:40:52.270318000 +0000
+++ b/Source/JavaScriptCore/bytecode/SharedJITStubSet.h	2026-05-20 15:24:29.485429402 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "DOMJITGetterSetter.h"
 #include "StructureStubInfo.h"
 
--- a/Source/JavaScriptCore/bytecode/SpeculatedType.cpp	2026-01-20 14:08:31.783707400 +0000
+++ b/Source/JavaScriptCore/bytecode/SpeculatedType.cpp	2026-05-20 15:24:29.505429568 +0000
@@ -27,6 +27,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "SpeculatedType.h"
 
 #include "DateInstance.h"
--- a/Source/JavaScriptCore/bytecode/SpeculatedType.h	2026-02-23 14:40:52.270318000 +0000
+++ b/Source/JavaScriptCore/bytecode/SpeculatedType.h	2026-05-20 15:24:29.526429742 +0000
@@ -28,6 +28,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/CPU.h>
 #include <JavaScriptCore/JSCJSValue.h>
 #include <JavaScriptCore/TypedArrayType.h>
--- a/Source/JavaScriptCore/bytecode/StructureStubClearingWatchpoint.cpp	2026-01-20 14:08:31.786707400 +0000
+++ b/Source/JavaScriptCore/bytecode/StructureStubClearingWatchpoint.cpp	2026-05-20 15:24:29.548429925 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "StructureStubClearingWatchpoint.h"
 
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/bytecode/StructureStubClearingWatchpoint.h	2026-01-20 14:08:31.786707400 +0000
+++ b/Source/JavaScriptCore/bytecode/StructureStubClearingWatchpoint.h	2026-05-20 15:24:29.568430091 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT)
 
 #include "AdaptiveInferredPropertyValueWatchpointBase.h"
--- a/Source/JavaScriptCore/bytecode/StructureStubInfo.cpp	2026-02-23 14:40:52.271318000 +0000
+++ b/Source/JavaScriptCore/bytecode/StructureStubInfo.cpp	2026-05-20 15:24:29.590430274 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "StructureStubInfo.h"
 
 #include "BaselineJITRegisters.h"
--- a/Source/JavaScriptCore/bytecode/StructureStubInfo.h	2026-01-20 14:08:31.787707300 +0000
+++ b/Source/JavaScriptCore/bytecode/StructureStubInfo.h	2026-05-20 15:24:29.611430448 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CacheableIdentifier.h"
 #include "CodeBlock.h"
 #include "CodeOrigin.h"
--- a/Source/JavaScriptCore/bytecode/UnlinkedCodeBlock.h	2026-02-23 14:40:52.271318000 +0000
+++ b/Source/JavaScriptCore/bytecode/UnlinkedCodeBlock.h	2026-05-20 15:24:29.631430614 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/ArithProfile.h>
 #include <JavaScriptCore/ArrayProfile.h>
 #include <JavaScriptCore/BytecodeConventions.h>
--- a/Source/JavaScriptCore/bytecode/UnlinkedMetadataTable.cpp	2026-02-23 14:40:52.272318000 +0000
+++ b/Source/JavaScriptCore/bytecode/UnlinkedMetadataTable.cpp	2026-05-20 15:24:29.650430772 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "UnlinkedMetadataTable.h"
 
 #include "BytecodeStructs.h"
--- a/Source/JavaScriptCore/bytecode/UnlinkedMetadataTable.h	2026-02-23 14:40:52.272318000 +0000
+++ b/Source/JavaScriptCore/bytecode/UnlinkedMetadataTable.h	2026-05-20 15:24:29.671430946 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/Opcode.h>
 #include <JavaScriptCore/ValueProfile.h>
 #include <wtf/Ref.h>
--- a/Source/JavaScriptCore/bytecode/UnlinkedMetadataTableInlines.h	2026-01-20 14:08:31.797707300 +0000
+++ b/Source/JavaScriptCore/bytecode/UnlinkedMetadataTableInlines.h	2026-05-20 15:24:29.692431121 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "MetadataTable.h"
 #include "UnlinkedMetadataTable.h"
 #include <wtf/FastMalloc.h>
--- a/Source/JavaScriptCore/bytecode/ValueRecovery.cpp	2026-01-20 14:08:31.799707400 +0000
+++ b/Source/JavaScriptCore/bytecode/ValueRecovery.cpp	2026-05-20 15:24:29.713431295 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "ValueRecovery.h"
 
 #include "JSCJSValueInlines.h"
--- a/Source/JavaScriptCore/bytecode/ValueRecovery.h	2026-02-23 14:40:52.274318000 +0000
+++ b/Source/JavaScriptCore/bytecode/ValueRecovery.h	2026-05-20 15:24:29.735431478 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/DFGMinifiedID.h>
 #include <JavaScriptCore/DataFormat.h>
 #if ENABLE(JIT)
--- a/Source/JavaScriptCore/bytecode/VirtualRegister.cpp	2026-01-20 14:08:31.799707400 +0000
+++ b/Source/JavaScriptCore/bytecode/VirtualRegister.cpp	2026-05-20 15:24:29.755431644 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "VirtualRegister.h"
 
 #include "RegisterID.h"
--- a/Source/JavaScriptCore/bytecode/Watchpoint.h	2026-02-23 14:40:52.274318000 +0000
+++ b/Source/JavaScriptCore/bytecode/Watchpoint.h	2026-05-20 15:24:29.775431810 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 #include <wtf/Atomics.h>
 #include <wtf/DebugHeap.h>
