--- a/Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackAndGenerateCode.h	2026-01-20 14:08:31.679089300 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackAndGenerateCode.h	2026-05-20 15:14:08.059704387 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirLiveness.h"
--- a/Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackByLinearScan.h	2026-01-20 14:08:31.683707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackByLinearScan.h	2026-05-20 15:14:08.081704603 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "CPU.h"
--- a/Source/JavaScriptCore/b3/air/AirAllocateRegistersByGreedy.h	2026-01-20 14:08:31.685707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateRegistersByGreedy.h	2026-05-20 15:14:08.101704800 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirAllocateStackByGraphColoring.h	2026-01-20 14:08:31.686707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateStackByGraphColoring.h	2026-05-20 15:14:08.120704987 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirArg.h	2026-05-20 15:05:39.100686926 +0000
+++ b/Source/JavaScriptCore/b3/air/AirArg.h	2026-05-20 15:14:08.141705193 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirArgInlines.h	2026-01-20 14:08:31.687707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirArgInlines.h	2026-05-20 15:14:08.166705439 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirArg.h"
--- a/Source/JavaScriptCore/b3/air/AirBasicBlock.h	2026-05-20 15:05:39.100686926 +0000
+++ b/Source/JavaScriptCore/b3/air/AirBasicBlock.h	2026-05-20 15:14:08.186705635 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirBlockInsertionSet.h	2026-01-20 14:08:31.687707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirBlockInsertionSet.h	2026-05-20 15:14:08.209705861 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3GenericBlockInsertionSet.h"
--- a/Source/JavaScriptCore/b3/air/AirBlockWorklist.h	2026-01-20 14:08:31.687707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirBlockWorklist.h	2026-05-20 15:14:08.243706195 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirBasicBlock.h"
--- a/Source/JavaScriptCore/b3/air/AirBreakCriticalEdges.h	2026-01-20 14:08:31.688707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirBreakCriticalEdges.h	2026-05-20 15:14:08.263706392 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirCCallingConvention.h	2026-05-20 15:05:39.101686936 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCCallingConvention.h	2026-05-20 15:14:08.290706657 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirCCallSpecial.h	2026-01-20 14:08:31.688707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCCallSpecial.h	2026-05-20 15:14:08.314706893 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirSpecial.h"
--- a/Source/JavaScriptCore/b3/air/AirCFG.h	2026-01-20 14:08:31.688707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCFG.h	2026-05-20 15:14:08.335707099 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirBasicBlock.h"
--- a/Source/JavaScriptCore/b3/air/AirCode.h	2026-05-20 15:05:39.101686936 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCode.h	2026-05-20 15:14:08.357707316 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirCustom.h	2026-01-20 14:08:31.690707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCustom.h	2026-05-20 15:14:08.382707561 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirCCallingConvention.h"
--- a/Source/JavaScriptCore/b3/air/AirDominators.h	2026-02-23 14:40:52.241318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirDominators.h	2026-05-20 15:14:08.409707827 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirCFG.h"
--- a/Source/JavaScriptCore/b3/air/AirEliminateDeadCode.h	2026-01-20 14:08:31.692707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirEliminateDeadCode.h	2026-05-20 15:14:08.436708092 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirEmitShuffle.h	2026-01-20 14:08:31.693707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirEmitShuffle.h	2026-05-20 15:14:08.459708318 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirArg.h"
--- a/Source/JavaScriptCore/b3/air/AirFixObviousSpills.h	2026-01-20 14:08:31.694707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirFixObviousSpills.h	2026-05-20 15:14:08.487708593 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirFixPartialRegisterStalls.h	2026-01-20 14:08:31.695707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirFixPartialRegisterStalls.h	2026-05-20 15:14:08.524708957 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirFixSpillsAfterTerminals.h	2026-01-20 14:08:31.695707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirFixSpillsAfterTerminals.h	2026-05-20 15:14:08.559709301 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirFormTable.h	2026-05-20 15:05:39.101686936 +0000
+++ b/Source/JavaScriptCore/b3/air/AirFormTable.h	2026-05-20 15:14:08.587709576 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirFrequentedBlock.h	2026-01-20 14:08:31.695707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirFrequentedBlock.h	2026-05-20 15:14:08.611709812 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3GenericFrequentedBlock.h"
--- a/Source/JavaScriptCore/b3/air/AirGenerate.h	2026-05-20 15:05:39.102686946 +0000
+++ b/Source/JavaScriptCore/b3/air/AirGenerate.h	2026-05-20 15:14:08.631710008 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirGenerationContext.h	2026-02-23 14:40:52.242318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirGenerationContext.h	2026-05-20 15:14:08.652710215 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirBasicBlock.h"
--- a/Source/JavaScriptCore/b3/air/AirHandleCalleeSaves.h	2026-01-20 14:08:31.696707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirHandleCalleeSaves.h	2026-05-20 15:14:08.675710441 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC {
--- a/Source/JavaScriptCore/b3/air/AirHelpers.h	2026-01-20 14:08:31.696707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirHelpers.h	2026-05-20 15:14:08.698710667 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirOpcode.h"
--- a/Source/JavaScriptCore/b3/air/AirInsertionSet.h	2026-01-20 14:08:31.696707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirInsertionSet.h	2026-05-20 15:14:08.722710903 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirInst.h"
--- a/Source/JavaScriptCore/b3/air/AirInst.h	2026-02-23 14:40:52.242318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirInst.h	2026-05-20 15:14:08.741711089 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirArg.h"
--- a/Source/JavaScriptCore/b3/air/AirInstInlines.h	2026-01-20 14:08:31.697707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirInstInlines.h	2026-05-20 15:14:08.761711286 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirInst.h"
--- a/Source/JavaScriptCore/b3/air/AirLivenessAdapter.h	2026-01-20 14:08:31.698707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLivenessAdapter.h	2026-05-20 15:14:08.801711679 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirBasicBlock.h"
--- a/Source/JavaScriptCore/b3/air/AirLiveness.h	2026-01-20 14:08:31.698707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLiveness.h	2026-05-20 15:14:08.821711875 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirLivenessAdapter.h"
--- a/Source/JavaScriptCore/b3/air/AirLogRegisterPressure.h	2026-01-20 14:08:31.698707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLogRegisterPressure.h	2026-05-20 15:14:08.843712092 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirLowerAfterRegAlloc.h	2026-01-20 14:08:31.699707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLowerAfterRegAlloc.h	2026-05-20 15:14:08.865712308 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirLowerEntrySwitch.h	2026-01-20 14:08:31.699707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLowerEntrySwitch.h	2026-05-20 15:14:08.888712534 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirLowerMacros.h	2026-01-20 14:08:31.700707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLowerMacros.h	2026-05-20 15:14:08.909712740 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirLowerStackArgs.h	2026-01-20 14:08:31.701707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLowerStackArgs.h	2026-05-20 15:14:08.935712996 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirNaturalLoops.h	2026-02-23 14:40:52.243318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirNaturalLoops.h	2026-05-20 15:14:08.956713202 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirDominators.h"
--- a/Source/JavaScriptCore/b3/air/AirOptimizeBlockOrder.h	2026-01-20 14:08:31.703707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirOptimizeBlockOrder.h	2026-05-20 15:14:08.977713409 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include <wtf/Vector.h>
--- a/Source/JavaScriptCore/b3/air/AirOptimizePairedLoadStore.h	2026-01-20 14:08:31.704707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirOptimizePairedLoadStore.h	2026-05-20 15:14:08.999713625 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CPU.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirPadInterference.h	2026-01-20 14:08:31.704707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirPadInterference.h	2026-05-20 15:14:09.024713870 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirPhaseInsertionSet.h	2026-01-20 14:08:31.705707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirPhaseInsertionSet.h	2026-05-20 15:14:09.045714077 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirInsertionSet.h"
--- a/Source/JavaScriptCore/b3/air/AirPhaseScope.h	2026-01-20 14:08:31.705707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirPhaseScope.h	2026-05-20 15:14:09.066714283 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "CompilerTimingScope.h"
--- a/Source/JavaScriptCore/b3/air/AirPrintSpecial.h	2026-02-23 14:40:52.243318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirPrintSpecial.h	2026-05-20 15:14:09.086714480 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirInst.h"
--- a/Source/JavaScriptCore/b3/air/AirRegisterAllocatorStats.h	2026-02-23 14:40:52.244318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirRegisterAllocatorStats.h	2026-05-20 15:14:09.109714706 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Bank.h"
--- a/Source/JavaScriptCore/b3/air/AirRegLiveness.h	2026-01-20 14:08:31.706707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirRegLiveness.h	2026-05-20 15:14:09.130714912 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirBasicBlock.h"
--- a/Source/JavaScriptCore/b3/air/AirReportUsedRegisters.h	2026-01-20 14:08:31.707707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirReportUsedRegisters.h	2026-05-20 15:14:09.152715128 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirSimplifyCFG.h	2026-01-20 14:08:31.707707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirSimplifyCFG.h	2026-05-20 15:14:09.173715335 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirSpecial.h	2026-05-20 15:05:39.102686946 +0000
+++ b/Source/JavaScriptCore/b3/air/AirSpecial.h	2026-05-20 15:14:09.194715541 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirStackAllocation.h	2026-01-20 14:08:31.708707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirStackAllocation.h	2026-05-20 15:14:09.213715728 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include <wtf/Vector.h>
--- a/Source/JavaScriptCore/b3/air/AirStackSlot.h	2026-01-20 14:08:31.708707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirStackSlot.h	2026-05-20 15:14:09.233715924 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirStackSlotKind.h"
--- a/Source/JavaScriptCore/b3/air/AirStackSlotKind.h	2026-01-20 14:08:31.709707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirStackSlotKind.h	2026-05-20 15:14:09.253716121 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/air/AirTmp.h	2026-02-23 14:40:52.244318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirTmp.h	2026-05-20 15:14:09.274716327 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Bank.h"
--- a/Source/JavaScriptCore/b3/air/AirTmpInlines.h	2026-01-20 14:08:31.709707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirTmpInlines.h	2026-05-20 15:14:09.296716543 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirArg.h"
--- a/Source/JavaScriptCore/b3/air/AirTmpMap.h	2026-02-23 14:40:52.244318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirTmpMap.h	2026-05-20 15:14:09.318716760 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirCode.h"
--- a/Source/JavaScriptCore/b3/air/AirTmpSet.h	2026-01-20 14:08:31.711707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirTmpSet.h	2026-05-20 15:14:09.337716946 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirTmp.h"
--- a/Source/JavaScriptCore/b3/air/AirTmpWidth.h	2026-01-20 14:08:31.711707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirTmpWidth.h	2026-05-20 15:14:09.358717153 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirArg.h"
--- a/Source/JavaScriptCore/b3/air/AirTmpWidthInlines.h	2026-01-20 14:08:31.711707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirTmpWidthInlines.h	2026-05-20 15:14:09.378717349 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirTmpInlines.h"
--- a/Source/JavaScriptCore/b3/air/AirUseCounts.h	2026-02-23 14:40:52.244318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirUseCounts.h	2026-05-20 15:14:09.398717546 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirArgInlines.h"
--- a/Source/JavaScriptCore/b3/air/AirValidate.h	2026-05-20 15:05:39.102686946 +0000
+++ b/Source/JavaScriptCore/b3/air/AirValidate.h	2026-05-20 15:14:09.420717762 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3BasicBlockInlines.h	2026-01-20 14:08:31.621707700 +0000
+++ b/Source/JavaScriptCore/b3/B3BasicBlockInlines.h	2026-05-20 15:14:09.440717959 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3BasicBlock.h"
--- a/Source/JavaScriptCore/b3/B3BasicBlockUtils.h	2026-01-20 14:08:31.621707700 +0000
+++ b/Source/JavaScriptCore/b3/B3BasicBlockUtils.h	2026-05-20 15:14:09.461718165 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include <wtf/GraphNodeWorklist.h>
--- a/Source/JavaScriptCore/b3/B3BlockInsertionSet.h	2026-01-20 14:08:31.622707800 +0000
+++ b/Source/JavaScriptCore/b3/B3BlockInsertionSet.h	2026-05-20 15:14:09.482718371 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3GenericBlockInsertionSet.h"
--- a/Source/JavaScriptCore/b3/B3BlockWorklist.h	2026-01-20 14:08:31.622707800 +0000
+++ b/Source/JavaScriptCore/b3/B3BlockWorklist.h	2026-05-20 15:14:09.503718578 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include <wtf/GraphNodeWorklist.h>
--- a/Source/JavaScriptCore/b3/B3BottomProvider.h	2026-01-20 14:08:31.622707800 +0000
+++ b/Source/JavaScriptCore/b3/B3BottomProvider.h	2026-05-20 15:14:09.523718774 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3InsertionSet.h"
--- a/Source/JavaScriptCore/b3/B3BottomTupleValue.h	2026-05-20 15:05:39.103686956 +0000
+++ b/Source/JavaScriptCore/b3/B3BottomTupleValue.h	2026-05-20 15:14:09.542718961 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3BreakCriticalEdges.h	2026-05-20 15:05:39.104686966 +0000
+++ b/Source/JavaScriptCore/b3/B3BreakCriticalEdges.h	2026-05-20 15:14:09.564719177 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3BulkMemoryValue.h	2026-05-20 15:05:39.104686966 +0000
+++ b/Source/JavaScriptCore/b3/B3BulkMemoryValue.h	2026-05-20 15:14:09.585719383 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3CaseCollection.h	2026-01-20 14:08:31.626707800 +0000
+++ b/Source/JavaScriptCore/b3/B3CaseCollection.h	2026-05-20 15:14:09.604719570 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3SwitchCase.h"
--- a/Source/JavaScriptCore/b3/B3CaseCollectionInlines.h	2026-01-20 14:08:31.626707800 +0000
+++ b/Source/JavaScriptCore/b3/B3CaseCollectionInlines.h	2026-05-20 15:14:09.624719767 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3CaseCollection.h"
--- a/Source/JavaScriptCore/b3/B3CCallValue.h	2026-05-20 15:05:39.104686966 +0000
+++ b/Source/JavaScriptCore/b3/B3CCallValue.h	2026-05-20 15:14:09.644719963 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3CFG.h	2026-01-20 14:08:31.624707700 +0000
+++ b/Source/JavaScriptCore/b3/B3CFG.h	2026-05-20 15:14:09.665720170 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3BasicBlock.h"
--- a/Source/JavaScriptCore/b3/B3CheckSpecial.h	2026-02-23 14:40:52.230318000 +0000
+++ b/Source/JavaScriptCore/b3/B3CheckSpecial.h	2026-05-20 15:14:09.688720396 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirKind.h"
--- a/Source/JavaScriptCore/b3/B3CheckValue.h	2026-05-20 15:05:39.104686966 +0000
+++ b/Source/JavaScriptCore/b3/B3CheckValue.h	2026-05-20 15:14:09.711720622 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Commutativity.h	2026-01-20 14:08:31.628707600 +0000
+++ b/Source/JavaScriptCore/b3/B3Commutativity.h	2026-05-20 15:14:09.732720828 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3ComputeDivisionMagic.h	2026-02-23 14:40:52.231318200 +0000
+++ b/Source/JavaScriptCore/b3/B3ComputeDivisionMagic.h	2026-05-20 15:14:09.754721044 +0000
@@ -73,6 +73,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include <wtf/MathExtras.h>
--- a/Source/JavaScriptCore/b3/B3Const128Value.h	2026-05-20 15:05:39.105686975 +0000
+++ b/Source/JavaScriptCore/b3/B3Const128Value.h	2026-05-20 15:14:09.775721251 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Const32Value.h	2026-05-20 15:05:39.105686975 +0000
+++ b/Source/JavaScriptCore/b3/B3Const32Value.h	2026-05-20 15:14:09.797721467 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Const64Value.h	2026-05-20 15:05:39.105686975 +0000
+++ b/Source/JavaScriptCore/b3/B3Const64Value.h	2026-05-20 15:14:09.818721673 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ConstDoubleValue.h	2026-05-20 15:05:39.105686975 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstDoubleValue.h	2026-05-20 15:14:09.840721889 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ConstFloatValue.h	2026-05-20 15:05:39.105686975 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstFloatValue.h	2026-05-20 15:14:09.860722086 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ConstPtrValue.h	2026-01-20 14:08:31.631707700 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstPtrValue.h	2026-05-20 15:14:09.881722292 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Const32Value.h"
--- a/Source/JavaScriptCore/b3/B3ConstrainedValue.h	2026-01-20 14:08:31.632707800 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstrainedValue.h	2026-05-20 15:14:09.903722509 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3ValueRep.h"
--- a/Source/JavaScriptCore/b3/B3DataSection.h	2026-01-20 14:08:31.633707800 +0000
+++ b/Source/JavaScriptCore/b3/B3DataSection.h	2026-05-20 15:14:09.924722715 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "JITOpaqueByproduct.h"
--- a/Source/JavaScriptCore/b3/B3Dominators.h	2026-01-20 14:08:31.633707800 +0000
+++ b/Source/JavaScriptCore/b3/B3Dominators.h	2026-05-20 15:14:09.944722911 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3CFG.h"
--- a/Source/JavaScriptCore/b3/B3DuplicateTails.h	2026-01-20 14:08:31.634707700 +0000
+++ b/Source/JavaScriptCore/b3/B3DuplicateTails.h	2026-05-20 15:14:09.965723118 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3Effects.h	2026-05-20 15:05:39.105686975 +0000
+++ b/Source/JavaScriptCore/b3/B3Effects.h	2026-05-20 15:14:09.988723344 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3EliminateCommonSubexpressions.h	2026-01-20 14:08:31.635707900 +0000
+++ b/Source/JavaScriptCore/b3/B3EliminateCommonSubexpressions.h	2026-05-20 15:14:10.007723531 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3EliminateDeadCode.h	2026-05-20 15:05:39.106686985 +0000
+++ b/Source/JavaScriptCore/b3/B3EliminateDeadCode.h	2026-05-20 15:14:10.029723747 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3EnsureLoopPreHeaders.h	2026-01-20 14:08:31.638707600 +0000
+++ b/Source/JavaScriptCore/b3/B3EnsureLoopPreHeaders.h	2026-05-20 15:14:10.048723933 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3EstimateStaticExecutionCounts.h	2026-01-20 14:08:31.639707800 +0000
+++ b/Source/JavaScriptCore/b3/B3EstimateStaticExecutionCounts.h	2026-05-20 15:14:10.068724130 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3ExtractValue.h	2026-05-20 15:05:39.106686985 +0000
+++ b/Source/JavaScriptCore/b3/B3ExtractValue.h	2026-05-20 15:14:10.089724336 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3FenceValue.h	2026-05-20 15:05:39.106686985 +0000
+++ b/Source/JavaScriptCore/b3/B3FenceValue.h	2026-05-20 15:14:10.110724543 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3FixSSA.h	2026-05-20 15:05:39.106686985 +0000
+++ b/Source/JavaScriptCore/b3/B3FixSSA.h	2026-05-20 15:14:10.132724759 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3FoldPathConstants.h	2026-01-20 14:08:31.641707700 +0000
+++ b/Source/JavaScriptCore/b3/B3FoldPathConstants.h	2026-05-20 15:14:10.153724965 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3FrequencyClass.h	2026-01-20 14:08:31.641707700 +0000
+++ b/Source/JavaScriptCore/b3/B3FrequencyClass.h	2026-05-20 15:14:10.175725181 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3FrequentedBlock.h	2026-01-20 14:08:31.641707700 +0000
+++ b/Source/JavaScriptCore/b3/B3FrequentedBlock.h	2026-05-20 15:14:10.198725408 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3GenericFrequentedBlock.h"
--- a/Source/JavaScriptCore/b3/B3Generate.h	2026-05-20 15:05:39.107686995 +0000
+++ b/Source/JavaScriptCore/b3/B3Generate.h	2026-05-20 15:14:10.217725594 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3GenericBlockInsertionSet.h	2026-02-23 14:40:52.231318200 +0000
+++ b/Source/JavaScriptCore/b3/B3GenericBlockInsertionSet.h	2026-05-20 15:14:10.238725801 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "PureNaN.h"
--- a/Source/JavaScriptCore/b3/B3GenericFrequentedBlock.h	2026-01-20 14:08:31.642707800 +0000
+++ b/Source/JavaScriptCore/b3/B3GenericFrequentedBlock.h	2026-05-20 15:14:10.260726017 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3FrequencyClass.h"
--- a/Source/JavaScriptCore/b3/B3HeapRange.h	2026-02-23 14:40:52.231318200 +0000
+++ b/Source/JavaScriptCore/b3/B3HeapRange.h	2026-05-20 15:14:10.280726213 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include <wtf/Range.h>
--- a/Source/JavaScriptCore/b3/B3HoistLoopInvariantValues.h	2026-01-20 14:08:31.643707800 +0000
+++ b/Source/JavaScriptCore/b3/B3HoistLoopInvariantValues.h	2026-05-20 15:14:10.301726420 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3InferSwitches.h	2026-01-20 14:08:31.643707800 +0000
+++ b/Source/JavaScriptCore/b3/B3InferSwitches.h	2026-05-20 15:14:10.321726616 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3InsertionSet.h	2026-01-20 14:08:31.644707700 +0000
+++ b/Source/JavaScriptCore/b3/B3InsertionSet.h	2026-05-20 15:14:10.345726852 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Origin.h"
--- a/Source/JavaScriptCore/b3/B3InsertionSetInlines.h	2026-01-20 14:08:31.644707700 +0000
+++ b/Source/JavaScriptCore/b3/B3InsertionSetInlines.h	2026-05-20 15:14:10.368727078 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3InsertionSet.h"
--- a/Source/JavaScriptCore/b3/B3LegalizeMemoryOffsets.h	2026-01-20 14:08:31.644707700 +0000
+++ b/Source/JavaScriptCore/b3/B3LegalizeMemoryOffsets.h	2026-05-20 15:14:10.403727422 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3LowerInt64.h	2026-01-20 14:08:31.645707600 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerInt64.h	2026-05-20 15:14:10.423727619 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3LowerMacrosAfterOptimizations.h	2026-01-20 14:08:31.647707700 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerMacrosAfterOptimizations.h	2026-05-20 15:14:10.443727815 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3LowerMacros.h	2026-01-20 14:08:31.646707800 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerMacros.h	2026-05-20 15:14:10.464728022 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3LowerToAir.h	2026-05-20 15:05:39.107686995 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerToAir.h	2026-05-20 15:14:10.487728248 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3MathExtras.h	2026-05-20 15:05:39.107686995 +0000
+++ b/Source/JavaScriptCore/b3/B3MathExtras.h	2026-05-20 15:14:10.508728454 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3MemoryValue.h	2026-05-20 15:05:39.108687005 +0000
+++ b/Source/JavaScriptCore/b3/B3MemoryValue.h	2026-05-20 15:14:10.529728660 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3MemoryValueInlines.h	2026-01-20 14:08:31.651707600 +0000
+++ b/Source/JavaScriptCore/b3/B3MemoryValueInlines.h	2026-05-20 15:14:10.548728847 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirArg.h"
--- a/Source/JavaScriptCore/b3/B3MoveConstants.h	2026-05-20 15:05:39.108687005 +0000
+++ b/Source/JavaScriptCore/b3/B3MoveConstants.h	2026-05-20 15:14:10.569729053 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3NativeTraits.h	2026-01-20 14:08:31.651707600 +0000
+++ b/Source/JavaScriptCore/b3/B3NativeTraits.h	2026-05-20 15:14:10.590729260 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Bank.h"
--- a/Source/JavaScriptCore/b3/B3NaturalLoops.h	2026-02-23 14:40:52.235318200 +0000
+++ b/Source/JavaScriptCore/b3/B3NaturalLoops.h	2026-05-20 15:14:10.612729476 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Dominators.h"
--- a/Source/JavaScriptCore/b3/B3Opcode.h	2026-05-20 15:05:39.108687005 +0000
+++ b/Source/JavaScriptCore/b3/B3Opcode.h	2026-05-20 15:14:10.635729702 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3OptimizeAssociativeExpressionTrees.h	2026-01-20 14:08:31.656707800 +0000
+++ b/Source/JavaScriptCore/b3/B3OptimizeAssociativeExpressionTrees.h	2026-05-20 15:14:10.655729899 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3OriginDump.h	2026-01-20 14:08:31.658707600 +0000
+++ b/Source/JavaScriptCore/b3/B3OriginDump.h	2026-05-20 15:14:10.674730085 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Origin.h"
--- a/Source/JavaScriptCore/b3/B3Origin.h	2026-02-23 14:40:52.236318000 +0000
+++ b/Source/JavaScriptCore/b3/B3Origin.h	2026-05-20 15:14:10.697730311 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include <wtf/PrintStream.h>
--- a/Source/JavaScriptCore/b3/B3PatchpointSpecial.h	2026-05-20 15:05:39.108687005 +0000
+++ b/Source/JavaScriptCore/b3/B3PatchpointSpecial.h	2026-05-20 15:14:10.717730508 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3PatchpointValue.h	2026-05-20 15:05:39.109687015 +0000
+++ b/Source/JavaScriptCore/b3/B3PatchpointValue.h	2026-05-20 15:14:10.738730714 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3PCToOriginMap.h	2026-02-23 14:40:52.236318000 +0000
+++ b/Source/JavaScriptCore/b3/B3PCToOriginMap.h	2026-05-20 15:14:10.759730921 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Origin.h"
--- a/Source/JavaScriptCore/b3/B3PhaseScope.h	2026-01-20 14:08:31.660707700 +0000
+++ b/Source/JavaScriptCore/b3/B3PhaseScope.h	2026-05-20 15:14:10.779731117 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "CompilerTimingScope.h"
--- a/Source/JavaScriptCore/b3/B3PhiChildren.h	2026-01-20 14:08:31.660707700 +0000
+++ b/Source/JavaScriptCore/b3/B3PhiChildren.h	2026-05-20 15:14:10.800731323 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Procedure.h"
--- a/Source/JavaScriptCore/b3/B3Procedure.h	2026-05-20 15:05:39.109687015 +0000
+++ b/Source/JavaScriptCore/b3/B3Procedure.h	2026-05-20 15:14:10.822731540 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ProcedureInlines.h	2026-01-20 14:08:31.662707600 +0000
+++ b/Source/JavaScriptCore/b3/B3ProcedureInlines.h	2026-05-20 15:14:10.843731746 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirCode.h"
--- a/Source/JavaScriptCore/b3/B3PureCSE.h	2026-01-20 14:08:31.662707600 +0000
+++ b/Source/JavaScriptCore/b3/B3PureCSE.h	2026-05-20 15:14:10.864731952 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3ValueKey.h"
--- a/Source/JavaScriptCore/b3/B3ReduceDoubleToFloat.h	2026-01-20 14:08:31.663707700 +0000
+++ b/Source/JavaScriptCore/b3/B3ReduceDoubleToFloat.h	2026-05-20 15:14:10.886732169 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3ReduceStrength.h	2026-05-20 15:05:39.109687015 +0000
+++ b/Source/JavaScriptCore/b3/B3ReduceStrength.h	2026-05-20 15:14:10.906732365 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3SIMDValue.h	2026-05-20 15:05:39.109687015 +0000
+++ b/Source/JavaScriptCore/b3/B3SIMDValue.h	2026-05-20 15:14:10.926732562 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3SlotBaseValue.h	2026-05-20 15:05:39.110687025 +0000
+++ b/Source/JavaScriptCore/b3/B3SlotBaseValue.h	2026-05-20 15:14:10.945732748 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3SSACalculator.h	2026-01-20 14:08:31.665707600 +0000
+++ b/Source/JavaScriptCore/b3/B3SSACalculator.h	2026-05-20 15:14:10.967732965 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Dominators.h"
--- a/Source/JavaScriptCore/b3/B3StackmapGenerationParams.h	2026-05-20 15:05:39.110687025 +0000
+++ b/Source/JavaScriptCore/b3/B3StackmapGenerationParams.h	2026-05-20 15:14:10.987733161 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3StackmapSpecial.h	2026-05-20 15:05:39.110687025 +0000
+++ b/Source/JavaScriptCore/b3/B3StackmapSpecial.h	2026-05-20 15:14:11.006733348 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3StackmapValue.h	2026-05-20 15:05:39.111687035 +0000
+++ b/Source/JavaScriptCore/b3/B3StackmapValue.h	2026-05-20 15:14:11.026733544 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3SuccessorCollection.h	2026-01-20 14:08:31.667707700 +0000
+++ b/Source/JavaScriptCore/b3/B3SuccessorCollection.h	2026-05-20 15:14:11.048733761 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3SwitchCase.h	2026-01-20 14:08:31.668707600 +0000
+++ b/Source/JavaScriptCore/b3/B3SwitchCase.h	2026-05-20 15:14:11.068733957 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3FrequentedBlock.h"
--- a/Source/JavaScriptCore/b3/B3SwitchValue.h	2026-05-20 15:05:39.111687035 +0000
+++ b/Source/JavaScriptCore/b3/B3SwitchValue.h	2026-05-20 15:14:11.092734193 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3TypedPointer.h	2026-02-23 14:40:52.237318000 +0000
+++ b/Source/JavaScriptCore/b3/B3TypedPointer.h	2026-05-20 15:14:11.110734370 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC::B3 {
--- a/Source/JavaScriptCore/b3/B3TypeMap.h	2026-01-20 14:08:31.669707800 +0000
+++ b/Source/JavaScriptCore/b3/B3TypeMap.h	2026-05-20 15:14:11.130734566 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Type.h"
--- a/Source/JavaScriptCore/b3/B3UpsilonValue.h	2026-05-20 15:05:39.111687035 +0000
+++ b/Source/JavaScriptCore/b3/B3UpsilonValue.h	2026-05-20 15:14:11.153734792 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3UseCounts.h	2026-05-20 15:05:39.111687035 +0000
+++ b/Source/JavaScriptCore/b3/B3UseCounts.h	2026-05-20 15:14:11.172734979 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Validate.h	2026-05-20 15:05:39.112687045 +0000
+++ b/Source/JavaScriptCore/b3/B3Validate.h	2026-05-20 15:14:11.193735185 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Value.h	2026-05-20 15:05:39.112687045 +0000
+++ b/Source/JavaScriptCore/b3/B3Value.h	2026-05-20 15:14:11.214735392 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ValueInlines.h	2026-02-23 14:40:52.238318200 +0000
+++ b/Source/JavaScriptCore/b3/B3ValueInlines.h	2026-05-20 15:14:11.236735608 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3ArgumentRegValue.h"
--- a/Source/JavaScriptCore/b3/B3ValueKey.h	2026-02-23 14:40:52.238318200 +0000
+++ b/Source/JavaScriptCore/b3/B3ValueKey.h	2026-05-20 15:14:11.257735814 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3HeapRange.h"
--- a/Source/JavaScriptCore/b3/B3ValueKeyInlines.h	2026-02-23 14:40:52.238318200 +0000
+++ b/Source/JavaScriptCore/b3/B3ValueKeyInlines.h	2026-05-20 15:14:11.278736021 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Procedure.h"
--- a/Source/JavaScriptCore/b3/B3Variable.h	2026-01-20 14:08:31.674707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Variable.h	2026-05-20 15:14:11.298736217 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Bank.h"
--- a/Source/JavaScriptCore/b3/B3VariableLiveness.h	2026-01-20 14:08:31.676707700 +0000
+++ b/Source/JavaScriptCore/b3/B3VariableLiveness.h	2026-05-20 15:14:11.319736424 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3BasicBlock.h"
--- a/Source/JavaScriptCore/b3/B3VariableValue.h	2026-05-20 15:05:39.112687045 +0000
+++ b/Source/JavaScriptCore/b3/B3VariableValue.h	2026-05-20 15:14:11.341736640 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3WasmAddressValue.h	2026-05-20 15:05:39.113687055 +0000
+++ b/Source/JavaScriptCore/b3/B3WasmAddressValue.h	2026-05-20 15:14:11.361736836 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3WasmBoundsCheckValue.h	2026-05-20 15:05:39.113687055 +0000
+++ b/Source/JavaScriptCore/b3/B3WasmBoundsCheckValue.h	2026-05-20 15:14:11.381737033 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Width.h	2026-01-20 14:08:31.677707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Width.h	2026-05-20 15:14:11.401737229 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Bank.h"
--- a/Source/JavaScriptCore/b3/testb3.h	2026-03-02 17:11:00.797034700 +0000
+++ b/Source/JavaScriptCore/b3/testb3.h	2026-05-20 15:14:11.424737456 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "AirCCallingConvention.h"
 #include "AirCode.h"
 #include "AirInstInlines.h"
--- a/Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackAndGenerateCode.cpp	2026-02-23 14:40:52.239318100 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackAndGenerateCode.cpp	2026-05-20 15:14:11.446737672 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirAllocateRegistersAndStackAndGenerateCode.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackByLinearScan.cpp	2026-02-23 14:40:52.239318100 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackByLinearScan.cpp	2026-05-20 15:14:11.468737888 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirAllocateRegistersAndStackByLinearScan.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirAllocateRegistersByGraphColoring.cpp	2026-02-23 14:40:52.239318100 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateRegistersByGraphColoring.cpp	2026-05-20 15:14:11.491738114 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirAllocateRegistersByGraphColoring.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirAllocateRegistersByGreedy.cpp	2026-02-23 14:40:52.240318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateRegistersByGreedy.cpp	2026-05-20 15:14:11.512738320 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirAllocateRegistersByGreedy.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirAllocateStackByGraphColoring.cpp	2026-02-23 14:40:52.240318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirAllocateStackByGraphColoring.cpp	2026-05-20 15:14:11.531738507 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirAllocateStackByGraphColoring.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirArg.cpp	2026-02-23 14:40:52.240318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirArg.cpp	2026-05-20 15:14:11.557738763 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirArg.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirBasicBlock.cpp	2026-01-20 14:08:31.687707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirBasicBlock.cpp	2026-05-20 15:14:11.575738939 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirBasicBlock.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirBlockInsertionSet.cpp	2026-01-20 14:08:31.687707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirBlockInsertionSet.cpp	2026-05-20 15:14:11.595739136 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirBlockInsertionSet.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirBreakCriticalEdges.cpp	2026-01-20 14:08:31.687707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirBreakCriticalEdges.cpp	2026-05-20 15:14:11.614739323 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirBreakCriticalEdges.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirCCallingConvention.cpp	2026-02-23 14:40:52.241318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCCallingConvention.cpp	2026-05-20 15:14:11.636739539 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirCCallingConvention.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirCCallSpecial.cpp	2026-01-20 14:08:31.688707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCCallSpecial.cpp	2026-05-20 15:14:11.656739735 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirCCallSpecial.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirCode.cpp	2026-02-23 14:40:52.241318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCode.cpp	2026-05-20 15:14:11.677739942 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirCode.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirCustom.cpp	2026-02-23 14:40:52.241318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCustom.cpp	2026-05-20 15:14:11.696740128 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirCustom.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirDisassembler.cpp	2026-01-20 14:08:31.690707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirDisassembler.cpp	2026-05-20 15:14:11.716740325 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirDisassembler.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirEliminateDeadCode.cpp	2026-01-20 14:08:31.692707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirEliminateDeadCode.cpp	2026-05-20 15:14:11.738740541 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirEliminateDeadCode.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirEmitShuffle.cpp	2026-02-23 14:40:52.242318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirEmitShuffle.cpp	2026-05-20 15:14:11.757740728 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirEmitShuffle.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirFixObviousSpills.cpp	2026-02-23 14:40:52.242318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirFixObviousSpills.cpp	2026-05-20 15:14:11.777740924 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirFixObviousSpills.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirFixPartialRegisterStalls.cpp	2026-01-20 14:08:31.694707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirFixPartialRegisterStalls.cpp	2026-05-20 15:14:11.797741121 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirFixPartialRegisterStalls.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirFixSpillsAfterTerminals.cpp	2026-01-20 14:08:31.695707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirFixSpillsAfterTerminals.cpp	2026-05-20 15:14:11.818741327 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirFixSpillsAfterTerminals.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirGenerate.cpp	2026-02-23 14:40:52.242318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirGenerate.cpp	2026-05-20 15:14:11.839741534 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirGenerate.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirGenerated.cpp	2026-01-20 14:08:31.696707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirGenerated.cpp	2026-05-20 15:14:11.862741760 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
 
--- a/Source/JavaScriptCore/b3/air/AirHandleCalleeSaves.cpp	2026-02-23 14:40:52.242318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirHandleCalleeSaves.cpp	2026-05-20 15:14:11.886741996 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirHandleCalleeSaves.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirInsertionSet.cpp	2026-02-23 14:40:52.242318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirInsertionSet.cpp	2026-05-20 15:14:11.908742212 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirInsertionSet.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirInst.cpp	2026-01-20 14:08:31.697707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirInst.cpp	2026-05-20 15:14:11.930742428 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirInst.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirKind.cpp	2026-01-20 14:08:31.697707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirKind.cpp	2026-05-20 15:14:11.951742634 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirKind.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirLogRegisterPressure.cpp	2026-01-20 14:08:31.698707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLogRegisterPressure.cpp	2026-05-20 15:14:11.973742851 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirLogRegisterPressure.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirLowerAfterRegAlloc.cpp	2026-01-20 14:08:31.699707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLowerAfterRegAlloc.cpp	2026-05-20 15:14:11.993743047 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirLowerAfterRegAlloc.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirLowerEntrySwitch.cpp	2026-02-23 14:40:52.242318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLowerEntrySwitch.cpp	2026-05-20 15:14:12.013743244 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirLowerEntrySwitch.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirLowerMacros.cpp	2026-01-20 14:08:31.700707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLowerMacros.cpp	2026-05-20 15:14:12.034743450 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirLowerMacros.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirLowerStackArgs.cpp	2026-01-20 14:08:31.700707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirLowerStackArgs.cpp	2026-05-20 15:14:12.053743637 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirLowerStackArgs.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirOptimizeBlockOrder.cpp	2026-02-23 14:40:52.243318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirOptimizeBlockOrder.cpp	2026-05-20 15:14:12.075743853 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirOptimizeBlockOrder.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirOptimizePairedLoadStore.cpp	2026-02-23 14:40:52.243318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirOptimizePairedLoadStore.cpp	2026-05-20 15:14:12.095744049 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirOptimizePairedLoadStore.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirPadInterference.cpp	2026-01-20 14:08:31.704707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirPadInterference.cpp	2026-05-20 15:14:12.115744246 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirPadInterference.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirPhaseInsertionSet.cpp	2026-02-23 14:40:52.243318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirPhaseInsertionSet.cpp	2026-05-20 15:14:12.135744443 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirPhaseInsertionSet.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirPhaseScope.cpp	2026-02-23 14:40:52.243318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirPhaseScope.cpp	2026-05-20 15:14:12.160744688 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirPhaseScope.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirPrintSpecial.cpp	2026-01-20 14:08:31.705707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirPrintSpecial.cpp	2026-05-20 15:14:12.180744885 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirPrintSpecial.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirRegLiveness.cpp	2026-01-20 14:08:31.705707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirRegLiveness.cpp	2026-05-20 15:14:12.202745101 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirRegLiveness.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirReportUsedRegisters.cpp	2026-01-20 14:08:31.707707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirReportUsedRegisters.cpp	2026-05-20 15:14:12.223745307 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirReportUsedRegisters.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirSimplifyCFG.cpp	2026-02-23 14:40:52.244318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirSimplifyCFG.cpp	2026-05-20 15:14:12.244745514 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirSimplifyCFG.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirSpecial.cpp	2026-01-20 14:08:31.707707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirSpecial.cpp	2026-05-20 15:14:12.263745700 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirSpecial.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirStackAllocation.cpp	2026-01-20 14:08:31.708707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirStackAllocation.cpp	2026-05-20 15:14:12.285745917 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirStackAllocation.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirStackSlot.cpp	2026-01-20 14:08:31.708707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirStackSlot.cpp	2026-05-20 15:14:12.306746123 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirStackSlot.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirStackSlotKind.cpp	2026-01-20 14:08:31.708707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirStackSlotKind.cpp	2026-05-20 15:14:12.325746310 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirStackSlotKind.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirTmp.cpp	2026-01-20 14:08:31.709707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirTmp.cpp	2026-05-20 15:14:12.346746516 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirTmp.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirTmpWidth.cpp	2026-01-20 14:08:31.711707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirTmpWidth.cpp	2026-05-20 15:14:12.368746732 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirTmpWidth.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/AirValidate.cpp	2026-01-20 14:08:31.712707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirValidate.cpp	2026-05-20 15:14:12.391746958 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "AirValidate.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/air/testair.cpp	2026-02-23 14:40:52.245318200 +0000
+++ b/Source/JavaScriptCore/b3/air/testair.cpp	2026-05-20 15:14:12.412747165 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #include "AirCode.h"
 #include "AirGenerate.h"
--- a/Source/JavaScriptCore/b3/B3AbstractHeap.cpp	2026-02-23 14:40:52.229318100 +0000
+++ b/Source/JavaScriptCore/b3/B3AbstractHeap.cpp	2026-05-20 15:14:12.435747391 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3AbstractHeap.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3AbstractHeapRepository.cpp	2026-02-23 14:40:52.229318100 +0000
+++ b/Source/JavaScriptCore/b3/B3AbstractHeapRepository.cpp	2026-05-20 15:14:12.458747617 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3AbstractHeapRepository.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ArgumentRegValue.cpp	2026-01-20 14:08:31.618708000 +0000
+++ b/Source/JavaScriptCore/b3/B3ArgumentRegValue.cpp	2026-05-20 15:14:12.480747833 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3ArgumentRegValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3AtomicValue.cpp	2026-01-20 14:08:31.618708000 +0000
+++ b/Source/JavaScriptCore/b3/B3AtomicValue.cpp	2026-05-20 15:14:12.502748049 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3AtomicValue.h"
 #include "B3ValueInlines.h"
 
--- a/Source/JavaScriptCore/b3/B3Bank.cpp	2026-01-20 14:08:31.620707800 +0000
+++ b/Source/JavaScriptCore/b3/B3Bank.cpp	2026-05-20 15:14:12.524748265 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Bank.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3BasicBlock.cpp	2026-01-20 14:08:31.621707700 +0000
+++ b/Source/JavaScriptCore/b3/B3BasicBlock.cpp	2026-05-20 15:14:12.545748472 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3BasicBlock.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3BlockInsertionSet.cpp	2026-02-23 14:40:52.230318000 +0000
+++ b/Source/JavaScriptCore/b3/B3BlockInsertionSet.cpp	2026-05-20 15:14:12.566748678 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3BlockInsertionSet.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3BottomTupleValue.cpp	2026-01-20 14:08:31.622707800 +0000
+++ b/Source/JavaScriptCore/b3/B3BottomTupleValue.cpp	2026-05-20 15:14:12.588748894 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3BottomTupleValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3BreakCriticalEdges.cpp	2026-01-20 14:08:31.623707800 +0000
+++ b/Source/JavaScriptCore/b3/B3BreakCriticalEdges.cpp	2026-05-20 15:14:12.611749120 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3BreakCriticalEdges.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3BulkMemoryValue.cpp	2026-02-23 14:40:52.230318000 +0000
+++ b/Source/JavaScriptCore/b3/B3BulkMemoryValue.cpp	2026-05-20 15:14:12.631749317 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3BulkMemoryValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3CanonicalizePrePostIncrements.cpp	2026-01-20 14:08:31.624707700 +0000
+++ b/Source/JavaScriptCore/b3/B3CanonicalizePrePostIncrements.cpp	2026-05-20 15:14:12.653749533 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3CanonicalizePrePostIncrements.h"
 
 #include "B3BackwardsDominators.h"
--- a/Source/JavaScriptCore/b3/B3CaseCollection.cpp	2026-01-20 14:08:31.625707900 +0000
+++ b/Source/JavaScriptCore/b3/B3CaseCollection.cpp	2026-05-20 15:14:12.676749759 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3CaseCollection.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3CCallValue.cpp	2026-01-20 14:08:31.624707700 +0000
+++ b/Source/JavaScriptCore/b3/B3CCallValue.cpp	2026-05-20 15:14:12.699749985 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3CCallValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3CheckSpecial.cpp	2026-02-23 14:40:52.230318000 +0000
+++ b/Source/JavaScriptCore/b3/B3CheckSpecial.cpp	2026-05-20 15:14:12.720750191 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3CheckSpecial.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3CheckValue.cpp	2026-01-20 14:08:31.626707800 +0000
+++ b/Source/JavaScriptCore/b3/B3CheckValue.cpp	2026-05-20 15:14:12.742750407 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3CheckValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Common.cpp	2026-01-20 14:08:31.627707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Common.cpp	2026-05-20 15:14:12.760750584 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Common.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Commutativity.cpp	2026-01-20 14:08:31.627707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Commutativity.cpp	2026-05-20 15:14:12.780750781 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Commutativity.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Compile.cpp	2026-01-20 14:08:31.628707600 +0000
+++ b/Source/JavaScriptCore/b3/B3Compile.cpp	2026-05-20 15:14:12.800750977 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Compile.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Const128Value.cpp	2026-01-20 14:08:31.629707800 +0000
+++ b/Source/JavaScriptCore/b3/B3Const128Value.cpp	2026-05-20 15:14:12.822751194 +0000
@@ -24,6 +24,7 @@
   */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
 #include "B3Const128Value.h"
--- a/Source/JavaScriptCore/b3/B3Const32Value.cpp	2026-01-20 14:08:31.630707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Const32Value.cpp	2026-05-20 15:14:12.841751380 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Const32Value.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Const64Value.cpp	2026-01-20 14:08:31.630707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Const64Value.cpp	2026-05-20 15:14:12.863751597 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Const64Value.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ConstDoubleValue.cpp	2026-01-20 14:08:31.630707700 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstDoubleValue.cpp	2026-05-20 15:14:12.881751773 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3ConstDoubleValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ConstFloatValue.cpp	2026-01-20 14:08:31.631707700 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstFloatValue.cpp	2026-05-20 15:14:12.902751980 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3ConstFloatValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ConstrainedValue.cpp	2026-01-20 14:08:31.632707800 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstrainedValue.cpp	2026-05-20 15:14:12.926752216 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3ConstrainedValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3DataSection.cpp	2026-01-20 14:08:31.633707800 +0000
+++ b/Source/JavaScriptCore/b3/B3DataSection.cpp	2026-05-20 15:14:12.947752422 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3DataSection.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3DuplicateTails.cpp	2026-01-20 14:08:31.634707700 +0000
+++ b/Source/JavaScriptCore/b3/B3DuplicateTails.cpp	2026-05-20 15:14:12.965752599 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3DuplicateTails.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Effects.cpp	2026-02-23 14:40:52.231318200 +0000
+++ b/Source/JavaScriptCore/b3/B3Effects.cpp	2026-05-20 15:14:12.986752805 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Effects.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3EliminateCommonSubexpressions.cpp	2026-02-23 14:40:52.231318200 +0000
+++ b/Source/JavaScriptCore/b3/B3EliminateCommonSubexpressions.cpp	2026-05-20 15:14:13.009753031 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3EliminateCommonSubexpressions.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3EliminateDeadCode.cpp	2026-01-20 14:08:31.635707900 +0000
+++ b/Source/JavaScriptCore/b3/B3EliminateDeadCode.cpp	2026-05-20 15:14:13.030753238 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3EliminateDeadCode.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3EnsureLoopPreHeaders.cpp	2026-01-20 14:08:31.637707700 +0000
+++ b/Source/JavaScriptCore/b3/B3EnsureLoopPreHeaders.cpp	2026-05-20 15:14:13.053753464 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3EnsureLoopPreHeaders.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3EstimateStaticExecutionCounts.cpp	2026-01-20 14:08:31.638707600 +0000
+++ b/Source/JavaScriptCore/b3/B3EstimateStaticExecutionCounts.cpp	2026-05-20 15:14:13.074753670 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3EstimateStaticExecutionCounts.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ExtractValue.cpp	2026-01-20 14:08:31.639707800 +0000
+++ b/Source/JavaScriptCore/b3/B3ExtractValue.cpp	2026-05-20 15:14:13.098753906 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3ExtractValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3FenceValue.cpp	2026-01-20 14:08:31.640707700 +0000
+++ b/Source/JavaScriptCore/b3/B3FenceValue.cpp	2026-05-20 15:14:13.117754093 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3FenceValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3FixSSA.cpp	2026-01-20 14:08:31.640707700 +0000
+++ b/Source/JavaScriptCore/b3/B3FixSSA.cpp	2026-05-20 15:14:13.139754309 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3FixSSA.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3FoldPathConstants.cpp	2026-01-20 14:08:31.641707700 +0000
+++ b/Source/JavaScriptCore/b3/B3FoldPathConstants.cpp	2026-05-20 15:14:13.159754505 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3FoldPathConstants.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3FrequencyClass.cpp	2026-01-20 14:08:31.641707700 +0000
+++ b/Source/JavaScriptCore/b3/B3FrequencyClass.cpp	2026-05-20 15:14:13.181754721 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3FrequencyClass.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Generate.cpp	2026-02-23 14:40:52.231318200 +0000
+++ b/Source/JavaScriptCore/b3/B3Generate.cpp	2026-05-20 15:14:13.203754938 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Generate.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3HoistLoopInvariantValues.cpp	2026-01-20 14:08:31.643707800 +0000
+++ b/Source/JavaScriptCore/b3/B3HoistLoopInvariantValues.cpp	2026-05-20 15:14:13.228755183 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3HoistLoopInvariantValues.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3InferSwitches.cpp	2026-02-23 14:40:52.232318200 +0000
+++ b/Source/JavaScriptCore/b3/B3InferSwitches.cpp	2026-05-20 15:14:13.246755360 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3InferSwitches.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3InsertionSet.cpp	2026-02-23 14:40:52.232318200 +0000
+++ b/Source/JavaScriptCore/b3/B3InsertionSet.cpp	2026-05-20 15:14:13.268755576 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3InsertionSet.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Kind.cpp	2026-01-20 14:08:31.644707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Kind.cpp	2026-05-20 15:14:13.288755773 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Kind.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3LegalizeMemoryOffsets.cpp	2026-01-20 14:08:31.644707700 +0000
+++ b/Source/JavaScriptCore/b3/B3LegalizeMemoryOffsets.cpp	2026-05-20 15:14:13.307755960 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3LegalizeMemoryOffsets.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3LowerMacrosAfterOptimizations.cpp	2026-02-23 14:40:52.232318200 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerMacrosAfterOptimizations.cpp	2026-05-20 15:14:13.324756127 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3LowerMacrosAfterOptimizations.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3LowerMacros.cpp	2026-02-23 14:40:52.232318200 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerMacros.cpp	2026-05-20 15:14:13.344756323 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3LowerMacros.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3LowerToAir32_64.cpp	2026-02-23 14:40:52.234318300 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerToAir32_64.cpp	2026-05-20 15:14:13.365756530 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3LowerToAir.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3LowerToAir.cpp	2026-02-23 14:40:52.234318300 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerToAir.cpp	2026-05-20 15:14:13.393756805 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3LowerToAir.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3MathExtras.cpp	2026-01-20 14:08:31.650707700 +0000
+++ b/Source/JavaScriptCore/b3/B3MathExtras.cpp	2026-05-20 15:14:13.417757041 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3MathExtras.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3MemoryValue.cpp	2026-01-20 14:08:31.650707700 +0000
+++ b/Source/JavaScriptCore/b3/B3MemoryValue.cpp	2026-05-20 15:14:13.442757286 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3MemoryValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3MoveConstants.cpp	2026-02-23 14:40:52.235318200 +0000
+++ b/Source/JavaScriptCore/b3/B3MoveConstants.cpp	2026-05-20 15:14:13.462757483 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3MoveConstants.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Opcode.cpp	2026-01-20 14:08:31.652707800 +0000
+++ b/Source/JavaScriptCore/b3/B3Opcode.cpp	2026-05-20 15:14:13.483757689 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Opcode.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Operations.cpp	2026-02-23 14:40:52.235318200 +0000
+++ b/Source/JavaScriptCore/b3/B3Operations.cpp	2026-05-20 15:14:13.501757866 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Operations.h"
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/b3/B3OptimizeAssociativeExpressionTrees.cpp	2026-02-23 14:40:52.236318000 +0000
+++ b/Source/JavaScriptCore/b3/B3OptimizeAssociativeExpressionTrees.cpp	2026-05-20 15:14:13.523758082 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3OptimizeAssociativeExpressionTrees.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Origin.cpp	2026-01-20 14:08:31.656707800 +0000
+++ b/Source/JavaScriptCore/b3/B3Origin.cpp	2026-05-20 15:14:13.543758279 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Origin.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3OriginDump.cpp	2026-01-20 14:08:31.657707700 +0000
+++ b/Source/JavaScriptCore/b3/B3OriginDump.cpp	2026-05-20 15:14:13.563758475 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3OriginDump.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3PatchpointSpecial.cpp	2026-01-20 14:08:31.658707600 +0000
+++ b/Source/JavaScriptCore/b3/B3PatchpointSpecial.cpp	2026-05-20 15:14:13.582758662 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3PatchpointSpecial.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3PatchpointValue.cpp	2026-01-20 14:08:31.659707800 +0000
+++ b/Source/JavaScriptCore/b3/B3PatchpointValue.cpp	2026-05-20 15:14:13.602758859 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3PatchpointValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3PhaseScope.cpp	2026-02-23 14:40:52.236318000 +0000
+++ b/Source/JavaScriptCore/b3/B3PhaseScope.cpp	2026-05-20 15:14:13.621759045 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3PhaseScope.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3PhiChildren.cpp	2026-01-20 14:08:31.660707700 +0000
+++ b/Source/JavaScriptCore/b3/B3PhiChildren.cpp	2026-05-20 15:14:13.643759261 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3PhiChildren.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Procedure.cpp	2026-02-23 14:40:52.236318000 +0000
+++ b/Source/JavaScriptCore/b3/B3Procedure.cpp	2026-05-20 15:14:13.664759468 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Procedure.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3PureCSE.cpp	2026-01-20 14:08:31.662707600 +0000
+++ b/Source/JavaScriptCore/b3/B3PureCSE.cpp	2026-05-20 15:14:13.683759655 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3PureCSE.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ReduceDoubleToFloat.cpp	2026-01-20 14:08:31.662707600 +0000
+++ b/Source/JavaScriptCore/b3/B3ReduceDoubleToFloat.cpp	2026-05-20 15:14:13.703759851 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3ReduceDoubleToFloat.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ReduceStrength.cpp	2026-03-02 17:11:00.796034800 +0000
+++ b/Source/JavaScriptCore/b3/B3ReduceStrength.cpp	2026-05-20 15:14:13.726760077 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3ReduceStrength.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3SIMDValue.cpp	2026-01-20 14:08:31.665707600 +0000
+++ b/Source/JavaScriptCore/b3/B3SIMDValue.cpp	2026-05-20 15:14:13.747760283 +0000
@@ -24,6 +24,7 @@
   */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3SIMDValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3SlotBaseValue.cpp	2026-01-20 14:08:31.665707600 +0000
+++ b/Source/JavaScriptCore/b3/B3SlotBaseValue.cpp	2026-05-20 15:14:13.768760490 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3SlotBaseValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3SSACalculator.cpp	2026-01-20 14:08:31.665707600 +0000
+++ b/Source/JavaScriptCore/b3/B3SSACalculator.cpp	2026-05-20 15:14:13.789760696 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3SSACalculator.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3StackmapGenerationParams.cpp	2026-02-23 14:40:52.237318000 +0000
+++ b/Source/JavaScriptCore/b3/B3StackmapGenerationParams.cpp	2026-05-20 15:14:13.809760893 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3StackmapGenerationParams.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3StackmapSpecial.cpp	2026-01-20 14:08:31.667707700 +0000
+++ b/Source/JavaScriptCore/b3/B3StackmapSpecial.cpp	2026-05-20 15:14:13.830761099 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3StackmapSpecial.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3StackmapValue.cpp	2026-01-20 14:08:31.667707700 +0000
+++ b/Source/JavaScriptCore/b3/B3StackmapValue.cpp	2026-05-20 15:14:13.851761305 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3StackmapValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3SwitchCase.cpp	2026-01-20 14:08:31.667707700 +0000
+++ b/Source/JavaScriptCore/b3/B3SwitchCase.cpp	2026-05-20 15:14:13.870761492 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3SwitchCase.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3SwitchValue.cpp	2026-01-20 14:08:31.668707600 +0000
+++ b/Source/JavaScriptCore/b3/B3SwitchValue.cpp	2026-05-20 15:14:13.890761689 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3SwitchValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Type.cpp	2026-01-20 14:08:31.668707600 +0000
+++ b/Source/JavaScriptCore/b3/B3Type.cpp	2026-05-20 15:14:13.911761895 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Type.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3TZoneImpls.cpp	2026-01-20 14:08:31.668707600 +0000
+++ b/Source/JavaScriptCore/b3/B3TZoneImpls.cpp	2026-05-20 15:14:13.931762092 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
 
--- a/Source/JavaScriptCore/b3/B3UpsilonValue.cpp	2026-01-20 14:08:31.669707800 +0000
+++ b/Source/JavaScriptCore/b3/B3UpsilonValue.cpp	2026-05-20 15:14:13.951762288 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3UpsilonValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3UseCounts.cpp	2026-02-23 14:40:52.237318000 +0000
+++ b/Source/JavaScriptCore/b3/B3UseCounts.cpp	2026-05-20 15:14:13.971762485 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3UseCounts.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Validate.cpp	2026-02-23 14:40:52.238318200 +0000
+++ b/Source/JavaScriptCore/b3/B3Validate.cpp	2026-05-20 15:14:13.993762701 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Validate.h"
 #include "B3HeapRange.h"
 
--- a/Source/JavaScriptCore/b3/B3Value.cpp	2026-02-23 14:40:52.238318200 +0000
+++ b/Source/JavaScriptCore/b3/B3Value.cpp	2026-05-20 15:14:14.013762897 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Value.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ValueKey.cpp	2026-02-23 14:40:52.238318200 +0000
+++ b/Source/JavaScriptCore/b3/B3ValueKey.cpp	2026-05-20 15:14:14.034763104 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3ValueKey.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3ValueRep.cpp	2026-01-20 14:08:31.673707700 +0000
+++ b/Source/JavaScriptCore/b3/B3ValueRep.cpp	2026-05-20 15:14:14.055763310 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3ValueRep.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Variable.cpp	2026-01-20 14:08:31.674707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Variable.cpp	2026-05-20 15:14:14.079763546 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Variable.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3VariableLiveness.cpp	2026-01-20 14:08:31.675707600 +0000
+++ b/Source/JavaScriptCore/b3/B3VariableLiveness.cpp	2026-05-20 15:14:14.101763762 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3VariableLiveness.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3VariableValue.cpp	2026-01-20 14:08:31.676707700 +0000
+++ b/Source/JavaScriptCore/b3/B3VariableValue.cpp	2026-05-20 15:14:14.126764008 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3VariableValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3WasmAddressValue.cpp	2026-01-20 14:08:31.676707700 +0000
+++ b/Source/JavaScriptCore/b3/B3WasmAddressValue.cpp	2026-05-20 15:14:14.145764195 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3WasmAddressValue.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3WasmBoundsCheckValue.cpp	2026-01-20 14:08:31.676707700 +0000
+++ b/Source/JavaScriptCore/b3/B3WasmBoundsCheckValue.cpp	2026-05-20 15:14:14.165764391 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3WasmBoundsCheckValue.h"
 #include "WasmMemory.h"
 
--- a/Source/JavaScriptCore/b3/B3Width.cpp	2026-01-20 14:08:31.677707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Width.cpp	2026-05-20 15:14:14.185764588 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "B3Width.h"
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/testb3_1.cpp	2026-03-02 17:11:00.797034700 +0000
+++ b/Source/JavaScriptCore/b3/testb3_1.cpp	2026-05-20 15:14:14.207764804 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "testb3.h"
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/b3/testb3_2.cpp	2026-02-23 14:40:52.247318000 +0000
+++ b/Source/JavaScriptCore/b3/testb3_2.cpp	2026-05-20 15:14:14.230765030 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "testb3.h"
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/b3/testb3_3.cpp	2026-02-23 14:40:52.247318000 +0000
+++ b/Source/JavaScriptCore/b3/testb3_3.cpp	2026-05-20 15:14:14.256765285 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "testb3.h"
 
 #include <numbers>
--- a/Source/JavaScriptCore/b3/testb3_4.cpp	2026-01-20 14:08:31.718707600 +0000
+++ b/Source/JavaScriptCore/b3/testb3_4.cpp	2026-05-20 15:14:14.277765492 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "testb3.h"
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/b3/testb3_5.cpp	2026-02-23 14:40:52.248318200 +0000
+++ b/Source/JavaScriptCore/b3/testb3_5.cpp	2026-05-20 15:14:14.299765708 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "testb3.h"
 
 #include <wtf/StdLibExtras.h>
--- a/Source/JavaScriptCore/b3/testb3_6.cpp	2026-01-20 14:08:31.721707600 +0000
+++ b/Source/JavaScriptCore/b3/testb3_6.cpp	2026-05-20 15:14:14.321765924 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "testb3.h"
 
 #include <numbers>
--- a/Source/JavaScriptCore/b3/testb3_7.cpp	2026-03-02 17:11:00.797034700 +0000
+++ b/Source/JavaScriptCore/b3/testb3_7.cpp	2026-05-20 15:14:14.345766160 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "testb3.h"
 #include <wtf/WasmSIMD128.h>
 
--- a/Source/JavaScriptCore/b3/testb3_8.cpp	2026-02-23 14:40:52.249318100 +0000
+++ b/Source/JavaScriptCore/b3/testb3_8.cpp	2026-05-20 15:14:14.368766386 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "testb3.h"
 
 #include <wtf/Int128.h>
