--- a/Source/WebCore/page/UserMessageHandler.h	2026-03-27 22:25:56.013023600 +0000
+++ b/Source/WebCore/page/UserMessageHandler.h	2026-05-20 17:09:42.235033068 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(USER_MESSAGE_HANDLERS)
 
 #include <WebCore/FrameDestructionObserver.h>
--- a/Source/WebCore/page/UserMessageHandler.cpp	2026-03-27 22:25:56.012023700 +0000
+++ b/Source/WebCore/page/UserMessageHandler.cpp	2026-05-20 17:09:42.256033266 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "UserMessageHandler.h"
 
 #if ENABLE(USER_MESSAGE_HANDLERS)
--- a/Source/WebCore/platform/graphics/FormatConverter.cpp	2026-01-20 14:08:45.929674400 +0000
+++ b/Source/WebCore/platform/graphics/FormatConverter.cpp	2026-05-20 17:09:42.276033454 +0000
@@ -26,6 +26,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(WEBGL)
 
--- a/Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp	2026-03-25 16:38:32.927259400 +0000
+++ b/Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp	2026-05-20 17:09:42.296033642 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(WEBGL)
 #include "GraphicsContextGLANGLE.h"
