--- a/Source/WebCore/page/FrameDestructionObserver.h	2026-02-23 14:40:55.400301000 +0000
+++ b/Source/WebCore/page/FrameDestructionObserver.h	2026-05-20 17:22:32.900329316 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include "WebCoreExport.h"
+
 #include <wtf/AbstractRefCountedAndCanMakeWeakPtr.h>
 #include <wtf/WeakPtr.h>
 
--- a/Source/WebCore/page/UserMessageHandlerDescriptor.h	2026-02-23 14:40:55.436300800 +0000
+++ b/Source/WebCore/page/UserMessageHandlerDescriptor.h	2026-05-20 17:22:32.920329506 +0000
@@ -25,6 +25,9 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+#include "WebCoreExport.h"
+
 #if ENABLE(USER_MESSAGE_HANDLERS)
 
 #include <wtf/Ref.h>
--- a/Source/WebCore/page/UserMessageHandlerDescriptor.cpp	2026-02-23 14:40:55.436300800 +0000
+++ b/Source/WebCore/page/UserMessageHandlerDescriptor.cpp	2026-05-20 17:22:32.942329715 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "UserMessageHandlerDescriptor.h"
 
 #if ENABLE(USER_MESSAGE_HANDLERS)
