--- a/Source/JavaScriptCore/ftl/FTLAvailableRecovery.cpp	2026-01-20 14:08:31.961228000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLAvailableRecovery.cpp	2026-05-20 15:42:18.497971599 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLAvailableRecovery.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLAvailableRecovery.h	2026-01-20 14:08:31.961228000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLAvailableRecovery.h	2026-05-20 15:42:18.520971812 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "DFGNode.h"
--- a/Source/JavaScriptCore/ftl/FTLCapabilities.cpp	2026-02-23 14:40:52.302404200 +0000
+++ b/Source/JavaScriptCore/ftl/FTLCapabilities.cpp	2026-05-20 15:42:18.543972024 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLCapabilities.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLCapabilities.h	2026-01-20 14:08:31.961228000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLCapabilities.h	2026-05-20 15:42:18.564972218 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "DFGGraph.h"
--- a/Source/JavaScriptCore/ftl/FTLCommonValues.cpp	2026-01-20 14:08:31.961228000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLCommonValues.cpp	2026-05-20 15:42:18.583972394 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLCommonValues.h"
 
 #include "B3BasicBlockInlines.h"
--- a/Source/JavaScriptCore/ftl/FTLCompile.cpp	2026-02-23 14:40:52.302404200 +0000
+++ b/Source/JavaScriptCore/ftl/FTLCompile.cpp	2026-05-20 15:42:18.603972579 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLCompile.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLCompile.h	2026-01-20 14:08:31.961706900 +0000
+++ b/Source/JavaScriptCore/ftl/FTLCompile.h	2026-05-20 15:42:18.626972791 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "FTLState.h"
--- a/Source/JavaScriptCore/ftl/FTLExceptionTarget.cpp	2026-01-20 14:08:31.961706900 +0000
+++ b/Source/JavaScriptCore/ftl/FTLExceptionTarget.cpp	2026-05-20 15:42:18.647972985 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLExceptionTarget.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLExceptionTarget.h	2026-01-20 14:08:31.962707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLExceptionTarget.h	2026-05-20 15:42:18.667973170 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "DFGCommon.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLExitArgument.cpp	2026-01-20 14:08:31.962707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLExitArgument.cpp	2026-05-20 15:42:18.688973364 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLExitArgument.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLExitArgumentForOperand.cpp	2026-01-20 14:08:31.962707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLExitArgumentForOperand.cpp	2026-05-20 15:42:18.710973568 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLExitArgumentForOperand.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLExitPropertyValue.cpp	2026-01-20 14:08:31.963707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLExitPropertyValue.cpp	2026-05-20 15:42:18.730973753 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLExitPropertyValue.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLExitPropertyValue.h	2026-01-20 14:08:31.964707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLExitPropertyValue.h	2026-05-20 15:42:18.751973947 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "DFGPromotedHeapLocation.h"
--- a/Source/JavaScriptCore/ftl/FTLExitTimeObjectMaterialization.cpp	2026-02-23 14:40:52.303317800 +0000
+++ b/Source/JavaScriptCore/ftl/FTLExitTimeObjectMaterialization.cpp	2026-05-20 15:42:18.771974132 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLExitTimeObjectMaterialization.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLExitTimeObjectMaterialization.h	2026-02-23 14:40:52.303317800 +0000
+++ b/Source/JavaScriptCore/ftl/FTLExitTimeObjectMaterialization.h	2026-05-20 15:42:18.792974326 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "DFGNodeType.h"
--- a/Source/JavaScriptCore/ftl/FTLExitValue.cpp	2026-02-23 14:40:52.303317800 +0000
+++ b/Source/JavaScriptCore/ftl/FTLExitValue.cpp	2026-05-20 15:42:18.813974520 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLExitValue.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLFail.cpp	2026-01-20 14:08:31.966707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLFail.cpp	2026-05-20 15:42:18.835974723 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLFail.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLFail.h	2026-01-20 14:08:31.966707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLFail.h	2026-05-20 15:42:18.855974908 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "FTLState.h"
--- a/Source/JavaScriptCore/ftl/FTLForOSREntryJITCode.cpp	2026-01-20 14:08:31.966707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLForOSREntryJITCode.cpp	2026-05-20 15:42:18.872975065 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLForOSREntryJITCode.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLForOSREntryJITCode.h	2026-02-23 14:40:52.303317800 +0000
+++ b/Source/JavaScriptCore/ftl/FTLForOSREntryJITCode.h	2026-05-20 15:42:18.893975259 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "FTLJITCode.h"
--- a/Source/JavaScriptCore/ftl/FTLJITCode.cpp	2026-02-23 14:40:52.304317700 +0000
+++ b/Source/JavaScriptCore/ftl/FTLJITCode.cpp	2026-05-20 15:42:18.914975453 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLJITCode.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLJITCode.h	2026-01-20 14:08:31.967707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLJITCode.h	2026-05-20 15:42:18.935975647 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "DFGCommonData.h"
--- a/Source/JavaScriptCore/ftl/FTLJITFinalizer.cpp	2026-01-20 14:08:31.968707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLJITFinalizer.cpp	2026-05-20 15:42:18.956975842 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLJITFinalizer.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLJITFinalizer.h	2026-01-20 14:08:31.968707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLJITFinalizer.h	2026-05-20 15:42:18.977976036 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "DFGFinalizer.h"
--- a/Source/JavaScriptCore/ftl/FTLLazySlowPathCall.h	2026-01-20 14:08:31.968707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLLazySlowPathCall.h	2026-05-20 15:42:18.998976230 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "CodeBlock.h"
--- a/Source/JavaScriptCore/ftl/FTLLazySlowPath.cpp	2026-01-20 14:08:31.968707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLLazySlowPath.cpp	2026-05-20 15:42:19.020976433 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLLazySlowPath.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLLazySlowPath.h	2026-01-20 14:08:31.968707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLLazySlowPath.h	2026-05-20 15:42:19.044976655 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/ftl/FTLLink.cpp	2026-02-23 14:40:52.304317700 +0000
+++ b/Source/JavaScriptCore/ftl/FTLLink.cpp	2026-05-20 15:42:19.066976858 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLLink.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLLink.h	2026-01-20 14:08:31.969707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLLink.h	2026-05-20 15:42:19.086977043 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "FTLState.h"
--- a/Source/JavaScriptCore/ftl/FTLLocation.cpp	2026-01-20 14:08:31.969707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLLocation.cpp	2026-05-20 15:42:19.109977256 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLLocation.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLLowerDFGToB3.cpp	2026-03-16 22:06:45.615290900 +0000
+++ b/Source/JavaScriptCore/ftl/FTLLowerDFGToB3.cpp	2026-05-20 15:42:19.136977505 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLLowerDFGToB3.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLLowerDFGToB3.h	2026-01-20 14:08:31.976707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLLowerDFGToB3.h	2026-05-20 15:42:19.164977764 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "DFGGraph.h"
--- a/Source/JavaScriptCore/ftl/FTLLoweredNodeValue.h	2026-01-20 14:08:31.976707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLLoweredNodeValue.h	2026-05-20 15:42:19.185977958 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "DFGBasicBlock.h"
--- a/Source/JavaScriptCore/ftl/FTLOperations.cpp	2026-03-17 09:27:34.222005800 +0000
+++ b/Source/JavaScriptCore/ftl/FTLOperations.cpp	2026-05-20 15:42:19.205978143 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLOperations.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLOperations.h	2026-02-23 14:40:52.309317800 +0000
+++ b/Source/JavaScriptCore/ftl/FTLOperations.h	2026-05-20 15:42:19.224978319 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "DFGOperations.h"
--- a/Source/JavaScriptCore/ftl/FTLOSREntry.cpp	2026-02-23 14:40:52.308317700 +0000
+++ b/Source/JavaScriptCore/ftl/FTLOSREntry.cpp	2026-05-20 15:42:19.242978485 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLOSREntry.h"
 
 #include "CallFrame.h"
--- a/Source/JavaScriptCore/ftl/FTLOSRExitCompiler.cpp	2026-02-23 14:40:52.308317700 +0000
+++ b/Source/JavaScriptCore/ftl/FTLOSRExitCompiler.cpp	2026-05-20 15:42:19.265978698 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLOSRExitCompiler.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLOSRExitCompiler.h	2026-01-20 14:08:31.977707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLOSRExitCompiler.h	2026-05-20 15:42:19.285978883 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "DFGOperations.h"
--- a/Source/JavaScriptCore/ftl/FTLOSRExit.cpp	2026-02-23 14:40:52.308317700 +0000
+++ b/Source/JavaScriptCore/ftl/FTLOSRExit.cpp	2026-05-20 15:42:19.304979058 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLOSRExit.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLOSRExit.h	2026-01-20 14:08:31.976707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLOSRExit.h	2026-05-20 15:42:19.322979225 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "B3ValueRep.h"
--- a/Source/JavaScriptCore/ftl/FTLOSRExitHandle.cpp	2026-01-20 14:08:31.977707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLOSRExitHandle.cpp	2026-05-20 15:42:19.343979419 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLOSRExitHandle.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLOSRExitHandle.h	2026-01-20 14:08:31.978706800 +0000
+++ b/Source/JavaScriptCore/ftl/FTLOSRExitHandle.h	2026-05-20 15:42:19.365979622 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "DFGCommon.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLOutput.cpp	2026-02-23 14:40:52.309317800 +0000
+++ b/Source/JavaScriptCore/ftl/FTLOutput.cpp	2026-05-20 15:42:19.385979807 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLOutput.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLOutput.h	2026-02-23 14:40:52.309317800 +0000
+++ b/Source/JavaScriptCore/ftl/FTLOutput.h	2026-05-20 15:42:19.406980001 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "DFGCommon.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLPatchpointExceptionHandle.cpp	2026-01-20 14:08:31.980707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLPatchpointExceptionHandle.cpp	2026-05-20 15:42:19.426980186 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLPatchpointExceptionHandle.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLPatchpointExceptionHandle.h	2026-01-20 14:08:31.980707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLPatchpointExceptionHandle.h	2026-05-20 15:42:19.446980371 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "DFGCommon.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLRecoveryOpcode.cpp	2026-01-20 14:08:31.981706900 +0000
+++ b/Source/JavaScriptCore/ftl/FTLRecoveryOpcode.cpp	2026-05-20 15:42:19.466980556 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLRecoveryOpcode.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLSaveRestore.cpp	2026-01-20 14:08:31.981706900 +0000
+++ b/Source/JavaScriptCore/ftl/FTLSaveRestore.cpp	2026-05-20 15:42:19.487980750 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLSaveRestore.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLSelectPredictability.h	2026-01-20 14:08:31.981706900 +0000
+++ b/Source/JavaScriptCore/ftl/FTLSelectPredictability.h	2026-05-20 15:42:19.508980944 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 namespace JSC::FTL {
--- a/Source/JavaScriptCore/ftl/FTLSlowPathCall.cpp	2026-01-20 14:08:31.981706900 +0000
+++ b/Source/JavaScriptCore/ftl/FTLSlowPathCall.cpp	2026-05-20 15:42:19.530981147 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLSlowPathCall.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLSlowPathCall.h	2026-01-20 14:08:31.982707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLSlowPathCall.h	2026-05-20 15:42:19.545981286 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/ftl/FTLSlowPathCallKey.cpp	2026-01-20 14:08:31.982707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLSlowPathCallKey.cpp	2026-05-20 15:42:19.566981480 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLSlowPathCallKey.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLSnippetParams.cpp	2026-01-20 14:08:31.982707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLSnippetParams.cpp	2026-05-20 15:42:19.585981656 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLSnippetParams.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLSnippetParams.h	2026-02-23 14:40:52.309317800 +0000
+++ b/Source/JavaScriptCore/ftl/FTLSnippetParams.h	2026-05-20 15:42:19.604981831 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "B3StackmapGenerationParams.h"
--- a/Source/JavaScriptCore/ftl/FTLState.cpp	2026-02-23 14:40:52.310317800 +0000
+++ b/Source/JavaScriptCore/ftl/FTLState.cpp	2026-05-20 15:42:19.632982090 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLState.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLState.h	2026-01-20 14:08:31.983707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLState.h	2026-05-20 15:42:19.654982293 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "B3Procedure.h"
--- a/Source/JavaScriptCore/ftl/FTLSwitchCase.h	2026-01-20 14:08:31.983707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLSwitchCase.h	2026-05-20 15:42:19.674982478 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "FTLAbbreviatedTypes.h"
--- a/Source/JavaScriptCore/ftl/FTLThunks.cpp	2026-02-23 14:40:52.310317800 +0000
+++ b/Source/JavaScriptCore/ftl/FTLThunks.cpp	2026-05-20 15:42:19.698982700 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLThunks.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLValueRange.cpp	2026-01-20 14:08:31.983707000 +0000
+++ b/Source/JavaScriptCore/ftl/FTLValueRange.cpp	2026-05-20 15:42:19.722982922 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "FTLValueRange.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/ftl/FTLWeightedTarget.h	2026-01-20 14:08:31.984706900 +0000
+++ b/Source/JavaScriptCore/ftl/FTLWeightedTarget.h	2026-05-20 15:42:19.743983116 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "FTLAbbreviatedTypes.h"
--- a/Source/JavaScriptCore/ftl/FTLWeight.h	2026-01-20 14:08:31.984706900 +0000
+++ b/Source/JavaScriptCore/ftl/FTLWeight.h	2026-05-20 15:42:19.765983319 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "B3FrequencyClass.h"
