--- a/Source/WebCore/platform/graphics/texmap/GraphicsContextGLTextureMapperANGLE.h
+++ b/Source/WebCore/platform/graphics/texmap/GraphicsContextGLTextureMapperANGLE.h
@@ -23,6 +23,9 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <WebCore/PlatformExportMacros.h>
+#include <wtf/Platform.h>
+
 #pragma once
 
 #include <wtf/Platform.h>
--- a/Source/WebCore/platform/graphics/texmap/TextureMapper.h
+++ b/Source/WebCore/platform/graphics/texmap/TextureMapper.h
@@ -17,6 +17,9 @@
  Boston, MA 02110-1301, USA.
  */
 
+#include <WebCore/PlatformExportMacros.h>
+#include <wtf/Platform.h>
+
 #pragma once
 
 #include <wtf/Platform.h>
--- a/Source/WebCore/platform/graphics/texmap/TextureMapperAnimation.h
+++ b/Source/WebCore/platform/graphics/texmap/TextureMapperAnimation.h
@@ -17,6 +17,9 @@
  * Boston, MA 02110-1301, USA.
  */
 
+#include <WebCore/PlatformExportMacros.h>
+#include <wtf/Platform.h>
+
 #pragma once
 
 #include <wtf/Platform.h>
--- a/Source/WebCore/platform/graphics/texmap/TextureMapperDamageVisualizer.h
+++ b/Source/WebCore/platform/graphics/texmap/TextureMapperDamageVisualizer.h
@@ -23,6 +23,8 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <wtf/Platform.h>
+
 #pragma once
 
 #include <wtf/Platform.h>
--- a/Source/WebCore/platform/graphics/texmap/TextureMapperPlatformLayer.h
+++ b/Source/WebCore/platform/graphics/texmap/TextureMapperPlatformLayer.h
@@ -17,6 +17,9 @@
     Boston, MA 02110-1301, USA.
 */
 
+#include <WebCore/PlatformExportMacros.h>
+#include <wtf/Platform.h>
+
 #pragma once
 
 #include <wtf/Platform.h>
