--- a/Source/JavaScriptCore/dfg/DFGAbstractHeap.cpp	2026-01-20 14:08:31.812707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAbstractHeap.cpp	2026-05-20 15:31:19.343969881 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGAbstractHeap.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGAbstractHeap.h	2026-02-23 14:40:52.278318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAbstractHeap.h	2026-05-20 15:31:19.362970050 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DOMJITHeapRange.h"
--- a/Source/JavaScriptCore/dfg/DFGAbstractInterpreterClobberState.cpp	2026-01-20 14:08:31.813864700 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAbstractInterpreterClobberState.cpp	2026-05-20 15:31:19.384970245 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGAbstractInterpreterClobberState.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGAbstractInterpreterClobberState.h	2026-01-20 14:08:31.813864700 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAbstractInterpreterClobberState.h	2026-05-20 15:31:19.404970423 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <algorithm>
 #include <cstdint>
 
--- a/Source/JavaScriptCore/dfg/DFGAbstractInterpreter.h	2026-02-23 14:40:52.278318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAbstractInterpreter.h	2026-05-20 15:31:19.425970610 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGAbstractValue.h"
--- a/Source/JavaScriptCore/dfg/DFGAbstractInterpreterInlines.h	2026-03-16 22:06:45.610290800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAbstractInterpreterInlines.h	2026-05-20 15:31:19.446970797 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "dfg/DFGNodeType.h"
 #if ENABLE(DFG_JIT)
 
--- a/Source/JavaScriptCore/dfg/DFGAbstractValueClobberEpoch.cpp	2026-01-20 14:08:31.816707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAbstractValueClobberEpoch.cpp	2026-05-20 15:31:19.466970975 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGAbstractValueClobberEpoch.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGAbstractValueClobberEpoch.h	2026-01-20 14:08:31.816707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAbstractValueClobberEpoch.h	2026-05-20 15:31:19.487971161 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGEpoch.h"
--- a/Source/JavaScriptCore/dfg/DFGAbstractValue.cpp	2026-01-20 14:08:31.815707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAbstractValue.cpp	2026-05-20 15:31:19.507971339 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGAbstractValue.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGAbstractValue.h	2026-02-23 14:40:52.279317900 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAbstractValue.h	2026-05-20 15:31:19.527971517 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "ArrayProfile.h"
--- a/Source/JavaScriptCore/dfg/DFGAdaptiveInferredPropertyValueWatchpoint.cpp	2026-01-20 14:08:31.816707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAdaptiveInferredPropertyValueWatchpoint.cpp	2026-05-20 15:31:19.545971677 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGAdaptiveInferredPropertyValueWatchpoint.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGAdaptiveInferredPropertyValueWatchpoint.h	2026-01-20 14:08:31.817707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAdaptiveInferredPropertyValueWatchpoint.h	2026-05-20 15:31:19.567971873 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "AdaptiveInferredPropertyValueWatchpointBase.h"
--- a/Source/JavaScriptCore/dfg/DFGAdaptiveStructureWatchpoint.cpp	2026-01-20 14:08:31.817707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAdaptiveStructureWatchpoint.cpp	2026-05-20 15:31:19.588972060 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGAdaptiveStructureWatchpoint.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGAdaptiveStructureWatchpoint.h	2026-01-20 14:08:31.817707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAdaptiveStructureWatchpoint.h	2026-05-20 15:31:19.607972229 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "CodeBlock.h"
--- a/Source/JavaScriptCore/dfg/DFGAdjacencyList.h	2026-01-20 14:08:31.818707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAdjacencyList.h	2026-05-20 15:31:19.625972389 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGEdge.h"
--- a/Source/JavaScriptCore/dfg/DFGArgumentPosition.h	2026-01-20 14:08:31.818707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGArgumentPosition.h	2026-05-20 15:31:19.645972567 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGDoubleFormatState.h"
--- a/Source/JavaScriptCore/dfg/DFGArgumentsEliminationPhase.cpp	2026-02-23 14:40:52.279317900 +0000
+++ b/Source/JavaScriptCore/dfg/DFGArgumentsEliminationPhase.cpp	2026-05-20 15:31:19.676972843 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGArgumentsEliminationPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGArgumentsEliminationPhase.h	2026-01-20 14:08:31.818707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGArgumentsEliminationPhase.h	2026-05-20 15:31:19.718973216 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGArgumentsUtilities.cpp	2026-02-23 14:40:52.279317900 +0000
+++ b/Source/JavaScriptCore/dfg/DFGArgumentsUtilities.cpp	2026-05-20 15:31:19.747973474 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGArgumentsUtilities.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGArgumentsUtilities.h	2026-01-20 14:08:31.819707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGArgumentsUtilities.h	2026-05-20 15:31:19.766973643 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGInsertionSet.h"
--- a/Source/JavaScriptCore/dfg/DFGArithMode.cpp	2026-01-20 14:08:31.819707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGArithMode.cpp	2026-05-20 15:31:19.788973839 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGArithMode.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGArithMode.h	2026-01-20 14:08:31.819707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGArithMode.h	2026-05-20 15:31:19.814974070 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "MathCommon.h"
 #include "OperationResult.h"
 
--- a/Source/JavaScriptCore/dfg/DFGArrayifySlowPathGenerator.h	2026-01-20 14:08:31.821707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGArrayifySlowPathGenerator.h	2026-05-20 15:31:19.833974239 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGArrayMode.h"
--- a/Source/JavaScriptCore/dfg/DFGArrayMode.cpp	2026-02-23 14:40:52.280318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGArrayMode.cpp	2026-05-20 15:31:19.852974408 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGArrayMode.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGArrayMode.h	2026-01-20 14:08:31.821707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGArrayMode.h	2026-05-20 15:31:19.874974604 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "ArrayProfile.h"
--- a/Source/JavaScriptCore/dfg/DFGAtTailAbstractState.cpp	2026-01-20 14:08:31.823707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAtTailAbstractState.cpp	2026-05-20 15:31:19.893974773 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGAtTailAbstractState.h"
 #include "DFGBlockMapInlines.h"
 
--- a/Source/JavaScriptCore/dfg/DFGAtTailAbstractState.h	2026-02-23 14:40:52.280318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAtTailAbstractState.h	2026-05-20 15:31:19.913974951 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGAbstractInterpreterClobberState.h"
--- a/Source/JavaScriptCore/dfg/DFGAvailability.cpp	2026-01-20 14:08:31.823707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAvailability.cpp	2026-05-20 15:31:19.932975120 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGAvailability.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGAvailability.h	2026-01-20 14:08:31.823707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAvailability.h	2026-05-20 15:31:19.953975306 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGFlushedAt.h"
--- a/Source/JavaScriptCore/dfg/DFGAvailabilityMap.cpp	2026-01-20 14:08:31.824707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAvailabilityMap.cpp	2026-05-20 15:31:19.975975502 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGAvailabilityMap.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGAvailabilityMap.h	2026-01-20 14:08:31.825707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGAvailabilityMap.h	2026-05-20 15:31:19.999975716 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGAvailability.h"
--- a/Source/JavaScriptCore/dfg/DFGBackwardsCFG.h	2026-01-20 14:08:31.825707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBackwardsCFG.h	2026-05-20 15:31:20.023975929 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCFG.h"
--- a/Source/JavaScriptCore/dfg/DFGBackwardsDominators.h	2026-01-20 14:08:31.825707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBackwardsDominators.h	2026-05-20 15:31:20.047976143 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGBackwardsCFG.h"
--- a/Source/JavaScriptCore/dfg/DFGBackwardsPropagationPhase.cpp	2026-04-07 12:17:48.309944000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBackwardsPropagationPhase.cpp	2026-05-20 15:31:20.069976338 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGBackwardsPropagationPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGBackwardsPropagationPhase.h	2026-01-20 14:08:31.825707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBackwardsPropagationPhase.h	2026-05-20 15:31:20.094976561 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 
--- a/Source/JavaScriptCore/dfg/DFGBasicBlock.cpp	2026-01-20 14:08:31.826707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBasicBlock.cpp	2026-05-20 15:31:20.119976783 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGBasicBlock.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGBasicBlock.h	2026-01-20 14:08:31.826707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBasicBlock.h	2026-05-20 15:31:20.146977023 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGAbstractValue.h"
--- a/Source/JavaScriptCore/dfg/DFGBasicBlockInlines.h	2026-01-20 14:08:31.826707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBasicBlockInlines.h	2026-05-20 15:31:20.170977237 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "DFGBasicBlock.h"
 #include "DFGGraph.h"
 
--- a/Source/JavaScriptCore/dfg/DFGBlockInsertionSet.cpp	2026-02-23 14:40:52.280318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBlockInsertionSet.cpp	2026-05-20 15:31:20.192977432 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGBlockInsertionSet.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGBlockInsertionSet.h	2026-01-20 14:08:31.827707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBlockInsertionSet.h	2026-05-20 15:31:20.214977628 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGGraph.h"
--- a/Source/JavaScriptCore/dfg/DFGBlockMap.h	2026-01-20 14:08:31.827707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBlockMap.h	2026-05-20 15:31:20.238977842 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGBasicBlock.h"
--- a/Source/JavaScriptCore/dfg/DFGBlockMapInlines.h	2026-01-20 14:08:31.827707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBlockMapInlines.h	2026-05-20 15:31:20.261978046 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGBlockMap.h"
--- a/Source/JavaScriptCore/dfg/DFGBlockSet.cpp	2026-01-20 14:08:31.827707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBlockSet.cpp	2026-05-20 15:31:20.281978224 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGBlockSet.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGBlockSet.h	2026-01-20 14:08:31.827707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBlockSet.h	2026-05-20 15:31:20.302978411 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGBasicBlock.h"
--- a/Source/JavaScriptCore/dfg/DFGBlockSetInlines.h	2026-01-20 14:08:31.828707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBlockSetInlines.h	2026-05-20 15:31:20.325978616 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGBlockSet.h"
--- a/Source/JavaScriptCore/dfg/DFGBlockWorklist.h	2026-01-20 14:08:31.828707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBlockWorklist.h	2026-05-20 15:31:20.345978793 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGBasicBlock.h"
--- a/Source/JavaScriptCore/dfg/DFGBranchDirection.h	2026-01-20 14:08:31.828707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGBranchDirection.h	2026-05-20 15:31:20.366978980 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGByteCodeParser.cpp	2026-02-23 14:40:52.282317900 +0000
+++ b/Source/JavaScriptCore/dfg/DFGByteCodeParser.cpp	2026-05-20 15:31:20.391979203 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGByteCodeParser.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGByteCodeParser.h	2026-01-20 14:08:31.832707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGByteCodeParser.h	2026-05-20 15:31:20.413979398 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGCallArrayAllocatorSlowPathGenerator.h	2026-01-20 14:08:31.838707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCallArrayAllocatorSlowPathGenerator.h	2026-05-20 15:31:20.433979576 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGSlowPathGenerator.h"
--- a/Source/JavaScriptCore/dfg/DFGCallCreateDirectArgumentsSlowPathGenerator.h	2026-01-20 14:08:31.838707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCallCreateDirectArgumentsSlowPathGenerator.h	2026-05-20 15:31:20.452979745 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGOperations.h"
--- a/Source/JavaScriptCore/dfg/DFGCapabilities.cpp	2026-02-23 14:40:52.282317900 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCapabilities.cpp	2026-05-20 15:31:20.472979923 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGCapabilities.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGCapabilities.h	2026-01-20 14:08:31.839707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCapabilities.h	2026-05-20 15:31:20.495980128 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CodeBlock.h"
 #include "DFGCommon.h"
 #include "Options.h"
--- a/Source/JavaScriptCore/dfg/DFGCFAPhase.cpp	2026-01-20 14:08:31.832707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCFAPhase.cpp	2026-05-20 15:31:20.515980306 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGCFAPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGCFAPhase.h	2026-01-20 14:08:31.833707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCFAPhase.h	2026-05-20 15:31:20.537980501 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGCFG.h	2026-02-23 14:40:52.282317900 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCFG.h	2026-05-20 15:31:20.558980688 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGBasicBlock.h"
--- a/Source/JavaScriptCore/dfg/DFGCFGSimplificationPhase.cpp	2026-01-20 14:08:31.835707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCFGSimplificationPhase.cpp	2026-05-20 15:31:20.578980866 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGCFGSimplificationPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGCFGSimplificationPhase.h	2026-01-20 14:08:31.835707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCFGSimplificationPhase.h	2026-05-20 15:31:20.597981035 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGCleanUpPhase.cpp	2026-01-20 14:08:31.839707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCleanUpPhase.cpp	2026-05-20 15:31:20.620981240 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGCleanUpPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGCleanUpPhase.h	2026-01-20 14:08:31.839707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCleanUpPhase.h	2026-05-20 15:31:20.642981436 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGClobberize.cpp	2026-01-20 14:08:31.840707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGClobberize.cpp	2026-05-20 15:31:20.663981622 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGClobberize.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGClobberize.h	2026-02-23 14:40:52.283317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGClobberize.h	2026-05-20 15:31:20.685981818 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGAbstractHeap.h"
--- a/Source/JavaScriptCore/dfg/DFGClobberSet.cpp	2026-01-20 14:08:31.839707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGClobberSet.cpp	2026-05-20 15:31:20.705981996 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGClobberSet.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGClobberSet.h	2026-01-20 14:08:31.839707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGClobberSet.h	2026-05-20 15:31:20.723982156 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGAbstractHeap.h"
--- a/Source/JavaScriptCore/dfg/DFGClobbersExitState.cpp	2026-01-20 14:08:31.841707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGClobbersExitState.cpp	2026-05-20 15:31:20.746982361 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGClobbersExitState.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGClobbersExitState.h	2026-01-20 14:08:31.841707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGClobbersExitState.h	2026-05-20 15:31:20.768982556 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGCloneHelper.cpp	2026-02-23 14:40:52.283317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCloneHelper.cpp	2026-05-20 15:31:20.787982725 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGCloneHelper.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGCloneHelper.h	2026-02-23 14:40:52.283317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCloneHelper.h	2026-05-20 15:31:20.807982903 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGBlockInsertionSet.h"
--- a/Source/JavaScriptCore/dfg/DFGCodeOriginPool.cpp	2026-01-20 14:08:31.842707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCodeOriginPool.cpp	2026-05-20 15:31:20.825983063 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGCodeOriginPool.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGCombinedLiveness.cpp	2026-01-20 14:08:31.842707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCombinedLiveness.cpp	2026-05-20 15:31:20.845983241 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGCombinedLiveness.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGCombinedLiveness.h	2026-01-20 14:08:31.843707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCombinedLiveness.h	2026-05-20 15:31:20.865983419 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGBlockMap.h"
--- a/Source/JavaScriptCore/dfg/DFGCommon.cpp	2026-04-10 09:30:55.859679700 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCommon.cpp	2026-05-20 15:31:20.885983597 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGCommon.h"
 
 #include <wtf/Lock.h>
--- a/Source/JavaScriptCore/dfg/DFGCommonData.cpp	2026-02-23 14:40:52.283317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCommonData.cpp	2026-05-20 15:31:20.905983775 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGCommonData.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGCommonData.h	2026-01-20 14:08:31.843707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCommonData.h	2026-05-20 15:31:20.925983953 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "BaselineJITCode.h"
--- a/Source/JavaScriptCore/dfg/DFGConstantFoldingPhase.cpp	2026-02-23 14:40:52.284318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGConstantFoldingPhase.cpp	2026-05-20 15:31:20.946984140 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGConstantFoldingPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGConstantFoldingPhase.h	2026-01-20 14:08:31.844707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGConstantFoldingPhase.h	2026-05-20 15:31:20.968984336 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGConstantHoistingPhase.cpp	2026-01-20 14:08:31.845707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGConstantHoistingPhase.cpp	2026-05-20 15:31:20.988984513 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGConstantHoistingPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGConstantHoistingPhase.h	2026-01-20 14:08:31.845707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGConstantHoistingPhase.h	2026-05-20 15:31:21.009984700 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGControlEquivalenceAnalysis.h	2026-01-20 14:08:31.845707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGControlEquivalenceAnalysis.h	2026-05-20 15:31:21.031984896 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGBackwardsDominators.h"
--- a/Source/JavaScriptCore/dfg/DFGCPSRethreadingPhase.cpp	2026-01-20 14:08:31.835707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCPSRethreadingPhase.cpp	2026-05-20 15:31:21.053985092 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGCPSRethreadingPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGCPSRethreadingPhase.h	2026-01-20 14:08:31.836707400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCPSRethreadingPhase.h	2026-05-20 15:31:21.076985296 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGCriticalEdgeBreakingPhase.cpp	2026-01-20 14:08:31.845707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCriticalEdgeBreakingPhase.cpp	2026-05-20 15:31:21.097985483 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGCriticalEdgeBreakingPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGCriticalEdgeBreakingPhase.h	2026-01-20 14:08:31.846707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCriticalEdgeBreakingPhase.h	2026-05-20 15:31:21.119985679 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGCSEPhase.cpp	2026-02-23 14:40:52.282317900 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCSEPhase.cpp	2026-05-20 15:31:21.140985866 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGCSEPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGCSEPhase.h	2026-01-20 14:08:31.838707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGCSEPhase.h	2026-05-20 15:31:21.161986053 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCommon.h"
--- a/Source/JavaScriptCore/dfg/DFGDCEPhase.cpp	2026-01-20 14:08:31.847707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDCEPhase.cpp	2026-05-20 15:31:21.182986239 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGDCEPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGDCEPhase.h	2026-01-20 14:08:31.847707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDCEPhase.h	2026-05-20 15:31:21.204986435 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCommon.h"
--- a/Source/JavaScriptCore/dfg/DFGDesiredGlobalProperties.cpp	2026-01-20 14:08:31.848707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDesiredGlobalProperties.cpp	2026-05-20 15:31:21.224986613 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGDesiredGlobalProperties.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGDesiredGlobalProperties.h	2026-02-23 14:40:52.284318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDesiredGlobalProperties.h	2026-05-20 15:31:21.246986809 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGDesiredGlobalProperty.h"
--- a/Source/JavaScriptCore/dfg/DFGDesiredGlobalProperty.h	2026-02-23 14:40:52.284318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDesiredGlobalProperty.h	2026-05-20 15:31:21.267986996 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DumpContext.h"
--- a/Source/JavaScriptCore/dfg/DFGDesiredIdentifiers.cpp	2026-02-23 14:40:52.284318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDesiredIdentifiers.cpp	2026-05-20 15:31:21.290987200 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGDesiredIdentifiers.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGDesiredIdentifiers.h	2026-01-20 14:08:31.850707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDesiredIdentifiers.h	2026-05-20 15:31:21.310987378 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "Identifier.h"
--- a/Source/JavaScriptCore/dfg/DFGDesiredTransitions.cpp	2026-02-23 14:40:52.284318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDesiredTransitions.cpp	2026-05-20 15:31:21.331987565 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGDesiredTransitions.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGDesiredTransitions.h	2026-01-20 14:08:31.850707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDesiredTransitions.h	2026-05-20 15:31:21.352987752 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <wtf/Vector.h>
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGDesiredWatchpoints.cpp	2026-02-23 14:40:52.284318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDesiredWatchpoints.cpp	2026-05-20 15:31:21.372987930 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGDesiredWatchpoints.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGDesiredWatchpoints.h	2026-02-23 14:40:52.284318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDesiredWatchpoints.h	2026-05-20 15:31:21.394988126 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCommonData.h"
--- a/Source/JavaScriptCore/dfg/DFGDesiredWeakReferences.cpp	2026-02-23 14:40:52.284318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDesiredWeakReferences.cpp	2026-05-20 15:31:21.414988303 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGDesiredWeakReferences.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGDesiredWeakReferences.h	2026-01-20 14:08:31.851707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDesiredWeakReferences.h	2026-05-20 15:31:21.435988490 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "StructureID.h"
--- a/Source/JavaScriptCore/dfg/DFGDisassembler.cpp	2026-01-20 14:08:31.852707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDisassembler.cpp	2026-05-20 15:31:21.455988668 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGDisassembler.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGDisassembler.h	2026-01-20 14:08:31.852707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDisassembler.h	2026-05-20 15:31:21.474988837 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "CodeOrigin.h"
--- a/Source/JavaScriptCore/dfg/DFGDoesGCCheck.cpp	2026-04-09 08:23:11.604884000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDoesGCCheck.cpp	2026-05-20 15:31:21.494989015 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGDoesGCCheck.h"
 
 #include "CallFrameInlines.h"
--- a/Source/JavaScriptCore/dfg/DFGDoesGCCheck.h	2026-04-09 08:23:11.604884000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDoesGCCheck.h	2026-05-20 15:31:21.515989202 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <JavaScriptCore/JSExportMacros.h>
 #include <cstdint>
 #include <wtf/Assertions.h>
--- a/Source/JavaScriptCore/dfg/DFGDoesGC.cpp	2026-02-23 14:40:52.285318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDoesGC.cpp	2026-05-20 15:31:21.538989407 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGDoesGC.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGDoesGC.h	2026-01-20 14:08:31.853707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDoesGC.h	2026-05-20 15:31:21.557989576 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGDominators.h	2026-02-23 14:40:52.285318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDominators.h	2026-05-20 15:31:21.580989780 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCFG.h"
--- a/Source/JavaScriptCore/dfg/DFGDriver.cpp	2026-02-23 14:40:52.285318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGDriver.cpp	2026-05-20 15:31:21.600989958 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGDriver.h"
 
 #include "CodeBlock.h"
--- a/Source/JavaScriptCore/dfg/DFGEdge.cpp	2026-01-20 14:08:31.855707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGEdge.cpp	2026-05-20 15:31:21.621990145 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGEdge.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGEdgeDominates.h	2026-01-20 14:08:31.856707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGEdgeDominates.h	2026-05-20 15:31:21.639990305 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGGraph.h"
--- a/Source/JavaScriptCore/dfg/DFGEdge.h	2026-02-23 14:40:52.285318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGEdge.h	2026-05-20 15:31:21.662990510 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCommon.h"
--- a/Source/JavaScriptCore/dfg/DFGEpoch.cpp	2026-01-20 14:08:31.856707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGEpoch.cpp	2026-05-20 15:31:21.682990688 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGEpoch.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGEpoch.h	2026-01-20 14:08:31.857707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGEpoch.h	2026-05-20 15:31:21.703990875 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include <wtf/PrintStream.h>
--- a/Source/JavaScriptCore/dfg/DFGFailedFinalizer.cpp	2026-01-20 14:08:31.857707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFailedFinalizer.cpp	2026-05-20 15:31:21.724991061 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGFailedFinalizer.h"
 #include <wtf/TZoneMallocInlines.h>
 
--- a/Source/JavaScriptCore/dfg/DFGFailedFinalizer.h	2026-01-20 14:08:31.857707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFailedFinalizer.h	2026-05-20 15:31:21.743991230 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGFinalizer.h"
--- a/Source/JavaScriptCore/dfg/DFGFiltrationResult.h	2026-01-20 14:08:31.857707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFiltrationResult.h	2026-05-20 15:31:21.764991417 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGFinalizer.cpp	2026-01-20 14:08:31.858707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFinalizer.cpp	2026-05-20 15:31:21.782991577 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGFinalizer.h"
 #include "JITCode.h"
 #include <wtf/TZoneMallocInlines.h>
--- a/Source/JavaScriptCore/dfg/DFGFinalizer.h	2026-01-20 14:08:31.858707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFinalizer.h	2026-05-20 15:31:21.801991746 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include <wtf/Noncopyable.h>
--- a/Source/JavaScriptCore/dfg/DFGFixupPhase.cpp	2026-02-23 14:40:52.286317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFixupPhase.cpp	2026-05-20 15:31:21.822991933 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGFixupPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGFixupPhase.h	2026-01-20 14:08:31.861707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFixupPhase.h	2026-05-20 15:31:21.844992129 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGFlowIndexing.cpp	2026-01-20 14:08:31.861707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFlowIndexing.cpp	2026-05-20 15:31:21.863992298 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGFlowIndexing.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGFlowIndexing.h	2026-01-20 14:08:31.861707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFlowIndexing.h	2026-05-20 15:31:21.884992485 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGGraph.h"
--- a/Source/JavaScriptCore/dfg/DFGFlowMap.h	2026-01-20 14:08:31.862707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFlowMap.h	2026-05-20 15:31:21.903992654 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGFlowIndexing.h"
--- a/Source/JavaScriptCore/dfg/DFGFlushedAt.cpp	2026-01-20 14:08:31.864707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFlushedAt.cpp	2026-05-20 15:31:21.921992814 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGFlushedAt.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGFlushedAt.h	2026-01-20 14:08:31.865707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFlushedAt.h	2026-05-20 15:31:21.941992992 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGFlushFormat.h"
--- a/Source/JavaScriptCore/dfg/DFGFlushFormat.cpp	2026-01-20 14:08:31.863707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFlushFormat.cpp	2026-05-20 15:31:21.962993179 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGFlushFormat.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGFlushFormat.h	2026-01-20 14:08:31.863707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFlushFormat.h	2026-05-20 15:31:21.984993375 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGNodeFlags.h"
--- a/Source/JavaScriptCore/dfg/DFGForAllKills.h	2026-01-20 14:08:31.865707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGForAllKills.h	2026-05-20 15:31:22.003993544 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCombinedLiveness.h"
--- a/Source/JavaScriptCore/dfg/DFGFrozenValue.cpp	2026-01-20 14:08:31.865707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFrozenValue.cpp	2026-05-20 15:31:22.026993748 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGFrozenValue.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGFrozenValue.h	2026-01-20 14:08:31.865707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGFrozenValue.h	2026-05-20 15:31:22.047993935 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGValueStrength.h"
--- a/Source/JavaScriptCore/dfg/DFGGenerationInfo.h	2026-01-20 14:08:31.865707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGGenerationInfo.h	2026-05-20 15:31:22.067994113 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGMinifiedIDInlines.h"
--- a/Source/JavaScriptCore/dfg/DFGGraph.cpp	2026-02-23 14:40:52.286317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGGraph.cpp	2026-05-20 15:31:22.090994318 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGGraph.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGGraph.h	2026-02-23 14:40:52.287317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGGraph.h	2026-05-20 15:31:22.111994505 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "AssemblyHelpers.h"
--- a/Source/JavaScriptCore/dfg/DFGGraphSafepoint.cpp	2026-01-20 14:08:31.866707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGGraphSafepoint.cpp	2026-05-20 15:31:22.131994683 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGGraphSafepoint.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGGraphSafepoint.h	2026-01-20 14:08:31.867707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGGraphSafepoint.h	2026-05-20 15:31:22.151994860 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "JITSafepoint.h"
--- a/Source/JavaScriptCore/dfg/DFGHeapLocation.cpp	2026-01-20 14:08:31.867707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGHeapLocation.cpp	2026-05-20 15:31:22.172995047 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGHeapLocation.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGHeapLocation.h	2026-02-23 14:40:52.287317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGHeapLocation.h	2026-05-20 15:31:22.194995243 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGAbstractHeap.h"
--- a/Source/JavaScriptCore/dfg/DFGInlineCacheWrapper.h	2026-01-20 14:08:31.868707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGInlineCacheWrapper.h	2026-05-20 15:31:22.214995421 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "JITInlineCacheGenerator.h"
--- a/Source/JavaScriptCore/dfg/DFGInlineCacheWrapperInlines.h	2026-01-20 14:08:31.870707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGInlineCacheWrapperInlines.h	2026-05-20 15:31:22.235995608 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGInlineCacheWrapper.h"
--- a/Source/JavaScriptCore/dfg/DFGInPlaceAbstractState.cpp	2026-01-20 14:08:31.867707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGInPlaceAbstractState.cpp	2026-05-20 15:31:22.256995795 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGInPlaceAbstractState.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGInPlaceAbstractState.h	2026-02-23 14:40:52.287317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGInPlaceAbstractState.h	2026-05-20 15:31:22.280996008 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGAbstractInterpreterClobberState.h"
--- a/Source/JavaScriptCore/dfg/DFGInsertionSet.cpp	2026-01-20 14:08:31.871707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGInsertionSet.cpp	2026-05-20 15:31:22.304996222 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGInsertionSet.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGInsertionSet.h	2026-01-20 14:08:31.872707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGInsertionSet.h	2026-05-20 15:31:22.325996409 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGGraph.h"
--- a/Source/JavaScriptCore/dfg/DFGIntegerCheckCombiningPhase.cpp	2026-02-23 14:40:52.287317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGIntegerCheckCombiningPhase.cpp	2026-05-20 15:31:22.347996604 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGIntegerCheckCombiningPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGIntegerCheckCombiningPhase.h	2026-01-20 14:08:31.873707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGIntegerCheckCombiningPhase.h	2026-05-20 15:31:22.366996773 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGIntegerRangeOptimizationPhase.cpp	2026-02-23 14:40:52.288318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGIntegerRangeOptimizationPhase.cpp	2026-05-20 15:31:22.386996951 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGIntegerRangeOptimizationPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGIntegerRangeOptimizationPhase.h	2026-01-20 14:08:31.874707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGIntegerRangeOptimizationPhase.h	2026-05-20 15:31:22.405997120 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGInvalidationPointInjectionPhase.cpp	2026-01-20 14:08:31.875707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGInvalidationPointInjectionPhase.cpp	2026-05-20 15:31:22.426997307 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGInvalidationPointInjectionPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGInvalidationPointInjectionPhase.h	2026-01-20 14:08:31.875707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGInvalidationPointInjectionPhase.h	2026-05-20 15:31:22.446997485 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGJITCode.cpp	2026-02-23 14:40:52.288318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGJITCode.cpp	2026-05-20 15:31:22.467997672 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGJITCode.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGJITCode.h	2026-02-23 14:40:52.288318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGJITCode.h	2026-05-20 15:31:22.487997850 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "CodeBlock.h"
--- a/Source/JavaScriptCore/dfg/DFGJITCompiler.cpp	2026-02-23 14:40:52.288318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGJITCompiler.cpp	2026-05-20 15:31:22.508998037 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGJITCompiler.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGJITCompiler.h	2026-01-20 14:08:31.878707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGJITCompiler.h	2026-05-20 15:31:22.529998224 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/dfg/DFGJITFinalizer.cpp	2026-02-23 14:40:52.288318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGJITFinalizer.cpp	2026-05-20 15:31:22.549998402 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGJITFinalizer.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGJITFinalizer.h	2026-01-20 14:08:31.878707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGJITFinalizer.h	2026-05-20 15:31:22.571998598 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGFinalizer.h"
--- a/Source/JavaScriptCore/dfg/DFGJumpReplacement.cpp	2026-01-20 14:08:31.879707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGJumpReplacement.cpp	2026-05-20 15:31:22.593998793 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGJumpReplacement.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGJumpReplacement.h	2026-01-20 14:08:31.880707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGJumpReplacement.h	2026-05-20 15:31:22.612998962 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "CodeLocation.h"
--- a/Source/JavaScriptCore/dfg/DFGLazyJSValue.cpp	2026-01-20 14:08:31.881707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLazyJSValue.cpp	2026-05-20 15:31:22.633999149 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGLazyJSValue.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGLazyJSValue.h	2026-01-20 14:08:31.881707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLazyJSValue.h	2026-05-20 15:31:22.654999336 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCommon.h"
--- a/Source/JavaScriptCore/dfg/DFGLazyNode.cpp	2026-01-20 14:08:31.881707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLazyNode.cpp	2026-05-20 15:31:22.675999523 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGLazyNode.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGLazyNode.h	2026-01-20 14:08:31.881707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLazyNode.h	2026-05-20 15:31:22.698999728 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCommon.h"
--- a/Source/JavaScriptCore/dfg/DFGLICMPhase.cpp	2026-01-20 14:08:31.880707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLICMPhase.cpp	2026-05-20 15:31:22.717999897 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGLICMPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGLICMPhase.h	2026-01-20 14:08:31.880707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLICMPhase.h	2026-05-20 15:31:22.740000092 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGLiveCatchVariablePreservationPhase.cpp	2026-01-20 14:08:31.882707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLiveCatchVariablePreservationPhase.cpp	2026-05-20 15:31:22.759000261 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGLiveCatchVariablePreservationPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGLivenessAnalysisPhase.cpp	2026-01-20 14:08:31.882707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLivenessAnalysisPhase.cpp	2026-05-20 15:31:22.781000457 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGLivenessAnalysisPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGLivenessAnalysisPhase.h	2026-01-20 14:08:31.882707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLivenessAnalysisPhase.h	2026-05-20 15:31:22.800000626 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 
--- a/Source/JavaScriptCore/dfg/DFGLoopPreHeaderCreationPhase.cpp	2026-01-20 14:08:31.883707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLoopPreHeaderCreationPhase.cpp	2026-05-20 15:31:22.821000813 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGLoopPreHeaderCreationPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGLoopPreHeaderCreationPhase.h	2026-01-20 14:08:31.883707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLoopPreHeaderCreationPhase.h	2026-05-20 15:31:22.841000991 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGLoopUnrollingPhase.cpp	2026-02-23 14:40:52.289317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLoopUnrollingPhase.cpp	2026-05-20 15:31:22.863001187 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGLoopUnrollingPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGLoopUnrollingPhase.h	2026-01-20 14:08:31.884707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGLoopUnrollingPhase.h	2026-05-20 15:31:22.883001365 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGMayExit.cpp	2026-02-23 14:40:52.289317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGMayExit.cpp	2026-05-20 15:31:22.904001552 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGMayExit.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGMayExit.h	2026-01-20 14:08:31.884707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGMayExit.h	2026-05-20 15:31:22.924001729 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGMinifiedGraph.cpp	2026-02-23 14:40:52.289317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGMinifiedGraph.cpp	2026-05-20 15:31:22.945001916 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGMinifiedGraph.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGMinifiedGraph.h	2026-01-20 14:08:31.885707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGMinifiedGraph.h	2026-05-20 15:31:22.967002112 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGMinifiedNode.h"
--- a/Source/JavaScriptCore/dfg/DFGMinifiedIDInlines.h	2026-01-20 14:08:31.885707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGMinifiedIDInlines.h	2026-05-20 15:31:22.988002299 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGMinifiedID.h"
--- a/Source/JavaScriptCore/dfg/DFGMinifiedNode.cpp	2026-01-20 14:08:31.886707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGMinifiedNode.cpp	2026-05-20 15:31:23.009002486 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGMinifiedNode.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGMinifiedNode.h	2026-01-20 14:08:31.887707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGMinifiedNode.h	2026-05-20 15:31:23.030002673 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGMinifiedID.h"
--- a/Source/JavaScriptCore/dfg/DFGMovHintRemovalPhase.cpp	2026-01-20 14:08:31.888707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGMovHintRemovalPhase.cpp	2026-05-20 15:31:23.051002860 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGMovHintRemovalPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGMovHintRemovalPhase.h	2026-01-20 14:08:31.888707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGMovHintRemovalPhase.h	2026-05-20 15:31:23.071003037 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGMultiGetByOffsetData.cpp	2026-01-20 14:08:31.888707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGMultiGetByOffsetData.cpp	2026-05-20 15:31:23.091003215 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGMultiGetByOffsetData.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGMultiGetByOffsetData.h	2026-03-03 21:30:17.008498400 +0000
+++ b/Source/JavaScriptCore/dfg/DFGMultiGetByOffsetData.h	2026-05-20 15:31:23.112003402 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGRegisteredStructureSet.h"
--- a/Source/JavaScriptCore/dfg/DFGNaturalLoops.h	2026-02-23 14:40:52.290317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNaturalLoops.h	2026-05-20 15:31:23.134003598 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGDominators.h"
--- a/Source/JavaScriptCore/dfg/DFGNodeAbstractValuePair.cpp	2026-01-20 14:08:31.890707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNodeAbstractValuePair.cpp	2026-05-20 15:31:23.156003794 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGNodeAbstractValuePair.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGNodeAbstractValuePair.h	2026-01-20 14:08:31.890707300 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNodeAbstractValuePair.h	2026-05-20 15:31:23.176003972 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGAbstractValue.h"
--- a/Source/JavaScriptCore/dfg/DFGNode.cpp	2026-02-23 14:40:52.290317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNode.cpp	2026-05-20 15:31:23.197004159 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGNode.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGNodeFlags.cpp	2026-01-20 14:08:31.891707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNodeFlags.cpp	2026-05-20 15:31:23.217004337 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGNodeFlags.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGNodeFlags.h	2026-01-20 14:08:31.891707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNodeFlags.h	2026-05-20 15:31:23.241004550 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include <wtf/PrintStream.h>
--- a/Source/JavaScriptCore/dfg/DFGNodeFlowProjection.cpp	2026-01-20 14:08:31.891707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNodeFlowProjection.cpp	2026-05-20 15:31:23.266004773 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGNodeFlowProjection.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGNodeFlowProjection.h	2026-02-23 14:40:52.290317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNodeFlowProjection.h	2026-05-20 15:31:23.286004951 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGNode.h"
--- a/Source/JavaScriptCore/dfg/DFGNode.h	2026-02-23 14:40:52.290317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNode.h	2026-05-20 15:31:23.308005146 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "B3SparseCollection.h"
--- a/Source/JavaScriptCore/dfg/DFGNodeOrigin.cpp	2026-01-20 14:08:31.892707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNodeOrigin.cpp	2026-05-20 15:31:23.327005316 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGNodeOrigin.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGNodeOrigin.h	2026-01-20 14:08:31.892707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNodeOrigin.h	2026-05-20 15:31:23.344005467 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "CodeOrigin.h"
--- a/Source/JavaScriptCore/dfg/DFGNodeType.h	2026-03-03 21:30:17.009498600 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNodeType.h	2026-05-20 15:31:23.366005663 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGNodeFlags.h"
--- a/Source/JavaScriptCore/dfg/DFGNullAbstractState.h	2026-01-20 14:08:31.892707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGNullAbstractState.h	2026-05-20 15:31:23.386005841 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGObjectAllocationSinkingPhase.cpp	2026-03-16 22:06:45.611290700 +0000
+++ b/Source/JavaScriptCore/dfg/DFGObjectAllocationSinkingPhase.cpp	2026-05-20 15:31:23.406006019 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGObjectAllocationSinkingPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGObjectAllocationSinkingPhase.h	2026-01-20 14:08:31.899707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGObjectAllocationSinkingPhase.h	2026-05-20 15:31:23.427006205 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGObjectMaterializationData.cpp	2026-01-20 14:08:31.900707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGObjectMaterializationData.cpp	2026-05-20 15:31:23.446006374 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGObjectMaterializationData.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGObjectMaterializationData.h	2026-02-23 14:40:52.292317900 +0000
+++ b/Source/JavaScriptCore/dfg/DFGObjectMaterializationData.h	2026-05-20 15:31:23.470006588 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGPromotedHeapLocation.h"
--- a/Source/JavaScriptCore/dfg/DFGOperations.cpp	2026-03-26 12:33:41.417457000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOperations.cpp	2026-05-20 15:31:23.492006784 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGOperations.h"
 
 #include "ArrayPrototypeInlines.h"
--- a/Source/JavaScriptCore/dfg/DFGOperations.h	2026-02-23 14:40:52.293317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOperations.h	2026-05-20 15:31:23.515006988 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGArithMode.h"
--- a/Source/JavaScriptCore/dfg/DFGOpInfo.h	2026-02-23 14:40:52.292317900 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOpInfo.h	2026-05-20 15:31:23.535007166 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CacheableIdentifier.h"
 #include "DFGRegisteredStructure.h"
 #include "HeapCell.h"
--- a/Source/JavaScriptCore/dfg/DFGOSRAvailabilityAnalysisPhase.cpp	2026-02-23 14:40:52.291318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSRAvailabilityAnalysisPhase.cpp	2026-05-20 15:31:23.557007362 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGOSRAvailabilityAnalysisPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGOSRAvailabilityAnalysisPhase.h	2026-01-20 14:08:31.893707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSRAvailabilityAnalysisPhase.h	2026-05-20 15:31:23.576007531 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGBasicBlock.h"
--- a/Source/JavaScriptCore/dfg/DFGOSREntry.cpp	2026-02-23 14:40:52.291318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSREntry.cpp	2026-05-20 15:31:23.598007727 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGOSREntry.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGOSREntry.h	2026-01-20 14:08:31.894707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSREntry.h	2026-05-20 15:31:23.620007923 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CodeLocation.h"
 #include "DFGAbstractValue.h"
 #include "DFGFlushFormat.h"
--- a/Source/JavaScriptCore/dfg/DFGOSREntrypointCreationPhase.cpp	2026-01-20 14:08:31.894707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSREntrypointCreationPhase.cpp	2026-05-20 15:31:23.641008110 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGOSREntrypointCreationPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGOSREntrypointCreationPhase.h	2026-01-20 14:08:31.894707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSREntrypointCreationPhase.h	2026-05-20 15:31:23.663008305 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGOSRExitBase.cpp	2026-01-20 14:08:31.895707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSRExitBase.cpp	2026-05-20 15:31:23.686008510 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGOSRExitBase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGOSRExitBase.h	2026-01-20 14:08:31.895707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSRExitBase.h	2026-05-20 15:31:23.710008724 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "CodeOrigin.h"
--- a/Source/JavaScriptCore/dfg/DFGOSRExitCompilationInfo.h	2026-01-20 14:08:31.896707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSRExitCompilationInfo.h	2026-05-20 15:31:23.733008928 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "MacroAssembler.h"
--- a/Source/JavaScriptCore/dfg/DFGOSRExitCompilerCommon.cpp	2026-01-20 14:08:31.897707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSRExitCompilerCommon.cpp	2026-05-20 15:31:23.754009115 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGOSRExitCompilerCommon.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGOSRExitCompilerCommon.h	2026-01-20 14:08:31.897707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSRExitCompilerCommon.h	2026-05-20 15:31:23.775009302 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "CCallHelpers.h"
--- a/Source/JavaScriptCore/dfg/DFGOSRExit.cpp	2026-02-23 14:40:52.291318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSRExit.cpp	2026-05-20 15:31:23.797009498 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGOSRExit.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGOSRExit.h	2026-01-20 14:08:31.895707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSRExit.h	2026-05-20 15:31:23.819009694 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGOSRExitBase.h"
--- a/Source/JavaScriptCore/dfg/DFGOSRExitJumpPlaceholder.cpp	2026-01-20 14:08:31.898707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSRExitJumpPlaceholder.cpp	2026-05-20 15:31:23.839009872 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGOSRExitJumpPlaceholder.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGOSRExitJumpPlaceholder.h	2026-01-20 14:08:31.898707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGOSRExitJumpPlaceholder.h	2026-05-20 15:31:23.861010067 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "MacroAssembler.h"
--- a/Source/JavaScriptCore/dfg/DFGPhantomInsertionPhase.cpp	2026-01-20 14:08:31.903707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPhantomInsertionPhase.cpp	2026-05-20 15:31:23.879010228 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGPhantomInsertionPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGPhantomInsertionPhase.h	2026-01-20 14:08:31.904707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPhantomInsertionPhase.h	2026-05-20 15:31:23.899010406 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGPhase.cpp	2026-01-20 14:08:31.904707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPhase.cpp	2026-05-20 15:31:23.917010566 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGPhase.h	2026-02-23 14:40:52.293317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPhase.h	2026-05-20 15:31:23.939010761 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "CompilerTimingScope.h"
--- a/Source/JavaScriptCore/dfg/DFGPhiChildren.cpp	2026-01-20 14:08:31.904707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPhiChildren.cpp	2026-05-20 15:31:23.961010957 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGPhiChildren.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGPhiChildren.h	2026-01-20 14:08:31.904707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPhiChildren.h	2026-05-20 15:31:23.984011162 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGNode.h"
--- a/Source/JavaScriptCore/dfg/DFGPlan.cpp	2026-02-23 14:40:52.293317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPlan.cpp	2026-05-20 15:31:24.006011358 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGPlan.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGPlan.h	2026-02-23 14:40:52.293317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPlan.h	2026-05-20 15:31:24.028011553 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include "CompilationResult.h"
 #include "DFGDesiredGlobalProperties.h"
 #include "DFGDesiredIdentifiers.h"
--- a/Source/JavaScriptCore/dfg/DFGPreciseLocalClobberize.h	2026-01-20 14:08:31.905707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPreciseLocalClobberize.h	2026-05-20 15:31:24.049011740 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGClobberize.h"
--- a/Source/JavaScriptCore/dfg/DFGPredictionInjectionPhase.cpp	2026-01-20 14:08:31.906707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPredictionInjectionPhase.cpp	2026-05-20 15:31:24.071011936 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGPredictionInjectionPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGPredictionInjectionPhase.h	2026-01-20 14:08:31.906707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPredictionInjectionPhase.h	2026-05-20 15:31:24.093012132 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGPredictionPropagationPhase.cpp	2026-02-23 14:40:52.293317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPredictionPropagationPhase.cpp	2026-05-20 15:31:24.114012319 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGPredictionPropagationPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGPredictionPropagationPhase.h	2026-01-20 14:08:31.906707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPredictionPropagationPhase.h	2026-05-20 15:31:24.135012506 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "SpeculatedType.h"
--- a/Source/JavaScriptCore/dfg/DFGPromotedHeapLocation.cpp	2026-01-20 14:08:31.907707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPromotedHeapLocation.cpp	2026-05-20 15:31:24.158012710 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGPromotedHeapLocation.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGPromotedHeapLocation.h	2026-02-23 14:40:52.293317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPromotedHeapLocation.h	2026-05-20 15:31:24.179012897 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGEdge.h"
--- a/Source/JavaScriptCore/dfg/DFGPropertyTypeKey.h	2026-02-23 14:40:52.293317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPropertyTypeKey.h	2026-05-20 15:31:24.202013102 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "Structure.h"
--- a/Source/JavaScriptCore/dfg/DFGPureValue.cpp	2026-01-20 14:08:31.907707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPureValue.cpp	2026-05-20 15:31:24.222013280 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGPureValue.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGPureValue.h	2026-02-23 14:40:52.294317700 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPureValue.h	2026-05-20 15:31:24.245013485 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGGraph.h"
--- a/Source/JavaScriptCore/dfg/DFGPutStackSinkingPhase.cpp	2026-02-23 14:40:52.294317700 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPutStackSinkingPhase.cpp	2026-05-20 15:31:24.265013663 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGPutStackSinkingPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGPutStackSinkingPhase.h	2026-01-20 14:08:31.909707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGPutStackSinkingPhase.h	2026-05-20 15:31:24.286013850 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGRegisterBank.h	2026-01-20 14:08:31.909707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGRegisterBank.h	2026-05-20 15:31:24.308014045 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCommon.h"
--- a/Source/JavaScriptCore/dfg/DFGRegisteredStructureSet.cpp	2026-01-20 14:08:31.909707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGRegisteredStructureSet.cpp	2026-05-20 15:31:24.329014232 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGRegisteredStructureSet.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGSafeToExecute.h	2026-02-23 14:40:52.294317700 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSafeToExecute.h	2026-05-20 15:31:24.349014410 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGGraph.h"
--- a/Source/JavaScriptCore/dfg/DFGSaneStringGetByValSlowPathGenerator.h	2026-01-20 14:08:31.912707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSaneStringGetByValSlowPathGenerator.h	2026-05-20 15:31:24.368014579 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGOperations.h"
--- a/Source/JavaScriptCore/dfg/DFGScoreBoard.h	2026-02-23 14:40:52.294317700 +0000
+++ b/Source/JavaScriptCore/dfg/DFGScoreBoard.h	2026-05-20 15:31:24.389014766 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGGraph.h"
--- a/Source/JavaScriptCore/dfg/DFGSilentRegisterSavePlan.h	2026-01-20 14:08:31.913707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSilentRegisterSavePlan.h	2026-05-20 15:31:24.409014944 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCommon.h"
--- a/Source/JavaScriptCore/dfg/DFGSlowPathGenerator.h	2026-02-23 14:40:52.295318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSlowPathGenerator.h	2026-05-20 15:31:24.430015131 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #include <wtf/Compiler.h>
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/dfg/DFGSnippetParams.cpp	2026-01-20 14:08:31.913707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSnippetParams.cpp	2026-05-20 15:31:24.452015327 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGSnippetParams.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGSnippetParams.h	2026-02-23 14:40:52.295318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSnippetParams.h	2026-05-20 15:31:24.475015532 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGSpeculativeJIT.h"
--- a/Source/JavaScriptCore/dfg/DFGSpeculativeJIT32_64.cpp	2026-02-23 14:40:52.297317700 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSpeculativeJIT32_64.cpp	2026-05-20 15:31:24.498015736 +0000
@@ -25,6 +25,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGSpeculativeJIT.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGSpeculativeJIT64.cpp	2026-02-23 14:40:52.298318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSpeculativeJIT64.cpp	2026-05-20 15:31:24.521015941 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGSpeculativeJIT.h"
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/dfg/DFGSpeculativeJIT.cpp	2026-02-23 14:40:52.296317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSpeculativeJIT.cpp	2026-05-20 15:31:24.545016155 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGSpeculativeJIT.h"
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/dfg/DFGSpeculativeJIT.h	2026-02-23 14:40:52.297317700 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSpeculativeJIT.h	2026-05-20 15:31:24.568016359 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "BlockDirectory.h"
--- a/Source/JavaScriptCore/dfg/DFGSSACalculator.cpp	2026-01-20 14:08:31.910707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSSACalculator.cpp	2026-05-20 15:31:24.589016546 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGSSACalculator.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGSSACalculator.h	2026-01-20 14:08:31.910707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSSACalculator.h	2026-05-20 15:31:24.608016715 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGDominators.h"
--- a/Source/JavaScriptCore/dfg/DFGSSAConversionPhase.cpp	2026-02-23 14:40:52.294317700 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSSAConversionPhase.cpp	2026-05-20 15:31:24.627016884 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGSSAConversionPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGSSAConversionPhase.h	2026-01-20 14:08:31.911707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSSAConversionPhase.h	2026-05-20 15:31:24.646017053 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGSSALoweringPhase.cpp	2026-01-20 14:08:31.912707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSSALoweringPhase.cpp	2026-05-20 15:31:24.665017222 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGSSALoweringPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGSSALoweringPhase.h	2026-01-20 14:08:31.912707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGSSALoweringPhase.h	2026-05-20 15:31:24.685017400 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGStackLayoutPhase.cpp	2026-02-23 14:40:52.298318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStackLayoutPhase.cpp	2026-05-20 15:31:24.707017596 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGStackLayoutPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGStackLayoutPhase.h	2026-01-20 14:08:31.920707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStackLayoutPhase.h	2026-05-20 15:31:24.728017783 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGStaticExecutionCountEstimationPhase.cpp	2026-01-20 14:08:31.920707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStaticExecutionCountEstimationPhase.cpp	2026-05-20 15:31:24.749017970 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGStaticExecutionCountEstimationPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGStaticExecutionCountEstimationPhase.h	2026-01-20 14:08:31.920707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStaticExecutionCountEstimationPhase.h	2026-05-20 15:31:24.770018157 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGStoreBarrierClusteringPhase.cpp	2026-02-23 14:40:52.298318000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStoreBarrierClusteringPhase.cpp	2026-05-20 15:31:24.790018335 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGStoreBarrierClusteringPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGStoreBarrierClusteringPhase.h	2026-01-20 14:08:31.920707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStoreBarrierClusteringPhase.h	2026-05-20 15:31:24.809018504 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGStoreBarrierInsertionPhase.cpp	2026-02-23 14:40:52.299317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStoreBarrierInsertionPhase.cpp	2026-05-20 15:31:24.830018691 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGStoreBarrierInsertionPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGStoreBarrierInsertionPhase.h	2026-01-20 14:08:31.921707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStoreBarrierInsertionPhase.h	2026-05-20 15:31:24.852018887 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGStrengthReductionPhase.cpp	2026-02-23 14:40:52.299317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStrengthReductionPhase.cpp	2026-05-20 15:31:24.874019082 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGStrengthReductionPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGStrengthReductionPhase.h	2026-01-20 14:08:31.922707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStrengthReductionPhase.h	2026-05-20 15:31:24.897019287 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGStructureAbstractValue.cpp	2026-02-23 14:40:52.299317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStructureAbstractValue.cpp	2026-05-20 15:31:24.919019483 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGStructureAbstractValue.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGStructureAbstractValue.h	2026-01-20 14:08:31.923707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStructureAbstractValue.h	2026-05-20 15:31:24.937019643 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGRegisteredStructureSet.h"
--- a/Source/JavaScriptCore/dfg/DFGStructureClobberState.h	2026-01-20 14:08:31.923707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGStructureClobberState.h	2026-05-20 15:31:24.957019821 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include <wtf/PrintStream.h>
--- a/Source/JavaScriptCore/dfg/DFGThunks.cpp	2026-01-20 14:08:31.924707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGThunks.cpp	2026-05-20 15:31:24.978020008 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGThunks.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGThunks.h	2026-01-20 14:08:31.924707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGThunks.h	2026-05-20 15:31:24.998020186 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "MacroAssemblerCodeRef.h"
--- a/Source/JavaScriptCore/dfg/DFGTierUpCheckInjectionPhase.cpp	2026-02-23 14:40:52.299317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGTierUpCheckInjectionPhase.cpp	2026-05-20 15:31:25.017020355 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGTierUpCheckInjectionPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGTierUpCheckInjectionPhase.h	2026-01-20 14:08:31.924707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGTierUpCheckInjectionPhase.h	2026-05-20 15:31:25.038020542 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGToFTLDeferredCompilationCallback.cpp	2026-01-20 14:08:31.924707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGToFTLDeferredCompilationCallback.cpp	2026-05-20 15:31:25.058020720 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGToFTLDeferredCompilationCallback.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/dfg/DFGToFTLDeferredCompilationCallback.h	2026-01-20 14:08:31.924707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGToFTLDeferredCompilationCallback.h	2026-05-20 15:31:25.079020907 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "DeferredCompilationCallback.h"
--- a/Source/JavaScriptCore/dfg/DFGToFTLForOSREntryDeferredCompilationCallback.cpp	2026-01-20 14:08:31.924707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGToFTLForOSREntryDeferredCompilationCallback.cpp	2026-05-20 15:31:25.100021094 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGToFTLForOSREntryDeferredCompilationCallback.h"
 
 #if ENABLE(FTL_JIT)
--- a/Source/JavaScriptCore/dfg/DFGToFTLForOSREntryDeferredCompilationCallback.h	2026-01-20 14:08:31.924707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGToFTLForOSREntryDeferredCompilationCallback.h	2026-05-20 15:31:25.122021290 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(FTL_JIT)
 
 #include "DFGJITCode.h"
--- a/Source/JavaScriptCore/dfg/DFGTransition.cpp	2026-01-20 14:08:31.925707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGTransition.cpp	2026-05-20 15:31:25.142021468 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGTransition.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGTransition.h	2026-01-20 14:08:31.925707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGTransition.h	2026-05-20 15:31:25.162021646 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGRegisteredStructure.h"
--- a/Source/JavaScriptCore/dfg/DFGTypeCheckHoistingPhase.cpp	2026-02-23 14:40:52.299317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGTypeCheckHoistingPhase.cpp	2026-05-20 15:31:25.181021815 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGTypeCheckHoistingPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGTypeCheckHoistingPhase.h	2026-01-20 14:08:31.926707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGTypeCheckHoistingPhase.h	2026-05-20 15:31:25.203022011 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGTZoneImpls.cpp	2026-01-20 14:08:31.923707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGTZoneImpls.cpp	2026-05-20 15:31:25.223022189 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(DFG_JIT)
 
--- a/Source/JavaScriptCore/dfg/DFGUnificationPhase.cpp	2026-01-20 14:08:31.928707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGUnificationPhase.cpp	2026-05-20 15:31:25.241022349 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGUnificationPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGUnificationPhase.h	2026-01-20 14:08:31.928707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGUnificationPhase.h	2026-05-20 15:31:25.264022554 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGUseKind.cpp	2026-02-23 14:40:52.299317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGUseKind.cpp	2026-05-20 15:31:25.286022749 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGUseKind.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGUseKind.h	2026-02-23 14:40:52.300317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGUseKind.h	2026-05-20 15:31:25.305022918 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGNodeFlags.h"
--- a/Source/JavaScriptCore/dfg/DFGValidate.cpp	2026-02-23 14:40:52.300317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGValidate.cpp	2026-05-20 15:31:25.324023088 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGValidate.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGValidate.h	2026-01-20 14:08:31.929707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGValidate.h	2026-05-20 15:31:25.345023274 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGGraph.h"
--- a/Source/JavaScriptCore/dfg/DFGValidateUnlinked.cpp	2026-01-20 14:08:31.930707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGValidateUnlinked.cpp	2026-05-20 15:31:25.366023461 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGValidateUnlinked.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGValidateUnlinked.h	2026-01-20 14:08:31.930707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGValidateUnlinked.h	2026-05-20 15:31:25.388023657 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCapabilities.h"
--- a/Source/JavaScriptCore/dfg/DFGValueRepReductionPhase.cpp	2026-03-25 16:38:32.892259600 +0000
+++ b/Source/JavaScriptCore/dfg/DFGValueRepReductionPhase.cpp	2026-05-20 15:31:25.409023844 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGValueRepReductionPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGValueRepReductionPhase.h	2026-01-20 14:08:31.931707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGValueRepReductionPhase.h	2026-05-20 15:31:25.432024049 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGValueSource.cpp	2026-01-20 14:08:31.931707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGValueSource.cpp	2026-05-20 15:31:25.452024227 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGValueSource.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGValueSource.h	2026-01-20 14:08:31.931707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGValueSource.h	2026-05-20 15:31:25.471024396 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGFlushFormat.h"
--- a/Source/JavaScriptCore/dfg/DFGValueStrength.cpp	2026-01-20 14:08:31.931707100 +0000
+++ b/Source/JavaScriptCore/dfg/DFGValueStrength.cpp	2026-05-20 15:31:25.492024583 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGValueStrength.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGValueStrength.h	2026-01-20 14:08:31.932707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGValueStrength.h	2026-05-20 15:31:25.512024761 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include <wtf/PrintStream.h>
--- a/Source/JavaScriptCore/dfg/DFGVarargsForwardingPhase.cpp	2026-01-20 14:08:31.932707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGVarargsForwardingPhase.cpp	2026-05-20 15:31:25.533024948 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGVarargsForwardingPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGVarargsForwardingPhase.h	2026-01-20 14:08:31.932707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGVarargsForwardingPhase.h	2026-05-20 15:31:25.553025126 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 namespace JSC { namespace DFG {
--- a/Source/JavaScriptCore/dfg/DFGVariableAccessData.cpp	2026-01-20 14:08:31.933707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGVariableAccessData.cpp	2026-05-20 15:31:25.574025313 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGVariableAccessData.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGVariableAccessDataDump.cpp	2026-01-20 14:08:31.933707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGVariableAccessDataDump.cpp	2026-05-20 15:31:25.592025473 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGVariableAccessDataDump.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGVariableAccessDataDump.h	2026-01-20 14:08:31.933707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGVariableAccessDataDump.h	2026-05-20 15:31:25.612025651 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include <wtf/PrintStream.h>
--- a/Source/JavaScriptCore/dfg/DFGVariableAccessData.h	2026-01-20 14:08:31.933707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGVariableAccessData.h	2026-05-20 15:31:25.632025829 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGCommon.h"
--- a/Source/JavaScriptCore/dfg/DFGVariableEvent.cpp	2026-01-20 14:08:31.933707000 +0000
+++ b/Source/JavaScriptCore/dfg/DFGVariableEvent.cpp	2026-05-20 15:31:25.650025989 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGVariableEvent.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGVariableEvent.h	2026-02-23 14:40:52.300317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGVariableEvent.h	2026-05-20 15:31:25.669026158 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGMinifiedID.h"
--- a/Source/JavaScriptCore/dfg/DFGVariableEventStream.cpp	2026-01-20 14:08:31.934707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGVariableEventStream.cpp	2026-05-20 15:31:25.689026336 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGVariableEventStream.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGVariableEventStream.h	2026-02-23 14:40:52.300317800 +0000
+++ b/Source/JavaScriptCore/dfg/DFGVariableEventStream.h	2026-05-20 15:31:25.710026523 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGMinifiedGraph.h"
--- a/Source/JavaScriptCore/dfg/DFGVirtualRegisterAllocationPhase.cpp	2026-01-20 14:08:31.934707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGVirtualRegisterAllocationPhase.cpp	2026-05-20 15:31:25.731026710 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "DFGVirtualRegisterAllocationPhase.h"
 
 #if ENABLE(DFG_JIT)
--- a/Source/JavaScriptCore/dfg/DFGVirtualRegisterAllocationPhase.h	2026-01-20 14:08:31.934707200 +0000
+++ b/Source/JavaScriptCore/dfg/DFGVirtualRegisterAllocationPhase.h	2026-05-20 15:31:25.752026897 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(DFG_JIT)
 
 #include "DFGPhase.h"
--- a/Source/JavaScriptCore/dfg/testdfg.cpp	2026-01-20 14:08:31.934707200 +0000
+++ b/Source/JavaScriptCore/dfg/testdfg.cpp	2026-05-20 15:31:25.772027075 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #include "JSCJSValueInlines.h"
 // The above are needed before DFGAbstractValue.h
