--- a/Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorGlib.cpp	2026-02-23 14:40:52.342317600 +0000
+++ b/Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorGlib.cpp	2026-05-20 16:01:30.770710970 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "RemoteInspector.h"
 
 #if ENABLE(REMOTE_INSPECTOR)
--- a/Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorServer.cpp	2026-02-23 14:40:52.342317600 +0000
+++ b/Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorServer.cpp	2026-05-20 16:01:30.796711214 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "RemoteInspectorServer.h"
 
 #if ENABLE(REMOTE_INSPECTOR)
--- a/Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorServer.h	2026-01-20 14:08:32.097985500 +0000
+++ b/Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorServer.h	2026-05-20 16:01:30.830711532 +0000
@@ -25,6 +25,9 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(REMOTE_INSPECTOR)
 
 #include "RemoteInspector.h"
--- a/Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorUtils.cpp	2026-02-23 14:40:52.342317600 +0000
+++ b/Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorUtils.cpp	2026-05-20 16:01:30.851711728 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "RemoteInspectorUtils.h"
 
 #if ENABLE(REMOTE_INSPECTOR)
--- a/Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorUtils.h	2026-01-20 14:08:32.097985500 +0000
+++ b/Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorUtils.h	2026-05-20 16:01:30.885712046 +0000
@@ -25,6 +25,9 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(REMOTE_INSPECTOR)
 
 #include <wtf/glib/GRefPtr.h>
