--- a/Source/WebCore/PAL/pal/crypto/CryptoDigest.h
+++ b/Source/WebCore/PAL/pal/crypto/CryptoDigest.h
@@ -25,9 +25,13 @@
 #pragma once
 
 #include <pal/crypto/CryptoDigestHashFunction.h>
+#include <memory>
+#include <pal/ExportMacros.h>
+#include <span>
 #include <wtf/HexNumber.h>
 #include <wtf/Noncopyable.h>
 #include <wtf/Vector.h>
+#include <wtf/TZoneMalloc.h>
 #include <wtf/text/WTFString.h>
 
 namespace PAL {
--- a/Source/WebCore/PAL/pal/system/Clock.h
+++ b/Source/WebCore/PAL/pal/system/Clock.h
@@ -27,6 +27,7 @@
 #pragma once
 
 #include <memory>
+#include <wtf/TZoneMalloc.h>
 
 namespace PAL {
 
--- a/Source/WebCore/PAL/pal/system/ClockGeneric.h
+++ b/Source/WebCore/PAL/pal/system/ClockGeneric.h
@@ -28,6 +28,7 @@
 #pragma once
 
 #include "Clock.h"
+#include <wtf/TZoneMalloc.h>
 #include <wtf/MonotonicTime.h>
 
 namespace PAL {
--- a/Source/WebCore/PAL/pal/text/KillRing.h
+++ b/Source/WebCore/PAL/pal/text/KillRing.h
@@ -25,6 +25,7 @@
 #pragma once
 
 #include <wtf/TZoneMalloc.h>
+#include <pal/ExportMacros.h>
 #include <wtf/text/WTFString.h>
 
 namespace PAL {
--- a/Source/WebCore/PAL/pal/text/TextEncodingRegistry.h
+++ b/Source/WebCore/PAL/pal/text/TextEncodingRegistry.h
@@ -25,6 +25,8 @@
 #pragma once
 
 #include <memory>
+#include <pal/ExportMacros.h>
+#include <wtf/Platform.h>
 #include <wtf/Forward.h>
 
 #if PLATFORM(COCOA)
