--- a/Source/WebCore/PAL/pal/system/Sound.cpp
+++ b/Source/WebCore/PAL/pal/system/Sound.cpp
@@ -24,6 +24,8 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
+
 #include "Sound.h"
 
 #if !PLATFORM(MAC)
--- a/Source/WebCore/PAL/pal/system/Sound.h
+++ b/Source/WebCore/PAL/pal/system/Sound.h
@@ -24,6 +24,8 @@
  */
 
 #pragma once
+
+#include <pal/ExportMacros.h>
 
 namespace PAL {
 
--- a/Source/WebCore/PAL/pal/system/SleepDisabler.cpp
+++ b/Source/WebCore/PAL/pal/system/SleepDisabler.cpp
@@ -24,6 +24,8 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
+
 #include "SleepDisabler.h"
 
 #include <wtf/text/WTFString.h>
--- a/Source/WebCore/PAL/pal/system/SleepDisabler.h
+++ b/Source/WebCore/PAL/pal/system/SleepDisabler.h
@@ -26,6 +26,7 @@
 #pragma once
 
 #include <pal/ExportMacros.h>
+#include <memory>
 #include <wtf/Forward.h>
 
 namespace PAL {
--- a/Source/WebCore/PAL/pal/system/SystemSleepListener.cpp
+++ b/Source/WebCore/PAL/pal/system/SystemSleepListener.cpp
@@ -24,6 +24,8 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
+
 #include "SystemSleepListener.h"
 
 namespace PAL {
--- a/Source/WebCore/PAL/pal/system/SystemSleepListener.h
+++ b/Source/WebCore/PAL/pal/system/SystemSleepListener.h
@@ -24,6 +24,9 @@
  */
 
 #pragma once
+
+#include <memory>
+#include <pal/ExportMacros.h>
 
 namespace PAL {
 
