--- a/Source/JavaScriptCore/lol/LOLJIT.cpp	2026-02-23 14:40:52.376317500 +0000
+++ b/Source/JavaScriptCore/lol/LOLJIT.cpp	2026-05-20 16:23:21.593993899 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "LOLJIT.h"
 
 #if ENABLE(JIT) && USE(JSVALUE64)
--- a/Source/JavaScriptCore/lol/LOLJIT.h	2026-02-23 14:40:52.377558000 +0000
+++ b/Source/JavaScriptCore/lol/LOLJIT.h	2026-05-20 16:23:21.613994115 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT) && USE(JSVALUE64)
 
 #include "BaselineJITCode.h"
--- a/Source/JavaScriptCore/lol/LOLJITOperations.cpp	2026-02-23 14:40:52.377558000 +0000
+++ b/Source/JavaScriptCore/lol/LOLJITOperations.cpp	2026-05-20 16:23:21.634994342 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "LOLJITOperations.h"
 
 #if ENABLE(JIT) && USE(JSVALUE64)
--- a/Source/JavaScriptCore/lol/LOLJITOperations.h	2026-02-23 14:40:52.377558000 +0000
+++ b/Source/JavaScriptCore/lol/LOLJITOperations.h	2026-05-20 16:23:21.653994547 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT) && USE(JSVALUE64)
 
 #include "JITOperations.h"
--- a/Source/JavaScriptCore/lol/LOLRegisterAllocator.h	2026-02-23 14:40:52.377558000 +0000
+++ b/Source/JavaScriptCore/lol/LOLRegisterAllocator.h	2026-05-20 16:23:21.675994785 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <wtf/Platform.h>
+
 #if ENABLE(JIT) && USE(JSVALUE64)
 
 #include "BytecodeStructs.h"
