--- a/Source/WTF/wtf/glib/ActivityObserver.h
+++ b/Source/WTF/wtf/glib/ActivityObserver.h
@@ -18,6 +18,7 @@
  */
 
 #pragma once
+#include <wtf/Platform.h>
 
 #if USE(GLIB_EVENT_LOOP)
 
--- a/Source/WTF/wtf/glib/Application.h
+++ b/Source/WTF/wtf/glib/Application.h
@@ -18,6 +18,7 @@
  */
 
 #pragma once
+#include <wtf/Platform.h>
 
 #if USE(GLIB)
 
--- a/Source/WTF/wtf/glib/GMutexLocker.h
+++ b/Source/WTF/wtf/glib/GMutexLocker.h
@@ -19,6 +19,7 @@
 
 #ifndef GMutexLocker_h
 #define GMutexLocker_h
+#include <wtf/Platform.h>
 
 #if USE(GLIB)
 
--- a/Source/WTF/wtf/glib/GRefPtr.h
+++ b/Source/WTF/wtf/glib/GRefPtr.h
@@ -22,6 +22,7 @@
  */
 
 #pragma once
+#include <wtf/Platform.h>
 
 #if USE(GLIB)
 
--- a/Source/WTF/wtf/glib/GResources.h
+++ b/Source/WTF/wtf/glib/GResources.h
@@ -18,6 +18,7 @@
  */
 
 #pragma once
+#include <wtf/Platform.h>
 
 #if USE(GLIB)
 
--- a/Source/WTF/wtf/glib/GThreadSafeWeakPtr.h
+++ b/Source/WTF/wtf/glib/GThreadSafeWeakPtr.h
@@ -19,6 +19,7 @@
  */
 
 #pragma once
+#include <wtf/Platform.h>
 
 #if USE(GLIB)
 
--- a/Source/WTF/wtf/glib/GTypedefs.h
+++ b/Source/WTF/wtf/glib/GTypedefs.h
@@ -22,6 +22,7 @@
 
 /* Vanilla C code does not seem to be able to handle forward-declaration typedefs. */
 #ifdef  __cplusplus
+#include <wtf/Platform.h>
 
 typedef char gchar;
 typedef double gdouble;
--- a/Source/WTF/wtf/glib/GUniquePtr.h
+++ b/Source/WTF/wtf/glib/GUniquePtr.h
@@ -19,6 +19,7 @@
  */
 
 #pragma once
+#include <wtf/Platform.h>
 
 #if USE(GLIB)
 
--- a/Source/WTF/wtf/glib/GWeakPtr.h
+++ b/Source/WTF/wtf/glib/GWeakPtr.h
@@ -19,6 +19,7 @@
  */
 
 #pragma once
+#include <wtf/Platform.h>
 
 #if USE(GLIB)
 
--- a/Source/WTF/wtf/glib/RunLoopSourcePriority.h
+++ b/Source/WTF/wtf/glib/RunLoopSourcePriority.h
@@ -18,6 +18,7 @@
  */
 
 #pragma once
+#include <wtf/Platform.h>
 
 #if USE(GLIB)
 
--- a/Source/WTF/wtf/glib/Application.cpp
+++ b/Source/WTF/wtf/glib/Application.cpp
@@ -18,6 +18,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "Application.h"
 
 #if USE(GLIB)
--- a/Source/WTF/wtf/glib/FileSystemGlib.cpp
+++ b/Source/WTF/wtf/glib/FileSystemGlib.cpp
@@ -20,6 +20,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include <wtf/FileSystem.h>
 
 #include <wtf/glib/GUniquePtr.h>
--- a/Source/WTF/wtf/glib/GRefPtr.cpp
+++ b/Source/WTF/wtf/glib/GRefPtr.cpp
@@ -17,6 +17,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include <wtf/glib/GRefPtr.h>
 
 #if USE(GLIB)
--- a/Source/WTF/wtf/glib/GResources.cpp
+++ b/Source/WTF/wtf/glib/GResources.cpp
@@ -18,6 +18,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "GResources.h"
 
 #if USE(GLIB)
--- a/Source/WTF/wtf/glib/Sandbox.cpp
+++ b/Source/WTF/wtf/glib/Sandbox.cpp
@@ -23,6 +23,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include <wtf/glib/Sandbox.h>
 
 #include <gio/gio.h>
--- a/Source/WTF/wtf/glib/URLGLib.cpp
+++ b/Source/WTF/wtf/glib/URLGLib.cpp
@@ -23,6 +23,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include "URL.h"
 
 #if USE(GLIB)
