--- a/Source/JavaScriptCore/CMakeLists.txt
+++ b/Source/JavaScriptCore/CMakeLists.txt
@@ -1674,6 +1674,8 @@
 
 target_include_directories(LowLevelInterpreterLib
     PRIVATE "$<TARGET_PROPERTY:JavaScriptCore,INCLUDE_DIRECTORIES>" "${JavaScriptCore_PRIVATE_FRAMEWORK_HEADERS_DIR}/JavaScriptCore")
+target_compile_definitions(LowLevelInterpreterLib
+    PRIVATE "$<TARGET_PROPERTY:JavaScriptCore,COMPILE_DEFINITIONS>")
 add_dependencies(LowLevelInterpreterLib Bytecodes JSCBuiltins)
 if (TARGET WTF_CopyHeaders)
     add_dependencies(LowLevelInterpreterLib WTF_CopyHeaders)
