--- a/Source/WTF/wtf/unix/LoggingUnix.cpp
+++ b/Source/WTF/wtf/unix/LoggingUnix.cpp
@@ -20,6 +20,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if !OS(ANDROID)
 
--- a/Source/WTF/wtf/unix/MemoryPressureHandlerUnix.cpp
+++ b/Source/WTF/wtf/unix/MemoryPressureHandlerUnix.cpp
@@ -26,6 +26,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 #include <wtf/MemoryPressureHandler.h>
 
 #include <malloc.h>
--- a/Source/WTF/wtf/posix/FileHandlePOSIX.cpp
+++ b/Source/WTF/wtf/posix/FileHandlePOSIX.cpp
@@ -28,6 +28,7 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <sys/uio.h>
+#include <wtf/Platform.h>
 #include <unistd.h>
 #include <wtf/CheckedArithmetic.h>
 #include <wtf/FileSystem.h>
--- a/Source/WTF/wtf/posix/FileSystemPOSIX.cpp
+++ b/Source/WTF/wtf/posix/FileSystemPOSIX.cpp
@@ -39,6 +39,7 @@
 #include <sys/statvfs.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <wtf/Platform.h>
 #include <wtf/EnumTraits.h>
 #include <wtf/FileHandle.h>
 #include <wtf/MallocSpan.h>
--- a/Source/WTF/wtf/posix/OSAllocatorPOSIX.cpp
+++ b/Source/WTF/wtf/posix/OSAllocatorPOSIX.cpp
@@ -28,6 +28,7 @@
 
 #include <errno.h>
 #include <sys/mman.h>
+#include <wtf/Platform.h>
 #include <wtf/Assertions.h>
 #include <wtf/DataLog.h>
 #include <wtf/MallocSpan.h>
--- a/Source/WTF/wtf/posix/ThreadingPOSIX.cpp
+++ b/Source/WTF/wtf/posix/ThreadingPOSIX.cpp
@@ -31,6 +31,7 @@
 
 #include "config.h"
 #include <wtf/Threading.h>
+#include <wtf/Platform.h>
 
 #if USE(PTHREADS)
 
