--- a/Source/JavaScriptCore/API/glib/JSCContextInternal.h
+++ b/Source/JavaScriptCore/API/glib/JSCContextInternal.h
@@ -19,5 +19,6 @@
 
 #pragma once
 
+#include "JSExportMacros.h"
 #include "JSCContext.h"
 
--- a/Source/JavaScriptCore/API/glib/JSCContextPrivate.h
+++ b/Source/JavaScriptCore/API/glib/JSCContextPrivate.h
@@ -19,6 +19,7 @@
 
 #pragma once
 
+#include "JSExportMacros.h"
 #include "JSCContext.h"
 #include "JSCValue.h"
 #include <wtf/glib/GRefPtr.h>
--- a/Source/JavaScriptCore/API/glib/JSCValuePrivate.h
+++ b/Source/JavaScriptCore/API/glib/JSCValuePrivate.h
@@ -19,6 +19,7 @@
 
 #pragma once
 
+#include "JSExportMacros.h"
 #include "JSCValue.h"
 
