--- a/Source/JavaScriptCore/assembler/MacroAssemblerARM64.cpp
+++ b/Source/JavaScriptCore/assembler/MacroAssemblerARM64.cpp
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(ASSEMBLER) && CPU(ARM64)
 #include "MacroAssembler.h"
--- a/Source/JavaScriptCore/assembler/MacroAssemblerARMv7.cpp
+++ b/Source/JavaScriptCore/assembler/MacroAssemblerARMv7.cpp
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(ASSEMBLER) && CPU(ARM_THUMB2)
 #include "MacroAssembler.h"
--- a/Source/JavaScriptCore/assembler/MacroAssemblerX86_64.cpp
+++ b/Source/JavaScriptCore/assembler/MacroAssemblerX86_64.cpp
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <wtf/Platform.h>
 
 #if ENABLE(ASSEMBLER) && CPU(X86_64)
 #include "MacroAssembler.h"
