--- a/Source/JavaScriptCore/assembler/CPU.h
+++ b/Source/JavaScriptCore/assembler/CPU.h
@@ -24,6 +24,7 @@
  */
 
 #pragma once
+#include <cstddef>
 
 #include <JavaScriptCore/JSExportMacros.h>
 #include <wtf/NumberOfCores.h>
--- a/Source/JavaScriptCore/b3/B3AbstractHeap.h
+++ b/Source/JavaScriptCore/b3/B3AbstractHeap.h
@@ -24,6 +24,8 @@
  */
 
 #pragma once
+
+#include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
 
--- a/Source/JavaScriptCore/b3/B3AbstractHeapRepository.h
+++ b/Source/JavaScriptCore/b3/B3AbstractHeapRepository.h
@@ -24,6 +24,8 @@
  */
 
 #pragma once
+
+#include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
 
