--- a/Source/WebCore/platform/ScrollAnimationKeyboard.cpp
+++ b/Source/WebCore/platform/ScrollAnimationKeyboard.cpp
@@ -136,8 +136,8 @@ void ScrollAnimationKeyboard::stopKeyboardScrollAnimation()
     auto displacementMagnitudeSquared = (m_velocity * m_velocity).scaled(params.springMass / params.springStiffness);
     FloatPoint displacement = {
-        std::copysign(sqrt(displacementMagnitudeSquared.width()), m_velocity.width()),
-        std::copysign(sqrt(displacementMagnitudeSquared.height()), m_velocity.height())
+        (float)std::copysign(sqrt(displacementMagnitudeSquared.width()), m_velocity.width()),
+        (float)std::copysign(sqrt(displacementMagnitudeSquared.height()), m_velocity.height())
     };
