--- a/Source/JavaScriptCore/b3/air/AirArg.h	2026-02-23 14:40:52.240318000 +0000
+++ b/Source/JavaScriptCore/b3/air/AirArg.h	2026-05-20 15:05:02.325322637 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirOpcode.h"
--- a/Source/JavaScriptCore/b3/air/AirBasicBlock.h	2026-01-20 14:08:31.687707700 +0000
+++ b/Source/JavaScriptCore/b3/air/AirBasicBlock.h	2026-05-20 15:05:02.346322845 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirFrequentedBlock.h"
--- a/Source/JavaScriptCore/b3/air/AirCCallingConvention.h	2026-01-20 14:08:31.688707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCCallingConvention.h	2026-05-20 15:05:02.366323043 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirArg.h"
--- a/Source/JavaScriptCore/b3/air/AirCode.h	2026-02-23 14:40:52.241318200 +0000
+++ b/Source/JavaScriptCore/b3/air/AirCode.h	2026-05-20 15:05:02.387323251 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirArg.h"
--- a/Source/JavaScriptCore/b3/air/AirFormTable.h	2026-01-20 14:08:31.695707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirFormTable.h	2026-05-20 15:05:02.408323459 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirArg.h"
--- a/Source/JavaScriptCore/b3/air/AirGenerate.h	2026-01-20 14:08:31.695707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirGenerate.h	2026-05-20 15:05:02.428323658 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Common.h"
--- a/Source/JavaScriptCore/b3/air/AirSpecial.h	2026-01-20 14:08:31.707707600 +0000
+++ b/Source/JavaScriptCore/b3/air/AirSpecial.h	2026-05-20 15:05:02.447323846 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirInst.h"
--- a/Source/JavaScriptCore/b3/air/AirValidate.h	2026-01-20 14:08:31.712707500 +0000
+++ b/Source/JavaScriptCore/b3/air/AirValidate.h	2026-05-20 15:05:02.467324044 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 { namespace Air {
--- a/Source/JavaScriptCore/b3/B3ArgumentRegValue.h	2026-01-20 14:08:31.618708000 +0000
+++ b/Source/JavaScriptCore/b3/B3ArgumentRegValue.h	2026-05-20 15:05:02.489324262 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3AtomicValue.h	2026-02-23 14:40:52.229318100 +0000
+++ b/Source/JavaScriptCore/b3/B3AtomicValue.h	2026-05-20 15:05:02.513324500 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3BasicBlock.h	2026-01-20 14:08:31.621707700 +0000
+++ b/Source/JavaScriptCore/b3/B3BasicBlock.h	2026-05-20 15:05:02.535324718 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3BottomTupleValue.h	2026-01-20 14:08:31.623707800 +0000
+++ b/Source/JavaScriptCore/b3/B3BottomTupleValue.h	2026-05-20 15:05:02.556324926 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3BreakCriticalEdges.h	2026-01-20 14:08:31.623707800 +0000
+++ b/Source/JavaScriptCore/b3/B3BreakCriticalEdges.h	2026-05-20 15:05:02.583325194 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3BulkMemoryValue.h	2026-02-23 14:40:52.230318000 +0000
+++ b/Source/JavaScriptCore/b3/B3BulkMemoryValue.h	2026-05-20 15:05:02.607325431 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3CCallValue.h	2026-01-20 14:08:31.624707700 +0000
+++ b/Source/JavaScriptCore/b3/B3CCallValue.h	2026-05-20 15:05:02.632325679 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Effects.h"
--- a/Source/JavaScriptCore/b3/B3CheckValue.h	2026-01-20 14:08:31.626707800 +0000
+++ b/Source/JavaScriptCore/b3/B3CheckValue.h	2026-05-20 15:05:02.653325887 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3StackmapValue.h"
--- a/Source/JavaScriptCore/b3/B3Compile.h	2026-02-23 14:40:52.230318000 +0000
+++ b/Source/JavaScriptCore/b3/B3Compile.h	2026-05-20 15:05:02.674326095 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT)
--- a/Source/JavaScriptCore/b3/B3Const128Value.h	2026-01-20 14:08:31.630707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Const128Value.h	2026-05-20 15:05:02.695326304 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3Const32Value.h	2026-01-20 14:08:31.630707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Const32Value.h	2026-05-20 15:05:02.716326512 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3Const64Value.h	2026-01-20 14:08:31.630707700 +0000
+++ b/Source/JavaScriptCore/b3/B3Const64Value.h	2026-05-20 15:05:02.734326690 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3ConstDoubleValue.h	2026-01-20 14:08:31.631707700 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstDoubleValue.h	2026-05-20 15:05:02.757326918 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3ConstFloatValue.h	2026-01-20 14:08:31.631707700 +0000
+++ b/Source/JavaScriptCore/b3/B3ConstFloatValue.h	2026-05-20 15:05:02.778327126 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3Effects.h	2026-02-23 14:40:52.231318200 +0000
+++ b/Source/JavaScriptCore/b3/B3Effects.h	2026-05-20 15:05:02.799327334 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3HeapRange.h"
--- a/Source/JavaScriptCore/b3/B3EliminateDeadCode.h	2026-01-20 14:08:31.637707700 +0000
+++ b/Source/JavaScriptCore/b3/B3EliminateDeadCode.h	2026-05-20 15:05:02.819327532 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3ExtractValue.h	2026-01-20 14:08:31.639707800 +0000
+++ b/Source/JavaScriptCore/b3/B3ExtractValue.h	2026-05-20 15:05:02.840327741 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3FenceValue.h	2026-01-20 14:08:31.640707700 +0000
+++ b/Source/JavaScriptCore/b3/B3FenceValue.h	2026-05-20 15:05:02.860327939 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3HeapRange.h"
--- a/Source/JavaScriptCore/b3/B3FixSSA.h	2026-01-20 14:08:31.640707700 +0000
+++ b/Source/JavaScriptCore/b3/B3FixSSA.h	2026-05-20 15:05:02.879328127 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3Generate.h	2026-01-20 14:08:31.642707800 +0000
+++ b/Source/JavaScriptCore/b3/B3Generate.h	2026-05-20 15:05:02.900328335 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Common.h"
--- a/Source/JavaScriptCore/b3/B3LowerToAir.h	2026-01-20 14:08:31.649707800 +0000
+++ b/Source/JavaScriptCore/b3/B3LowerToAir.h	2026-05-20 15:05:02.919328524 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3MathExtras.h	2026-01-20 14:08:31.650707700 +0000
+++ b/Source/JavaScriptCore/b3/B3MathExtras.h	2026-05-20 15:05:02.937328702 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Origin.h"
--- a/Source/JavaScriptCore/b3/B3MemoryValue.h	2026-02-23 14:40:52.234318300 +0000
+++ b/Source/JavaScriptCore/b3/B3MemoryValue.h	2026-05-20 15:05:02.956328890 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Bank.h"
--- a/Source/JavaScriptCore/b3/B3MoveConstants.h	2026-01-20 14:08:31.651707600 +0000
+++ b/Source/JavaScriptCore/b3/B3MoveConstants.h	2026-05-20 15:05:02.978329108 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3Opcode.h	2026-02-23 14:40:52.235318200 +0000
+++ b/Source/JavaScriptCore/b3/B3Opcode.h	2026-05-20 15:05:02.998329306 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Type.h"
--- a/Source/JavaScriptCore/b3/B3PatchpointSpecial.h	2026-01-20 14:08:31.658707600 +0000
+++ b/Source/JavaScriptCore/b3/B3PatchpointSpecial.h	2026-05-20 15:05:03.021329534 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3StackmapSpecial.h"
--- a/Source/JavaScriptCore/b3/B3PatchpointValue.h	2026-01-20 14:08:31.659707800 +0000
+++ b/Source/JavaScriptCore/b3/B3PatchpointValue.h	2026-05-20 15:05:03.042329742 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Effects.h"
--- a/Source/JavaScriptCore/b3/B3Procedure.h	2026-02-23 14:40:52.236318000 +0000
+++ b/Source/JavaScriptCore/b3/B3Procedure.h	2026-05-20 15:05:03.064329960 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Origin.h"
--- a/Source/JavaScriptCore/b3/B3ReduceStrength.h	2026-01-20 14:08:31.664707700 +0000
+++ b/Source/JavaScriptCore/b3/B3ReduceStrength.h	2026-05-20 15:05:03.085330169 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3SIMDValue.h	2026-01-20 14:08:31.665707600 +0000
+++ b/Source/JavaScriptCore/b3/B3SIMDValue.h	2026-05-20 15:05:03.108330397 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3SlotBaseValue.h	2026-01-20 14:08:31.666707800 +0000
+++ b/Source/JavaScriptCore/b3/B3SlotBaseValue.h	2026-05-20 15:05:03.130330615 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3StackmapGenerationParams.h	2026-02-23 14:40:52.237318000 +0000
+++ b/Source/JavaScriptCore/b3/B3StackmapGenerationParams.h	2026-05-20 15:05:03.149330803 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirGenerationContext.h"
--- a/Source/JavaScriptCore/b3/B3StackmapSpecial.h	2026-01-20 14:08:31.667707700 +0000
+++ b/Source/JavaScriptCore/b3/B3StackmapSpecial.h	2026-05-20 15:05:03.170331011 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "AirArg.h"
--- a/Source/JavaScriptCore/b3/B3StackmapValue.h	2026-01-20 14:08:31.667707700 +0000
+++ b/Source/JavaScriptCore/b3/B3StackmapValue.h	2026-05-20 15:05:03.192331229 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3ConstrainedValue.h"
--- a/Source/JavaScriptCore/b3/B3SwitchValue.h	2026-01-20 14:08:31.668707600 +0000
+++ b/Source/JavaScriptCore/b3/B3SwitchValue.h	2026-05-20 15:05:03.212331427 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3CaseCollection.h"
--- a/Source/JavaScriptCore/b3/B3UpsilonValue.h	2026-01-20 14:08:31.670707700 +0000
+++ b/Source/JavaScriptCore/b3/B3UpsilonValue.h	2026-05-20 15:05:03.231331615 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3UseCounts.h	2026-01-20 14:08:31.670707700 +0000
+++ b/Source/JavaScriptCore/b3/B3UseCounts.h	2026-05-20 15:05:03.250331804 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3Validate.h	2026-01-20 14:08:31.671707600 +0000
+++ b/Source/JavaScriptCore/b3/B3Validate.h	2026-05-20 15:05:03.271332012 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 namespace JSC { namespace B3 {
--- a/Source/JavaScriptCore/b3/B3Value.h	2026-02-23 14:40:52.238318200 +0000
+++ b/Source/JavaScriptCore/b3/B3Value.h	2026-05-20 15:05:03.292332220 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Bank.h"
--- a/Source/JavaScriptCore/b3/B3ValueRep.h	2026-02-23 14:40:52.239318100 +0000
+++ b/Source/JavaScriptCore/b3/B3ValueRep.h	2026-05-20 15:05:03.313332428 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #include <wtf/Platform.h>
 
 #if ENABLE(B3_JIT) || ENABLE(WEBASSEMBLY_BBQJIT)
--- a/Source/JavaScriptCore/b3/B3VariableValue.h	2026-01-20 14:08:31.676707700 +0000
+++ b/Source/JavaScriptCore/b3/B3VariableValue.h	2026-05-20 15:05:03.333332626 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3WasmAddressValue.h	2026-01-20 14:08:31.676707700 +0000
+++ b/Source/JavaScriptCore/b3/B3WasmAddressValue.h	2026-05-20 15:05:03.353332824 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
--- a/Source/JavaScriptCore/b3/B3WasmBoundsCheckValue.h	2026-01-20 14:08:31.676707700 +0000
+++ b/Source/JavaScriptCore/b3/B3WasmBoundsCheckValue.h	2026-05-20 15:05:03.373333023 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <JavaScriptCore/JSExportMacros.h>
+
 #if ENABLE(B3_JIT)
 
 #include "B3Value.h"
