--- a/Source/WebCore/PAL/pal/text/EncodingTables.h
+++ b/Source/WebCore/PAL/pal/text/EncodingTables.h
@@ -28,8 +28,10 @@
 #include <array>
 #include <iterator>
 #include <optional>
+#include <span>
 #include <unicode/umachine.h>
 #include <utility>
+#include <wtf/Assertions.h>
 
 namespace PAL {
 
--- a/Source/WebCore/PAL/pal/text/TextCodecASCIIFastPath.h
+++ b/Source/WebCore/PAL/pal/text/TextCodecASCIIFastPath.h
@@ -24,6 +24,7 @@
 
 #pragma once
 
+#include <span>
 #include <wtf/StdLibExtras.h>
 #include <wtf/text/ASCIIFastPath.h>
 
--- a/Source/WebCore/PAL/pal/text/TextCodecCJK.h
+++ b/Source/WebCore/PAL/pal/text/TextCodecCJK.h
@@ -25,6 +25,7 @@
 #pragma once
 
 #include "TextCodec.h"
+#include <span>
 #include <optional>
 #include <wtf/TZoneMalloc.h>
 
--- a/Source/WebCore/PAL/pal/text/TextCodecICU.h
+++ b/Source/WebCore/PAL/pal/text/TextCodecICU.h
@@ -25,6 +25,7 @@
 #pragma once
 
 #include "TextCodec.h"
+#include <span>
 #include <unicode/ucnv.h>
 #include <wtf/TZoneMalloc.h>
 #include <wtf/text/ASCIILiteral.h>
--- a/Source/WebCore/PAL/pal/text/TextCodecLatin1.h
+++ b/Source/WebCore/PAL/pal/text/TextCodecLatin1.h
@@ -25,6 +25,7 @@
 #pragma once
 
 #include "TextCodec.h"
+#include <span>
 
 namespace PAL {
 
--- a/Source/WebCore/PAL/pal/text/TextCodecReplacement.h
+++ b/Source/WebCore/PAL/pal/text/TextCodecReplacement.h
@@ -25,6 +25,7 @@
 #pragma once
 
 #include "TextCodecUTF8.h"
+#include <span>
 #include <wtf/TZoneMalloc.h>
 
 namespace PAL {
--- a/Source/WebCore/PAL/pal/text/TextCodecSingleByte.h
+++ b/Source/WebCore/PAL/pal/text/TextCodecSingleByte.h
@@ -25,6 +25,7 @@
 #pragma once
 
 #include "TextCodec.h"
+#include <span>
 #include <wtf/TZoneMalloc.h>
 
 namespace PAL {
--- a/Source/WebCore/PAL/pal/text/TextCodecUserDefined.h
+++ b/Source/WebCore/PAL/pal/text/TextCodecUserDefined.h
@@ -25,6 +25,7 @@
 #pragma once
 
 #include "TextCodec.h"
+#include <span>
 #include <wtf/TZoneMalloc.h>
 
 namespace PAL {
--- a/Source/WebCore/PAL/pal/text/TextCodecUTF16.h
+++ b/Source/WebCore/PAL/pal/text/TextCodecUTF16.h
@@ -25,6 +25,7 @@
 #pragma once
 
 #include "TextCodec.h"
+#include <span>
 #include <optional>
 #include <wtf/TZoneMalloc.h>
 
--- a/Source/WebCore/PAL/pal/text/TextCodecUTF8.h
+++ b/Source/WebCore/PAL/pal/text/TextCodecUTF8.h
@@ -25,6 +25,7 @@
 #pragma once
 
 #include "TextCodec.h"
+#include <span>
 #include <unicode/utf8.h>
 #include <wtf/TZoneMalloc.h>
 #include <wtf/text/Latin1Character.h>
--- a/Source/WebCore/PAL/pal/text/TextEncoding.h
+++ b/Source/WebCore/PAL/pal/text/TextEncoding.h
@@ -25,6 +25,7 @@
 #pragma once
 
 #include "UnencodableHandling.h"
+#include <span>
 #include <pal/ExportMacros.h>
 #include <wtf/URL.h>
 #include <wtf/text/StringView.h>
