--- a/Source/WTF/wtf/glib/ActivityObserver.h
+++ b/Source/WTF/wtf/glib/ActivityObserver.h
@@ -20,5 +20,6 @@
 #pragma once
 #include <wtf/Platform.h>
 
+#include <wtf/TZoneMalloc.h>
 #if USE(GLIB_EVENT_LOOP)
 
 #include <wtf/Lock.h>
--- a/Source/WTF/wtf/glib/Application.h
+++ b/Source/WTF/wtf/glib/Application.h
@@ -20,5 +20,6 @@
 #pragma once
 #include <wtf/Platform.h>
 
+#include <wtf/ExportMacros.h>
 #if USE(GLIB)
 
 #include <wtf/text/CString.h>
--- a/Source/WTF/wtf/glib/ChassisType.h
+++ b/Source/WTF/wtf/glib/ChassisType.h
@@ -19,6 +19,7 @@
 
 #pragma once
 
+#include <wtf/ExportMacros.h>
 namespace WTF {
 
 enum class ChassisType {
--- a/Source/WTF/wtf/glib/GRefPtr.h
+++ b/Source/WTF/wtf/glib/GRefPtr.h
@@ -24,5 +24,6 @@
 #pragma once
 #include <wtf/Platform.h>
 
+#include <wtf/ExportMacros.h>
 #if USE(GLIB)
 
 #include <algorithm>
--- a/Source/WTF/wtf/glib/GResources.h
+++ b/Source/WTF/wtf/glib/GResources.h
@@ -20,5 +20,6 @@
 #pragma once
 #include <wtf/Platform.h>
 
+#include <wtf/ExportMacros.h>
 #if USE(GLIB)
 
 namespace WTF {
--- a/Source/WTF/wtf/glib/GUniquePtr.h
+++ b/Source/WTF/wtf/glib/GUniquePtr.h
@@ -21,8 +21,10 @@
 #pragma once
 #include <wtf/Platform.h>
 
+#include <memory>
+#include <wtf/Assertions.h>
 #if USE(GLIB)
 
 #include <gio/gio.h>
 #include <utility>
 #include <wtf/Noncopyable.h>
--- a/Source/WTF/wtf/glib/GSocketMonitor.h
+++ b/Source/WTF/wtf/glib/GSocketMonitor.h
@@ -25,6 +25,7 @@
 
 #pragma once
 
+#include <wtf/ExportMacros.h>
 #include <glib.h>
 #include <wtf/Forward.h>
 #include <wtf/Noncopyable.h>
--- a/Source/WTF/wtf/glib/GSpanExtras.h
+++ b/Source/WTF/wtf/glib/GSpanExtras.h
@@ -19,6 +19,7 @@
 
 #pragma once
 
+#include <wtf/ExportMacros.h>
 #include <wtf/Expected.h>
 #include <wtf/MallocSpan.h>
 #include <wtf/StdLibExtras.h>
--- a/Source/WTF/wtf/glib/Sandbox.h
+++ b/Source/WTF/wtf/glib/Sandbox.h
@@ -19,6 +19,7 @@
 
 #pragma once
 
+#include <wtf/ExportMacros.h>
 #include <wtf/text/WTFString.h>
 
 namespace WTF {
--- a/Source/WTF/wtf/glib/SocketConnection.h
+++ b/Source/WTF/wtf/glib/SocketConnection.h
@@ -19,6 +19,7 @@
 
 #pragma once
 
+#include <wtf/ExportMacros.h>
 #include <wtf/Function.h>
 #include <wtf/HashMap.h>
 #include <wtf/RefCounted.h>
