--- a/Source/JavaScriptCore/heap/AbstractSlotVisitor.h	2026-02-23 14:40:52.311317700 +0000
+++ b/Source/JavaScriptCore/heap/AbstractSlotVisitor.h	2026-05-20 15:50:27.242511923 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/JSCJSValue.h>
 #include <JavaScriptCore/MarkStack.h>
 #include <JavaScriptCore/RootMarkReason.h>
--- a/Source/JavaScriptCore/heap/AbstractSlotVisitorInlines.h	2026-02-23 14:40:52.311317700 +0000
+++ b/Source/JavaScriptCore/heap/AbstractSlotVisitorInlines.h	2026-05-20 15:50:27.263512119 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/AbstractSlotVisitor.h>
 #include <JavaScriptCore/Heap.h>
 #include <JavaScriptCore/WeakInlines.h>
--- a/Source/JavaScriptCore/heap/AlignedMemoryAllocator.h	2026-01-20 14:08:31.990637300 +0000
+++ b/Source/JavaScriptCore/heap/AlignedMemoryAllocator.h	2026-05-20 15:50:27.280512277 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <wtf/PrintStream.h>
 #include <wtf/SinglyLinkedListWithTail.h>
 
--- a/Source/JavaScriptCore/heap/Allocator.h	2026-02-23 14:40:52.311317700 +0000
+++ b/Source/JavaScriptCore/heap/Allocator.h	2026-05-20 15:50:27.300512464 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/AllocationFailureMode.h>
 #include <climits>
 #include <cstdint>
--- a/Source/JavaScriptCore/heap/AllocatorInlines.h	2026-02-23 14:40:52.311317700 +0000
+++ b/Source/JavaScriptCore/heap/AllocatorInlines.h	2026-05-20 15:50:27.319512641 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/Allocator.h>
 #include <JavaScriptCore/LocalAllocator.h>
 
--- a/Source/JavaScriptCore/heap/BlockDirectoryBits.h	2026-02-23 14:40:52.312317800 +0000
+++ b/Source/JavaScriptCore/heap/BlockDirectoryBits.h	2026-05-20 15:50:27.342512855 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <array>
 #include <wtf/FastBitVector.h>
 #include <wtf/TZoneMalloc.h>
--- a/Source/JavaScriptCore/heap/BlockDirectory.cpp	2026-02-23 14:40:52.311317700 +0000
+++ b/Source/JavaScriptCore/heap/BlockDirectory.cpp	2026-05-20 15:50:27.359513014 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "BlockDirectory.h"
 
 #include "BlockDirectoryInlines.h"
--- a/Source/JavaScriptCore/heap/BlockDirectory.h	2026-02-23 14:40:52.311317700 +0000
+++ b/Source/JavaScriptCore/heap/BlockDirectory.h	2026-05-20 15:50:27.378513191 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/AllocationFailureMode.h>
 #include <JavaScriptCore/BlockDirectoryBits.h>
 #include <JavaScriptCore/CellAttributes.h>
--- a/Source/JavaScriptCore/heap/BlockDirectoryInlines.h	2026-02-23 14:40:52.312317800 +0000
+++ b/Source/JavaScriptCore/heap/BlockDirectoryInlines.h	2026-05-20 15:50:27.400513396 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/BlockDirectory.h>
 #include <JavaScriptCore/FreeListInlines.h>
 #include <JavaScriptCore/VM.h>
--- a/Source/JavaScriptCore/heap/CellContainer.h	2026-01-20 14:08:31.993707000 +0000
+++ b/Source/JavaScriptCore/heap/CellContainer.h	2026-05-20 15:50:27.422513601 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <wtf/StdLibExtras.h>
 
 namespace JSC {
--- a/Source/JavaScriptCore/heap/CellContainerInlines.h	2026-02-23 14:40:52.312317800 +0000
+++ b/Source/JavaScriptCore/heap/CellContainerInlines.h	2026-05-20 15:50:27.442513788 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/CellContainer.h>
 #include <JavaScriptCore/JSCast.h>
 #include <JavaScriptCore/MarkedBlock.h>
--- a/Source/JavaScriptCore/heap/CompleteSubspace.cpp	2026-02-23 14:40:52.312317800 +0000
+++ b/Source/JavaScriptCore/heap/CompleteSubspace.cpp	2026-05-20 15:50:27.463513983 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "Subspace.h"
 
 #include "AlignedMemoryAllocator.h"
--- a/Source/JavaScriptCore/heap/CompleteSubspace.h	2026-02-23 14:40:52.312317800 +0000
+++ b/Source/JavaScriptCore/heap/CompleteSubspace.h	2026-05-20 15:50:27.483514170 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/Subspace.h>
 
 namespace JSC {
--- a/Source/JavaScriptCore/heap/CompleteSubspaceInlines.h	2026-02-23 14:40:52.313317800 +0000
+++ b/Source/JavaScriptCore/heap/CompleteSubspaceInlines.h	2026-05-20 15:50:27.500514328 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/CompleteSubspace.h>
 #include <JavaScriptCore/VM.h>
 
--- a/Source/JavaScriptCore/heap/ConservativeRoots.cpp	2026-01-20 14:08:31.995706800 +0000
+++ b/Source/JavaScriptCore/heap/ConservativeRoots.cpp	2026-05-20 15:50:27.516514477 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "ConservativeRoots.h"
 
 #include "CalleeBits.h"
--- a/Source/JavaScriptCore/heap/ConservativeRoots.h	2026-02-23 14:40:52.313317800 +0000
+++ b/Source/JavaScriptCore/heap/ConservativeRoots.h	2026-05-20 15:50:27.533514636 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/Heap.h>
 
 namespace JSC {
--- a/Source/JavaScriptCore/heap/EdenGCActivityCallback.cpp	2026-01-20 14:08:31.997707000 +0000
+++ b/Source/JavaScriptCore/heap/EdenGCActivityCallback.cpp	2026-05-20 15:50:27.548514776 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "EdenGCActivityCallback.h"
 
 #include "VM.h"
--- a/Source/JavaScriptCore/heap/EdenGCActivityCallback.h	2026-02-23 14:40:52.313317800 +0000
+++ b/Source/JavaScriptCore/heap/EdenGCActivityCallback.h	2026-05-20 15:50:27.562514906 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/GCActivityCallback.h>
 
 namespace JSC {
--- a/Source/JavaScriptCore/heap/FastMallocAlignedMemoryAllocator.cpp	2026-01-20 14:08:31.998706800 +0000
+++ b/Source/JavaScriptCore/heap/FastMallocAlignedMemoryAllocator.cpp	2026-05-20 15:50:27.578515055 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "FastMallocAlignedMemoryAllocator.h"
 
 #include <wtf/FastMalloc.h>
--- a/Source/JavaScriptCore/heap/FastMallocAlignedMemoryAllocator.h	2026-02-23 14:40:52.313317800 +0000
+++ b/Source/JavaScriptCore/heap/FastMallocAlignedMemoryAllocator.h	2026-05-20 15:50:27.597515233 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/AlignedMemoryAllocator.h>
 
 #if ENABLE(MALLOC_HEAP_BREAKDOWN)
--- a/Source/JavaScriptCore/heap/FreeList.h	2026-02-23 14:40:52.314317700 +0000
+++ b/Source/JavaScriptCore/heap/FreeList.h	2026-05-20 15:50:27.617515419 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/JSExportMacros.h>
 #include <wtf/Compiler.h>
 #include <wtf/MathExtras.h>
--- a/Source/JavaScriptCore/heap/FreeListInlines.h	2026-02-23 14:40:52.314317700 +0000
+++ b/Source/JavaScriptCore/heap/FreeListInlines.h	2026-05-20 15:50:27.636515596 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/FreeList.h>
 #include <JavaScriptCore/MarkedBlock.h>
 
--- a/Source/JavaScriptCore/heap/FullGCActivityCallback.cpp	2026-01-20 14:08:31.999707000 +0000
+++ b/Source/JavaScriptCore/heap/FullGCActivityCallback.cpp	2026-05-20 15:50:27.657515792 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "FullGCActivityCallback.h"
 #include <wtf/MemoryPressureHandler.h>
 
--- a/Source/JavaScriptCore/heap/FullGCActivityCallback.h	2026-02-23 14:40:52.314317700 +0000
+++ b/Source/JavaScriptCore/heap/FullGCActivityCallback.h	2026-05-20 15:50:27.679515997 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/GCActivityCallback.h>
 
 namespace JSC {
--- a/Source/JavaScriptCore/heap/GCActivityCallback.cpp	2026-01-20 14:08:31.999707000 +0000
+++ b/Source/JavaScriptCore/heap/GCActivityCallback.cpp	2026-05-20 15:50:27.699516183 +0000
@@ -27,6 +27,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "GCActivityCallback.h"
 
 #include "HeapInlines.h"
--- a/Source/JavaScriptCore/heap/GCActivityCallback.h	2026-02-23 14:40:52.314317700 +0000
+++ b/Source/JavaScriptCore/heap/GCActivityCallback.h	2026-05-20 15:50:27.718516361 +0000
@@ -28,6 +28,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/JSRunLoopTimer.h>
 #include <JavaScriptCore/Synchronousness.h>
 #include <wtf/RefPtr.h>
--- a/Source/JavaScriptCore/heap/GCIncomingRefCounted.h	2026-01-20 14:08:32.002707000 +0000
+++ b/Source/JavaScriptCore/heap/GCIncomingRefCounted.h	2026-05-20 15:50:27.738516547 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <wtf/DeferrableRefCounted.h>
 #include <wtf/Forward.h>
 
--- a/Source/JavaScriptCore/heap/GCIncomingRefCountedInlines.h	2026-02-23 14:40:52.314317700 +0000
+++ b/Source/JavaScriptCore/heap/GCIncomingRefCountedInlines.h	2026-05-20 15:50:27.757516724 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/GCIncomingRefCounted.h>
 #include <JavaScriptCore/Heap.h>
 
--- a/Source/JavaScriptCore/heap/GCIncomingRefCountedSet.h	2026-02-23 14:40:52.314317700 +0000
+++ b/Source/JavaScriptCore/heap/GCIncomingRefCountedSet.h	2026-05-20 15:50:27.778516920 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/CollectionScope.h>
 #include <JavaScriptCore/GCIncomingRefCounted.h>
 
--- a/Source/JavaScriptCore/heap/GCIncomingRefCountedSetInlines.h	2026-02-23 14:40:52.314317700 +0000
+++ b/Source/JavaScriptCore/heap/GCIncomingRefCountedSetInlines.h	2026-05-20 15:50:27.798517106 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/GCIncomingRefCountedSet.h>
 #include <JavaScriptCore/VM.h>
 
--- a/Source/JavaScriptCore/heap/GCMemoryOperations.h	2026-02-23 14:40:52.315317600 +0000
+++ b/Source/JavaScriptCore/heap/GCMemoryOperations.h	2026-05-20 15:50:27.818517293 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/CPU.h>
 #include <JavaScriptCore/JSCJSValue.h>
 
--- a/Source/JavaScriptCore/heap/GCSegmentedArray.h	2026-02-23 14:40:52.315317600 +0000
+++ b/Source/JavaScriptCore/heap/GCSegmentedArray.h	2026-05-20 15:50:27.836517461 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/JSExportMacros.h>
 #include <wtf/Compiler.h>
 #include <wtf/DebugHeap.h>
--- a/Source/JavaScriptCore/heap/GCSegmentedArrayInlines.h	2026-02-23 14:40:52.315317600 +0000
+++ b/Source/JavaScriptCore/heap/GCSegmentedArrayInlines.h	2026-05-20 15:50:27.854517628 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/GCSegmentedArray.h>
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
--- a/Source/JavaScriptCore/heap/GigacageAlignedMemoryAllocator.cpp	2026-01-20 14:08:32.005706800 +0000
+++ b/Source/JavaScriptCore/heap/GigacageAlignedMemoryAllocator.cpp	2026-05-20 15:50:27.874517815 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "GigacageAlignedMemoryAllocator.h"
 
 #if ENABLE(MALLOC_HEAP_BREAKDOWN)
--- a/Source/JavaScriptCore/heap/GigacageAlignedMemoryAllocator.h	2026-01-20 14:08:32.005706800 +0000
+++ b/Source/JavaScriptCore/heap/GigacageAlignedMemoryAllocator.h	2026-05-20 15:50:27.897518029 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include "AlignedMemoryAllocator.h"
 #include <wtf/Gigacage.h>
 
--- a/Source/JavaScriptCore/heap/HandleBlock.h	2026-02-23 14:40:52.315317600 +0000
+++ b/Source/JavaScriptCore/heap/HandleBlock.h	2026-05-20 15:50:27.917518216 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <wtf/DoublyLinkedList.h>
 #include <wtf/StdLibExtras.h>
 
--- a/Source/JavaScriptCore/heap/HeapCell.h	2026-02-23 14:40:52.316317800 +0000
+++ b/Source/JavaScriptCore/heap/HeapCell.h	2026-05-20 15:50:27.936518393 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/DestructionMode.h>
 #include <JavaScriptCore/EnsureStillAliveHere.h>
 #include <bit>
--- a/Source/JavaScriptCore/heap/HeapCellInlines.h	2026-02-23 14:40:52.316317800 +0000
+++ b/Source/JavaScriptCore/heap/HeapCellInlines.h	2026-05-20 15:50:27.957518589 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/CellContainer.h>
 #include <JavaScriptCore/HeapCell.h>
 #include <JavaScriptCore/PreciseAllocation.h>
--- a/Source/JavaScriptCore/heap/Heap.cpp	2026-02-23 14:40:52.316317800 +0000
+++ b/Source/JavaScriptCore/heap/Heap.cpp	2026-05-20 15:50:27.976518766 +0000
@@ -19,6 +19,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "Heap.h"
 
 #include "BuiltinExecutables.h"
--- a/Source/JavaScriptCore/heap/Heap.h	2026-02-23 14:40:52.316317800 +0000
+++ b/Source/JavaScriptCore/heap/Heap.h	2026-05-20 15:50:27.996518952 +0000
@@ -21,6 +21,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/ArrayBuffer.h>
 #include <JavaScriptCore/CellState.h>
 #include <JavaScriptCore/CollectionScope.h>
--- a/Source/JavaScriptCore/heap/HeapInlines.h	2026-02-23 14:40:52.317317700 +0000
+++ b/Source/JavaScriptCore/heap/HeapInlines.h	2026-05-20 15:50:28.016519139 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/GCDeferralContext.h>
 #include <JavaScriptCore/Heap.h>
 #include <JavaScriptCore/HeapCellInlines.h>
--- a/Source/JavaScriptCore/heap/IsoCellSet.cpp	2026-01-20 14:08:32.018706800 +0000
+++ b/Source/JavaScriptCore/heap/IsoCellSet.cpp	2026-05-20 15:50:28.037519334 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "IsoCellSet.h"
 
 #include "MarkedBlockInlines.h"
--- a/Source/JavaScriptCore/heap/IsoSubspace.cpp	2026-02-23 14:40:52.319317600 +0000
+++ b/Source/JavaScriptCore/heap/IsoSubspace.cpp	2026-05-20 15:50:28.057519521 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "IsoSubspace.h"
 
 #include "FastMallocAlignedMemoryAllocator.h"
--- a/Source/JavaScriptCore/heap/IsoSubspace.h	2026-02-23 14:40:52.319317600 +0000
+++ b/Source/JavaScriptCore/heap/IsoSubspace.h	2026-05-20 15:50:28.079519726 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/AlignedMemoryAllocator.h>
 #include <JavaScriptCore/BlockDirectory.h>
 #include <JavaScriptCore/Subspace.h>
--- a/Source/JavaScriptCore/heap/IsoSubspaceInlines.h	2026-02-23 14:40:52.319317600 +0000
+++ b/Source/JavaScriptCore/heap/IsoSubspaceInlines.h	2026-05-20 15:50:28.096519884 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/AllocationFailureMode.h>
 #include <JavaScriptCore/IsoSubspace.h>
 #include <JavaScriptCore/VM.h>
--- a/Source/JavaScriptCore/heap/LocalAllocator.cpp	2026-02-23 14:40:52.319317600 +0000
+++ b/Source/JavaScriptCore/heap/LocalAllocator.cpp	2026-05-20 15:50:28.116520071 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "LocalAllocator.h"
 
 #include "AllocatingScope.h"
--- a/Source/JavaScriptCore/heap/LocalAllocator.h	2026-02-23 14:40:52.320317700 +0000
+++ b/Source/JavaScriptCore/heap/LocalAllocator.h	2026-05-20 15:50:28.136520257 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/AllocationFailureMode.h>
 #include <JavaScriptCore/FreeList.h>
 #include <JavaScriptCore/MarkedBlock.h>
--- a/Source/JavaScriptCore/heap/LocalAllocatorInlines.h	2026-02-23 14:40:52.320317700 +0000
+++ b/Source/JavaScriptCore/heap/LocalAllocatorInlines.h	2026-05-20 15:50:28.156520444 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/HeapInlines.h>
 #include <JavaScriptCore/LocalAllocator.h>
 
--- a/Source/JavaScriptCore/heap/MachineStackMarker.cpp	2026-02-23 14:40:52.320317700 +0000
+++ b/Source/JavaScriptCore/heap/MachineStackMarker.cpp	2026-05-20 15:50:28.173520602 +0000
@@ -20,6 +20,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "MachineStackMarker.h"
 
 #include "ConservativeRoots.h"
--- a/Source/JavaScriptCore/heap/MachineStackMarker.h	2026-02-23 14:40:52.320317700 +0000
+++ b/Source/JavaScriptCore/heap/MachineStackMarker.h	2026-05-20 15:50:28.195520807 +0000
@@ -21,6 +21,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/RegisterState.h>
 #include <wtf/Lock.h>
 #include <wtf/ScopedLambda.h>
--- a/Source/JavaScriptCore/heap/MarkedBlock.cpp	2026-02-23 14:40:52.320317700 +0000
+++ b/Source/JavaScriptCore/heap/MarkedBlock.cpp	2026-05-20 15:50:28.216521003 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "MarkedBlock.h"
 
 #include "AlignedMemoryAllocator.h"
--- a/Source/JavaScriptCore/heap/MarkedBlock.h	2026-02-23 14:40:52.321317700 +0000
+++ b/Source/JavaScriptCore/heap/MarkedBlock.h	2026-05-20 15:50:28.237521199 +0000
@@ -21,6 +21,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/CellAttributes.h>
 #include <JavaScriptCore/DestructionMode.h>
 #include <JavaScriptCore/HeapCell.h>
--- a/Source/JavaScriptCore/heap/MarkedBlockInlines.h	2026-02-23 14:40:52.321317700 +0000
+++ b/Source/JavaScriptCore/heap/MarkedBlockInlines.h	2026-05-20 15:50:28.257521385 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/BlockDirectory.h>
 #include <JavaScriptCore/JSCast.h>
 #include <JavaScriptCore/MarkedBlock.h>
--- a/Source/JavaScriptCore/heap/MarkedSpace.cpp	2026-02-23 14:40:52.321317700 +0000
+++ b/Source/JavaScriptCore/heap/MarkedSpace.cpp	2026-05-20 15:50:28.277521572 +0000
@@ -19,6 +19,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "MarkedSpace.h"
 
 #include "BlockDirectoryInlines.h"
--- a/Source/JavaScriptCore/heap/MarkedSpace.h	2026-02-23 14:40:52.322317600 +0000
+++ b/Source/JavaScriptCore/heap/MarkedSpace.h	2026-05-20 15:50:28.297521758 +0000
@@ -21,6 +21,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/BlockDirectory.h>
 #include <JavaScriptCore/MarkedBlock.h>
 #include <JavaScriptCore/MarkedBlockSet.h>
--- a/Source/JavaScriptCore/heap/MarkedSpaceInlines.h	2026-02-23 14:40:52.322317600 +0000
+++ b/Source/JavaScriptCore/heap/MarkedSpaceInlines.h	2026-05-20 15:50:28.318521954 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/MarkedBlockInlines.h>
 #include <JavaScriptCore/MarkedSpace.h>
 
--- a/Source/JavaScriptCore/heap/MarkingConstraint.h	2026-02-23 14:40:52.322317600 +0000
+++ b/Source/JavaScriptCore/heap/MarkingConstraint.h	2026-05-20 15:50:28.338522140 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/ConstraintConcurrency.h>
 #include <JavaScriptCore/ConstraintParallelism.h>
 #include <JavaScriptCore/ConstraintVolatility.h>
--- a/Source/JavaScriptCore/heap/MarkingConstraintSolver.cpp	2026-02-23 14:40:52.322317600 +0000
+++ b/Source/JavaScriptCore/heap/MarkingConstraintSolver.cpp	2026-05-20 15:50:28.360522345 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "MarkingConstraintSolver.h"
 
 #include "JSCInlines.h"
--- a/Source/JavaScriptCore/heap/MarkStack.cpp	2026-01-20 14:08:32.022706700 +0000
+++ b/Source/JavaScriptCore/heap/MarkStack.cpp	2026-05-20 15:50:28.385522578 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "MarkStack.h"
 
 #include "GCSegmentedArrayInlines.h"
--- a/Source/JavaScriptCore/heap/MarkStack.h	2026-02-23 14:40:52.320317700 +0000
+++ b/Source/JavaScriptCore/heap/MarkStack.h	2026-05-20 15:50:28.406522774 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/GCSegmentedArray.h>
 
 namespace JSC {
--- a/Source/JavaScriptCore/heap/MarkStackMergingConstraint.cpp	2026-01-20 14:08:32.023707000 +0000
+++ b/Source/JavaScriptCore/heap/MarkStackMergingConstraint.cpp	2026-05-20 15:50:28.426522961 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "MarkStackMergingConstraint.h"
 
 #include "GCSegmentedArrayInlines.h"
--- a/Source/JavaScriptCore/heap/PreciseAllocation.cpp	2026-01-20 14:08:32.029706700 +0000
+++ b/Source/JavaScriptCore/heap/PreciseAllocation.cpp	2026-05-20 15:50:28.447523156 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "PreciseAllocation.h"
 
 #include "AlignedMemoryAllocator.h"
--- a/Source/JavaScriptCore/heap/PreciseAllocation.h	2026-02-23 14:40:52.324317700 +0000
+++ b/Source/JavaScriptCore/heap/PreciseAllocation.h	2026-05-20 15:50:28.472523390 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/MarkedBlock.h>
 #include <JavaScriptCore/WeakSet.h>
 #include <wtf/StdLibExtras.h>
--- a/Source/JavaScriptCore/heap/PreciseSubspace.cpp	2026-01-20 14:08:32.030707000 +0000
+++ b/Source/JavaScriptCore/heap/PreciseSubspace.cpp	2026-05-20 15:50:28.493523585 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "PreciseSubspace.h"
 
 #include "AllocatingScope.h"
--- a/Source/JavaScriptCore/heap/PreciseSubspace.h	2026-02-23 14:40:52.324317700 +0000
+++ b/Source/JavaScriptCore/heap/PreciseSubspace.h	2026-05-20 15:50:28.514523781 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/Subspace.h>
 #include <JavaScriptCore/SubspaceAccess.h>
 
--- a/Source/JavaScriptCore/heap/SlotVisitor.cpp	2026-01-20 14:08:32.032706700 +0000
+++ b/Source/JavaScriptCore/heap/SlotVisitor.cpp	2026-05-20 15:50:28.535523977 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "SlotVisitor.h"
 
 #include "ConservativeRoots.h"
--- a/Source/JavaScriptCore/heap/SlotVisitor.h	2026-02-23 14:40:52.324317700 +0000
+++ b/Source/JavaScriptCore/heap/SlotVisitor.h	2026-05-20 15:50:28.555524163 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/AbstractSlotVisitor.h>
 #include <JavaScriptCore/HandleTypes.h>
 #include <wtf/Forward.h>
--- a/Source/JavaScriptCore/heap/SlotVisitorInlines.h	2026-02-23 14:40:52.324317700 +0000
+++ b/Source/JavaScriptCore/heap/SlotVisitorInlines.h	2026-05-20 15:50:28.575524350 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/AbstractSlotVisitorInlines.h>
 #include <JavaScriptCore/HeapCellInlines.h>
 #include <JavaScriptCore/MarkedBlock.h>
--- a/Source/JavaScriptCore/heap/StructureAlignedMemoryAllocator.cpp	2026-02-23 14:40:52.325317600 +0000
+++ b/Source/JavaScriptCore/heap/StructureAlignedMemoryAllocator.cpp	2026-05-20 15:50:28.598524564 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "StructureAlignedMemoryAllocator.h"
 
 #include "JSCConfig.h"
--- a/Source/JavaScriptCore/heap/StructureAlignedMemoryAllocator.h	2026-01-20 14:08:32.037706900 +0000
+++ b/Source/JavaScriptCore/heap/StructureAlignedMemoryAllocator.h	2026-05-20 15:50:28.618524751 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include "AlignedMemoryAllocator.h"
 #include <wtf/Gigacage.h>
 
--- a/Source/JavaScriptCore/heap/SubspaceInlines.h	2026-02-23 14:40:52.325317600 +0000
+++ b/Source/JavaScriptCore/heap/SubspaceInlines.h	2026-05-20 15:50:28.636524918 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/BlockDirectoryInlines.h>
 #include <JavaScriptCore/HeapCellType.h>
 #include <JavaScriptCore/JSCast.h>
--- a/Source/JavaScriptCore/heap/VerifierSlotVisitor.cpp	2026-02-23 14:40:52.326317500 +0000
+++ b/Source/JavaScriptCore/heap/VerifierSlotVisitor.cpp	2026-05-20 15:50:28.655525095 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "VerifierSlotVisitor.h"
 
 #include "BlockDirectoryInlines.h"
--- a/Source/JavaScriptCore/heap/VerifierSlotVisitor.h	2026-02-23 14:40:52.326317500 +0000
+++ b/Source/JavaScriptCore/heap/VerifierSlotVisitor.h	2026-05-20 15:50:28.673525263 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/AbstractSlotVisitor.h>
 #include <JavaScriptCore/JSCJSValue.h>
 #include <JavaScriptCore/MarkedBlock.h>
--- a/Source/JavaScriptCore/heap/VisitCounter.h	2026-01-20 14:08:32.041706800 +0000
+++ b/Source/JavaScriptCore/heap/VisitCounter.h	2026-05-20 15:50:28.694525459 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include "AbstractSlotVisitor.h"
 
 namespace JSC {
--- a/Source/JavaScriptCore/heap/WeakBlock.cpp	2026-02-23 14:40:52.326317500 +0000
+++ b/Source/JavaScriptCore/heap/WeakBlock.cpp	2026-05-20 15:50:28.715525655 +0000
@@ -24,6 +24,7 @@
  */
 
 #include "config.h"
+#include <cstddef>
 #include "WeakBlock.h"
 
 #include "CellContainerInlines.h"
--- a/Source/JavaScriptCore/heap/WeakBlock.h	2026-02-23 14:40:52.326317500 +0000
+++ b/Source/JavaScriptCore/heap/WeakBlock.h	2026-05-20 15:50:28.736525851 +0000
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstddef>
+
 #include <JavaScriptCore/CellContainer.h>
 #include <JavaScriptCore/WeakImpl.h>
 #include <wtf/DebugHeap.h>
